<!DOCTYPE HTML>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>Sequence - FHIR v3.5.0</title>

  
  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
  <meta name="author" content="http://hl7.org/fhir"/>

  <link rel="stylesheet" href="fhir.css"/>
  <link rel="Prev" href="http://hl7.org/fhir/sequence.html"/>

    <!-- Bootstrap core CSS -->
  <link rel="stylesheet" href="./dist/css/bootstrap.css"/>
  <link rel="stylesheet" href="./assets/css/bootstrap-fhir.css"/>

    <!-- Project extras -->
  <link rel="stylesheet" href="./assets/css/project.css"/>
  <link rel="stylesheet" href="./assets/css/pygments-manni.css"/>
	<link rel="stylesheet" href="jquery-ui.css"/>

    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- [if lt IE 9]>
  <script src="./assets/js/html5shiv.js"></script>
  <script src="./assets/js/respond.min.js"></script>
  <![endif] -->

    <!-- Favicons -->
  <link sizes="144x144" rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-144-precomposed.png"/>
  <link sizes="114x114" rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-114-precomposed.png"/>
  <link sizes="72x72" rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-72-precomposed.png"/>
  <link rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-57-precomposed.png"/>
  <link rel="shortcut icon" href="./assets/ico/favicon.png"/>

</head>
<body>
	<div id="segment-header" class="segment">  <!-- segment-header -->	
		<div class="container">  <!-- container -->
			<a no-external="true" id="logo" href="http://hl7.org/fhir"><img src="./assets/images/fhir-logo-www.png" alt="logo fhir"/> </a>
			<div id="hl7-status">
				<b>Custom NCATS Translator JHU Build</b><br /> <a href="https://ncats.nih.gov"><img src="https://ncats.nih.gov/sites/all/themes/ncats-2014/images/assets/ncats-logo.png" height="50" /></a> <a href="https://www.hopkinsmedicine.org/"><img src="/jhulogo.png" height="100" /></a>
			</div>       	
			
			<div id="hl7-nav">
				 <a no-external="true" id="hl7-logo" href="http://www.hl7.org">
					<img src="./assets/images/hl7-logo.png" width="42" alt="visit the hl7 website" height="50"/>
				</a> 
			</div>       	
			<div id="hl7-nav"><a no-external="true" id="hl7-logo" href="http://hl7.org/fhir/search.cfm"><img src="./assets/images/search.png" alt="Search FHIR"/></a></div> 
		</div>
		<div class="container">  <!-- container -->
	</div></div>  <!-- /segment-header -->	


	<div id="segment-navbar" class="segment">  <!-- segment-navbar -->
		<div id="stripe"> </div>
		<div class="container">  <!-- container -->
   <!-- HEADER CONTENT -->
 	      		 	      		      		
			<nav class="navbar navbar-inverse">
				<div class="container">
					<button data-target=".navbar-inverse-collapse" data-toggle="collapse" type="button" class="navbar-toggle">
						<span class="icon-bar"> </span>
						<span class="icon-bar"> </span>
						<span class="icon-bar"> </span>
					</button>
					<a href="index.html" class="navbar-brand hidden">FHIR</a>
					<div class="nav-collapse collapse navbar-inverse-collapse">
						<ul class="nav navbar-nav">
              <li><a href="./index.html">Home</a></li>
              <li><a href="./modules.html">Getting Started</a></li>
              <li><a href="./documentation.html">Documentation</a></li>
              <li><a href="./resourcelist.html">Resources</a></li>
              <li><a href="./profilelist.html">Profiles</a></li>
              <li><a href="./extensibility-registry.html">Extensions</a></li>
              <li><a href="./operationslist.html">Operations</a></li>
              <li><a href="./terminologies-systems.html">Terminologies</a></li>
						</ul>
					</div>  <!-- /.nav-collapse -->
				</div>  <!-- /.container -->
			</nav>  <!-- /.navbar -->
      		
  <!-- /HEADER CONTENT -->				
		</div>  <!-- /container -->
	</div>  <!-- /segment-navbar -->
	

	<div id="segment-breadcrumb" class="segment">  <!-- segment-breadcrumb -->
		<div class="container">  <!-- container -->	
			<ul class="breadcrumb">
        <li><a href="diagnostics-module.html"><img src="diagnostics.jpg"/> Diagnostics</a></li>        <li><b>Sequence</b></li>          <!-- sequence.html / resource / Sequence -->

			</ul>	
		</div>  <!-- /container -->
	</div>  <!-- /segment-breadcrumb -->
	

	<div id="segment-content" class="segment">  <!-- segment-content -->
	<div class="container">  <!-- container -->
            <div class="row">
            	<div class="inner-wrapper">
  <!-- CONTENT CONTENT -->


<div class="col-12">

<ul class="nav nav-tabs"><li class="active"><a href="#">Content</a></li><li><a href="sequence-examples.html">Examples</a></li><li><a href="sequence-definitions.html">Detailed Descriptions</a></li><li><a href="sequence-mappings.html">Mappings</a></li><li><a href="sequence-profiles.html">Profiles &amp; Extensions</a></li><li><a href="sequence-version-maps.html">R3 Conversions</a></li></ul>

<h1 class="self-link-parent"><span class="sectioncount">10.6<a name="10.6"> </a></span> Resource Sequence - Content <a href="sequence.html#10.6" title="link to here" class="self-link"><svg viewBox="0 0 1792 1792" width="20" class="self-link" height="20"><path d="M1520 1216q0-40-28-68l-208-208q-28-28-68-28-42 0-72 32 3 3 19 18.5t21.5 21.5 15 19 13 25.5 3.5 27.5q0 40-28 68t-68 28q-15 0-27.5-3.5t-25.5-13-19-15-21.5-21.5-18.5-19q-33 31-33 73 0 40 28 68l206 207q27 27 68 27 40 0 68-26l147-146q28-28 28-67zm-703-705q0-40-28-68l-206-207q-28-28-68-28-39 0-68 27l-147 146q-28 28-28 67 0 40 28 68l208 208q27 27 68 27 42 0 72-31-3-3-19-18.5t-21.5-21.5-15-19-13-25.5-3.5-27.5q0-40 28-68t68-28q15 0 27.5 3.5t25.5 13 19 15 21.5 21.5 18.5 19q33-31 33-73zm895 705q0 120-85 203l-147 146q-83 83-203 83-121 0-204-85l-206-207q-83-83-83-203 0-123 88-209l-88-88q-86 88-208 88-120 0-204-84l-208-208q-84-84-84-204t85-203l147-146q83-83 203-83 121 0 204 85l206 207q83 83 83 203 0 123-88 209l88 88q86-88 208-88 120 0 204 84l208 208q84 84 84 204z" fill="navy"></path></svg></a></h1>

<table class="colstu"><tr><td><a href="http://www.hl7.org/Special/committees/clingenomics/index.cfm">Clinical Genomics <img src="external.png" style="text-align: baseline"/></a> Work Group</td><td><a href="versions.html#maturity">Maturity Level</a>: 1</td><td>&nbsp;<a href="versions.html#std-process" title="Maturity Level">Trial Use</a></td>  <!-- !ns! --><td><a href="compartmentdefinition.html">Compartments</a>: Not linked to any defined compartments</td></tr></table>


<p>
Raw data describing a biological sequence.
</p>

<blockquote class="stu-note">
  <p>
At present, the Sequence resource is multi-purpose.  It can be used to represent a “definitional” sequence, as might be found in a knowledge-base source in a patient- and specimen-independent manner.  It can also be used to represent information about a specific finding when testing a specimen, including qualifying information about confidence, uncertainty, etc.  The Clinical Genomics workgroup is considering modifying the resource to be definition-only, removing the ability to convey patient, specimen and observation-specific information.  Such data elements would instead be conveyed using the Observation resource, as this is more typically used in the lab setting to report patient-specific results.  The Sequence resource would be retained for sharing of genetic knowledge, such as reference sequences, quality metrics, and links to external repository sequences.  Feedback about this proposed change is welcome.
  </p>
  <p>
For insight into what the new approach to representing patient-specific genetic information, refer to the Implementation Guide (IG) for clinical genomics reporting, which can be found <a href="http://hl7.org/fhir/uv/genomics-reporting/index.html">here <img src="external.png" style="text-align: baseline"/></a>.
</p>
</blockquote>
<div>
    
<h2 class="self-link-parent"><span class="sectioncount">10.6.1<a name="10.6.1"> </a></span> Scope and Usage <a href="sequence.html#10.6.1" title="link to here" class="self-link"><svg viewBox="0 0 1792 1792" width="20" class="self-link" height="20"><path d="M1520 1216q0-40-28-68l-208-208q-28-28-68-28-42 0-72 32 3 3 19 18.5t21.5 21.5 15 19 13 25.5 3.5 27.5q0 40-28 68t-68 28q-15 0-27.5-3.5t-25.5-13-19-15-21.5-21.5-18.5-19q-33 31-33 73 0 40 28 68l206 207q27 27 68 27 40 0 68-26l147-146q28-28 28-67zm-703-705q0-40-28-68l-206-207q-28-28-68-28-39 0-68 27l-147 146q-28 28-28 67 0 40 28 68l208 208q27 27 68 27 42 0 72-31-3-3-19-18.5t-21.5-21.5-15-19-13-25.5-3.5-27.5q0-40 28-68t68-28q15 0 27.5 3.5t25.5 13 19 15 21.5 21.5 18.5 19q33-31 33-73zm895 705q0 120-85 203l-147 146q-83 83-203 83-121 0-204-85l-206-207q-83-83-83-203 0-123 88-209l-88-88q-86 88-208 88-120 0-204-84l-208-208q-84-84-84-204t85-203l147-146q83-83 203-83 121 0 204 85l206 207q83 83 83 203 0 123-88 209l88 88q86-88 208-88 120 0 204 84l208 208q84 84 84 204z" fill="navy"></path></svg></a></h2>
<p>
    The Sequence resource is designed to describe an atomic sequence which contains the alignment sequencing test result and multiple variations. Atomic sequences can be connected by link element and they will lead to sequence graph. By this method, a sequence can be reported. Complete genetic sequence information, of which specific genetic variations are a part, is reported by reference to the GA4GH repository.  Thus, the FHIR Sequence resource avoids large genomic payloads in a manner analogous to how the FHIR ImagingStudy resource references large images maintained in other systems.
    For use cases, details on how this resource interact with other Clinical Genomics resources or profiles, please refer to implementation guidance document <a href="genomics.html">here </a>.
</p>
<h3 class="self-link-parent"><span class="sectioncount">10.6.1.1<a name="10.6.1.1"> </a></span> Genetic Standards and Resources include: <a href="sequence.html#10.6.1.1" title="link to here" class="self-link"><svg viewBox="0 0 1792 1792" width="20" class="self-link" height="20"><path d="M1520 1216q0-40-28-68l-208-208q-28-28-68-28-42 0-72 32 3 3 19 18.5t21.5 21.5 15 19 13 25.5 3.5 27.5q0 40-28 68t-68 28q-15 0-27.5-3.5t-25.5-13-19-15-21.5-21.5-18.5-19q-33 31-33 73 0 40 28 68l206 207q27 27 68 27 40 0 68-26l147-146q28-28 28-67zm-703-705q0-40-28-68l-206-207q-28-28-68-28-39 0-68 27l-147 146q-28 28-28 67 0 40 28 68l208 208q27 27 68 27 42 0 72-31-3-3-19-18.5t-21.5-21.5-15-19-13-25.5-3.5-27.5q0-40 28-68t68-28q15 0 27.5 3.5t25.5 13 19 15 21.5 21.5 18.5 19q33-31 33-73zm895 705q0 120-85 203l-147 146q-83 83-203 83-121 0-204-85l-206-207q-83-83-83-203 0-123 88-209l-88-88q-86 88-208 88-120 0-204-84l-208-208q-84-84-84-204t85-203l147-146q83-83 203-83 121 0 204 85l206 207q83 83 83 203 0 123-88 209l88 88q86-88 208-88 120 0 204 84l208 208q84 84 84 204z" fill="navy"></path></svg></a></h3>
<ul>
    <li>Variant Databases: <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi">dbSNP <img src="external.png" style="text-align: baseline"/></a> , <a href="http://www.ncbi.nlm.nih.gov/clinvar/">ClinVar <img src="external.png" style="text-align: baseline"/></a>, and <a href="http://cancer.sanger.ac.uk/cosmic/">COSMIC <img src="external.png" style="text-align: baseline"/></a> </li>
    <li>Reference Sequences: <a href="http://www.ncbi.nlm.nih.gov/refseq/">RefSeq <img src="external.png" style="text-align: baseline"/></a> and <a href="http://www.ensembl.org/index.html">ENSEMBL <img src="external.png" style="text-align: baseline"/></a> </li>
</ul>
<p>
    This resource is designed to describe sequence variations with clinical significance with information such as:
</p>
    <ul>
        <li>Name of the variation represented</li>
        <li>Type of the variation</li>
        <li>Gene region occupied by the variation</li>
        <li>Tissue source used to determine genotype of the variation</li>
	 <li>Quality of the result</li>
    </ul>
<p>
    It is strongly encouraged to provide all available information in this resource for any reported variants, because receiving systems (e.g. discovery research, outcomes analysis, and public health reporting) may use this information to normalize variants over time or across sources.  However, these data should not be used to dynamically correct/change variant representations for clinical use outside of the laboratory, due to insufficient information.
</p>
<p>
    Implementers should be aware that semantic equivalency of results of genetic variants cannot be guaranteed unless there is an agreed upon standard between sending and receiving systems.
</p>

</div>

<div>
<h2 class="self-link-parent"><span class="sectioncount">10.6.2<a name="10.6.2"> </a></span> Boundaries and Relationships <a href="sequence.html#10.6.2" title="link to here" class="self-link"><svg viewBox="0 0 1792 1792" width="20" class="self-link" height="20"><path d="M1520 1216q0-40-28-68l-208-208q-28-28-68-28-42 0-72 32 3 3 19 18.5t21.5 21.5 15 19 13 25.5 3.5 27.5q0 40-28 68t-68 28q-15 0-27.5-3.5t-25.5-13-19-15-21.5-21.5-18.5-19q-33 31-33 73 0 40 28 68l206 207q27 27 68 27 40 0 68-26l147-146q28-28 28-67zm-703-705q0-40-28-68l-206-207q-28-28-68-28-39 0-68 27l-147 146q-28 28-28 67 0 40 28 68l208 208q27 27 68 27 42 0 72-31-3-3-19-18.5t-21.5-21.5-15-19-13-25.5-3.5-27.5q0-40 28-68t68-28q15 0 27.5 3.5t25.5 13 19 15 21.5 21.5 18.5 19q33-31 33-73zm895 705q0 120-85 203l-147 146q-83 83-203 83-121 0-204-85l-206-207q-83-83-83-203 0-123 88-209l-88-88q-86 88-208 88-120 0-204-84l-208-208q-84-84-84-204t85-203l147-146q83-83 203-83 121 0 204 85l206 207q83 83 83 203 0 123-88 209l88 88q86-88 208-88 120 0 204 84l208 208q84 84 84 204z" fill="navy"></path></svg></a></h2>

<p>
    Focus of the resource is to provide sequencing alignment data immediately relevant to what the interpretation on clinical decision-making originates from. Hence data such as precise read of DNA sequences and sequence alignment are not included; such data are nonetheless accessible through references to <a href="http://ga4gh.org/#/">GA4GH <img src="external.png" style="text-align: baseline"/></a> (Global Alliance for Genomics and Health) API.
    
    The Sequence resource will be referenced by <a href="observation.html">Observation</a> to provide variant information. As clinical assessments/diagnosis of a patient are typically captured in the <a href="condition.html">Condition</a> resource or the ClinicalImpression resource, the Sequence resource can be referenced by the Condition resource to provide specific genetic data to support an assertions. This is analogous to how Condition references other resources, such as <a href="allergyintolerance.html">AllergyIntolerance</a>, <a href="procedure.html">Procedure</a>, and <a href="questionnaire.html">Questionnaire</a> resources.
</p>

</div>

  <!-- (optional):
<div>
<h2>Background and Context</h2>
<p>
Provides additional detail on exactly how the resource is to be used
</p>
</div> -->

<p>This resource is referenced by <a href="observation.html#Observation">Observation</a> and itself</p>


<a name="resource"></a>
<h2 class="self-link-parent"><span class="sectioncount">10.6.3</span> 
Resource Content
 <a href="sequence.html#resource" title="link to here" class="self-link"><svg viewBox="0 0 1792 1792" width="20" class="self-link" height="20"><path d="M1520 1216q0-40-28-68l-208-208q-28-28-68-28-42 0-72 32 3 3 19 18.5t21.5 21.5 15 19 13 25.5 3.5 27.5q0 40-28 68t-68 28q-15 0-27.5-3.5t-25.5-13-19-15-21.5-21.5-18.5-19q-33 31-33 73 0 40 28 68l206 207q27 27 68 27 40 0 68-26l147-146q28-28 28-67zm-703-705q0-40-28-68l-206-207q-28-28-68-28-39 0-68 27l-147 146q-28 28-28 67 0 40 28 68l208 208q27 27 68 27 42 0 72-31-3-3-19-18.5t-21.5-21.5-15-19-13-25.5-3.5-27.5q0-40 28-68t68-28q15 0 27.5 3.5t25.5 13 19 15 21.5 21.5 18.5 19q33-31 33-73zm895 705q0 120-85 203l-147 146q-83 83-203 83-121 0-204-85l-206-207q-83-83-83-203 0-123 88-209l-88-88q-86 88-208 88-120 0-204-84l-208-208q-84-84-84-204t85-203l147-146q83-83 203-83 121 0 204 85l206 207q83 83 83 203 0 123-88 209l88 88q86-88 208-88 120 0 204 84l208 208q84 84 84 204z" fill="navy"></path></svg></a></h2>
<a name="def"></a>
<a name="sequence"></a>
<a name="Sequence"></a>

<div id="tabs">
  <ul>
    <li><a href="#tabs-struc">Structure</a></li>
    <li><a href="#tabs-uml">UML</a></li>
    <li><a href="#tabs-xml">XML</a></li>
    <li><a href="#tabs-json">JSON</a></li>
    <li><a href="#tabs-ttl">Turtle</a></li>
    <li><a href="#tabs-diff">R3 Diff</a></li>
    <li><a href="#tabs-all">All</a></li>
  </ul>
  <div id="tabs-struc">

<div id="tbl">
<p><b>Structure</b></p>
 <div id="tbl-inner">
  <table border="0" cellpadding="0" cellspacing="0" style="border: 0px #F0F0F0 solid; font-size: 11px; font-family: verdana; vertical-align: top;"><tr style="border: 1px #F0F0F0 solid; font-size: 11px; font-family: verdana; vertical-align: top;"><th style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="formats.html#table" title="The logical name of the element">Name</a></th><th style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="formats.html#table" title="Information about the use of the element">Flags</a></th><th style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="formats.html#table" title="Minimum and Maximum # of times the the element can appear in the instance">Card.</a></th><th style="width: 100px" class="hierarchy"><a href="formats.html#table" title="Reference to the type of the element">Type</a></th><th style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="formats.html#table" title="Additional information about the element">Description &amp; Constraints</a><span style="float: right"><a href="formats.html#table" title="Legend for this format"><img src="help16.png" alt="doco" style="background-color: inherit"/></a></span></th></tr><tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_resource.png" alt="." style="background-color: white; background-color: inherit" title="Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence" title="Sequence : Raw data describing a biological sequence.">Sequence</a><a name="Sequence"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="conformance-rules.html#constraints" title="This element has or is affected by some invariants">I</a><a style="padding-left: 3px; padding-right: 3px; border: 1px grey solid; font-weight: bold; color: black; background-color: #fff5e6" href="versions.html#std-process" title="Ballot Status = Trial Use">TU</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="domainresource.html">DomainResource</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Information about a biological sequence<br/><span style="font-style: italic" title="seq-3">+ Rule: Only 0 and 1 are valid for coordinateSystem</span><br/>Elements defined in Ancestors: <a href="resource.html#Resource" title="The logical id of the resource, as used in the URL for the resource. Once assigned, this value never changes.">id</a>, <a href="resource.html#Resource" title="The metadata about the resource. This is content that is maintained by the infrastructure. Changes to the content might not always be associated with version changes to the resource.">meta</a>, <a href="resource.html#Resource" title="A reference to a set of rules that were followed when the resource was constructed, and which must be understood when processing the content. Often, this is a reference to an implementation guide that defines the special rules along with other profiles etc.">implicitRules</a>, <a href="resource.html#Resource" title="The base language in which the resource is written.">language</a>, <a href="domainresource.html#DomainResource" title="A human-readable narrative that contains a summary of the resource and can be used to represent the content of the resource to a human. The narrative need not encode all the structured data, but is required to contain sufficient detail to make it &quot;clinically safe&quot; for a human to just read the narrative. Resource definitions may define what content should be represented in the narrative to ensure clinical safety.">text</a>, <a href="domainresource.html#DomainResource" title="These resources do not have an independent existence apart from the resource that contains them - they cannot be identified independently, and nor can they have their own independent transaction scope.">contained</a>, <a href="domainresource.html#DomainResource" title="May be used to represent additional information that is not part of the basic definition of the resource. To make the use of extensions safe and manageable, there is a strict set of governance  applied to the definition and use of extensions. Though any implementer can define an extension, there is a set of requirements that SHALL be met as part of the definition of the extension.">extension</a>, <a href="domainresource.html#DomainResource" title="May be used to represent additional information that is not part of the basic definition of the resource, and that modifies the understanding of the element that contains it. Usually modifier elements provide negation or qualification. To make the use of extensions safe and manageable, there is a strict set of governance applied to the definition and use of extensions. Though any implementer is allowed to define an extension, there is a set of requirements that SHALL be met as part of the definition of the extension. Applications processing a resource are required to check for modifier extensions.

Modifier extensions SHALL NOT change the meaning of any elements on Resource or DomainResource (including cannot change the meaning of modifierExtension itself).">modifierExtension</a></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.identifier" title="Sequence.identifier : A unique identifier for this particular sequence instance. This is a FHIR-defined id.">identifier</a><a name="Sequence.identifier"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#Identifier">Identifier</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Unique ID for this particular sequence. This is a FHIR-defined id<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.type" title="Sequence.type : Amino Acid Sequence/ DNA Sequence / RNA Sequence.">type</a><a name="Sequence.type"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#code">code</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">aa | dna | rna<br/><a href="valueset-sequence-type.html" title="Type if a sequence -- DNA, RNA, or amino acid sequence.">sequenceType</a> (<a href="terminologies.html#example" title="Instances are not expected or even encouraged to draw from the specified value set.  The value set merely provides examples of the types of concepts intended to be included.">Example</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.coordinateSystem" title="Sequence.coordinateSystem : Whether the sequence is numbered starting at 0 (0-based numbering or coordinates, inclusive start, exclusive end) or starting at 1 (1-based numbering, inclusive start and inclusive end).">coordinateSystem</a><a name="Sequence.coordinateSystem"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Base number of coordinate system (0 for 0-based numbering or coordinates, inclusive start, exclusive end, 1 for 1-based numbering, inclusive start, inclusive end)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.patient" title="Sequence.patient : The patient whose sequencing results are described by this resource.">patient</a><a name="Sequence.patient"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html#Reference">Reference</a>(<a href="patient.html">Patient</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Who and/or what this is about</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.specimen" title="Sequence.specimen : Specimen used for sequencing.">specimen</a><a name="Sequence.specimen"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html#Reference">Reference</a>(<a href="specimen.html">Specimen</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Specimen used for sequencing</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.device" title="Sequence.device : The method for sequencing, for example, chip information.">device</a><a name="Sequence.device"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html#Reference">Reference</a>(<a href="device.html">Device</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">The method for sequencing</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.performer" title="Sequence.performer : The organization or lab that should be responsible for this result.">performer</a><a name="Sequence.performer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html#Reference">Reference</a>(<a href="organization.html">Organization</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Who should be responsible for test result</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quantity" title="Sequence.quantity : The number of copies of the seqeunce of interest. (RNASeq).">quantity</a><a name="Sequence.quantity"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#Quantity">Quantity</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">The number of copies of the seqeunce of interest.  (RNASeq)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck11.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq" title="Sequence.referenceSeq : A sequence that is used as a reference to describe variants that are present in a sequence analyzed.">referenceSeq</a><a name="Sequence.referenceSeq"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="conformance-rules.html#constraints" title="This element has or is affected by some invariants">I</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">A sequence used as reference<br/><span style="font-style: italic" title="seq-5">+ Rule: GenomeBuild and chromosome must be both contained if either one of them is contained</span><br/><span style="font-style: italic" title="seq-6">+ Rule: Have and only have one of the following elements in referenceSeq : 1. genomeBuild ; 2 referenceSeqId; 3. referenceSeqPointer;  4. referenceSeqString;</span></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.chromosome" title="Sequence.referenceSeq.chromosome : Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340)).">chromosome</a><a name="Sequence.referenceSeq.chromosome"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Chromosome containing genetic finding<br/><a href="valueset-chromosome-human.html" title="Chromosome number for human.">chromosome-human</a> (<a href="terminologies.html#example" title="Instances are not expected or even encouraged to draw from the specified value set.  The value set merely provides examples of the types of concepts intended to be included.">Example</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild" title="Sequence.referenceSeq.genomeBuild : The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used.">genomeBuild</a><a name="Sequence.referenceSeq.genomeBuild"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.orientation" title="Sequence.referenceSeq.orientation : A relative reference to a DNA strand based on gene orientation. The strand that contains the open reading frame of the gene is the &quot;sense&quot; strand, and the opposite complementary strand is the &quot;antisense&quot; strand.">orientation</a><a name="Sequence.referenceSeq.orientation"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#code">code</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">sense | antisense<br/><a href="valueset-orientation-type.html" title="Type for orientation.">orientationType</a> (<a href="terminologies.html#required" title="To be conformant, the concept in this element SHALL be from the specified value set.">Required</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId" title="Sequence.referenceSeq.referenceSeqId : Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences.">referenceSeqId</a><a name="Sequence.referenceSeq.referenceSeqId"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Reference identifier<br/><a href="valueset-sequence-referenceSeq.html" title="Reference identifier.">E n s e m b l</a> (<a href="terminologies.html#example" title="Instances are not expected or even encouraged to draw from the specified value set.  The value set merely provides examples of the types of concepts intended to be included.">Example</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer" title="Sequence.referenceSeq.referenceSeqPointer : A Pointer to another Sequence entity as reference sequence.">referenceSeqPointer</a><a name="Sequence.referenceSeq.referenceSeqPointer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html#Reference">Reference</a>(<a href="sequence.html">Sequence</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">A Pointer to another Sequence entity as reference sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString" title="Sequence.referenceSeq.referenceSeqString : A string like &quot;ACGT&quot;.">referenceSeqString</a><a name="Sequence.referenceSeq.referenceSeqString"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">A string to represent reference sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.strand" title="Sequence.referenceSeq.strand : An absolute reference to a strand. The Watson strand is the strand whose 5'-end is on the short arm of the chromosome, and the Crick strand as the one whose 5'-end is on the long arm.">strand</a><a name="Sequence.referenceSeq.strand"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#code">code</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">watson | crick<br/><a href="valueset-strand-type.html" title="Type for strand.">strandType</a> (<a href="terminologies.html#required" title="To be conformant, the concept in this element SHALL be from the specified value set.">Required</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.windowStart" title="Sequence.referenceSeq.windowStart : Start position of the window on the reference sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive.">windowStart</a><a name="Sequence.referenceSeq.windowStart"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Start position of the window on the  reference sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.windowEnd" title="Sequence.referenceSeq.windowEnd : End position of the window on the reference sequence. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position.">windowEnd</a><a name="Sequence.referenceSeq.windowEnd"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">End position of the window on the reference sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck11.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant" title="Sequence.variant : The definition of variant here originates from Sequence ontology ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of)). This element can represent amino acid or nucleic sequence change(including insertion,deletion,SNP,etc.)  It can represent some complex mutation or segment variation with the assist of CIGAR string.">variant</a><a name="Sequence.variant"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Variant in sequence<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.start" title="Sequence.variant.start : Start position of the variant on the  reference sequence.If the coordinate system is either 0-based or 1-based, then start position is inclusive.">start</a><a name="Sequence.variant.start"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Start position of the variant on the  reference sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.end" title="Sequence.variant.end : End position of the variant on the reference sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position.">end</a><a name="Sequence.variant.end"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">End position of the variant on the reference sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.observedAllele" title="Sequence.variant.observedAllele : An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)).  Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed  sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end.">observedAllele</a><a name="Sequence.variant.observedAllele"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Allele that was observed</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.referenceAllele" title="Sequence.variant.referenceAllele : An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end.">referenceAllele</a><a name="Sequence.variant.referenceAllele"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Allele in the reference sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.cigar" title="Sequence.variant.cigar : Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm).">cigar</a><a name="Sequence.variant.cigar"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Extended CIGAR string for aligning the sequence with reference bases</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.variantPointer" title="Sequence.variant.variantPointer : A pointer to an Observation containing variant information.">variantPointer</a><a name="Sequence.variant.variantPointer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html#Reference">Reference</a>(<a href="observation.html">Observation</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Pointer to observed variant information</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.observedSeq" title="Sequence.observedSeq : Sequence that was observed. It is the result marked by referenceSeq along with variant records on referenceSeq. This shall starts from referenceSeq.windowStart and end by referenceSeq.windowEnd.">observedSeq</a><a name="Sequence.observedSeq"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Sequence that was observed</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck11.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality" title="Sequence.quality : An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686)).">quality</a><a name="Sequence.quality"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">An set of value as quality of sequence<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.type" title="Sequence.quality.type : INDEL / SNP / Undefined variant.">type</a><a name="Sequence.quality.type"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#code">code</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">indel | snp | unknown<br/><a href="valueset-quality-type.html" title="Type for quality report.">qualityType</a> (<a href="terminologies.html#required" title="To be conformant, the concept in this element SHALL be from the specified value set.">Required</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.standardSequence" title="Sequence.quality.standardSequence : Gold standard sequence used for comparing against.">standardSequence</a><a name="Sequence.quality.standardSequence"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Standard sequence for comparison<br/><a href="valueset-sequence-quality-standardSequence.html" title="Reference identifier of the sequence that used to mark the quality of tested samples.">F d a- standard sequence</a> (<a href="terminologies.html#example" title="Instances are not expected or even encouraged to draw from the specified value set.  The value set merely provides examples of the types of concepts intended to be included.">Example</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.start" title="Sequence.quality.start : Start position of the sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive.">start</a><a name="Sequence.quality.start"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Start position of the sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.end" title="Sequence.quality.end : End position of the sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position.">end</a><a name="Sequence.quality.end"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">End position of the sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.score" title="Sequence.quality.score : The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685)).">score</a><a name="Sequence.quality.score"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#Quantity">Quantity</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Quality score for the comparison</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.method" title="Sequence.quality.method : Which method is used to get sequence quality.">method</a><a name="Sequence.quality.method"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Method to get quality<br/><a href="valueset-sequence-quality-method.html" title="The method used to evaluate the numerical quality of the observed sequence.">F d a- method</a> (<a href="terminologies.html#example" title="Instances are not expected or even encouraged to draw from the specified value set.  The value set merely provides examples of the types of concepts intended to be included.">Example</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.truthTP" title="Sequence.quality.truthTP : True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event.">truthTP</a><a name="Sequence.quality.truthTP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">True positives from the perspective of the truth data</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.queryTP" title="Sequence.quality.queryTP : True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event.">queryTP</a><a name="Sequence.quality.queryTP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">True positives from the perspective of the query data</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.truthFN" title="Sequence.quality.truthFN : False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here.">truthFN</a><a name="Sequence.quality.truthFN"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">False negatives</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.queryFP" title="Sequence.quality.queryFP : False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here.">queryFP</a><a name="Sequence.quality.queryFP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">False positives</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.gtFP" title="Sequence.quality.gtFP : The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar).">gtFP</a><a name="Sequence.quality.gtFP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">False positives where the non-REF alleles in the Truth and Query Call Sets match</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.precision" title="Sequence.quality.precision : QUERY.TP / (QUERY.TP + QUERY.FP).">precision</a><a name="Sequence.quality.precision"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Precision of comparison</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.recall" title="Sequence.quality.recall : TRUTH.TP / (TRUTH.TP + TRUTH.FN).">recall</a><a name="Sequence.quality.recall"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Recall of comparison</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.fScore" title="Sequence.quality.fScore : Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall).">fScore</a><a name="Sequence.quality.fScore"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">F-score</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck101.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.roc" title="Sequence.quality.roc : Receiver Operator Characteristic (ROC) Curve  to give sensitivity/specificity tradeoff.">roc</a><a name="Sequence.quality.roc"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Receiver Operator Characteristic (ROC) Curve</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.roc.score" title="Sequence.quality.roc.score : Invidual data point representing the GQ (genotype quality) score threshold.">score</a><a name="Sequence.quality.roc.score"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Genotype quality score<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.roc.numTP" title="Sequence.quality.roc.numTP : The number of true positives if the GQ score threshold was set to &quot;score&quot; field value.">numTP</a><a name="Sequence.quality.roc.numTP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Roc score true positive numbers<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.roc.numFP" title="Sequence.quality.roc.numFP : The number of false positives if the GQ score threshold was set to &quot;score&quot; field value.">numFP</a><a name="Sequence.quality.roc.numFP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Roc score false positive numbers<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.roc.numFN" title="Sequence.quality.roc.numFN : The number of false negatives if the GQ score threshold was set to &quot;score&quot; field value.">numFN</a><a name="Sequence.quality.roc.numFN"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Roc score false negative numbers<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.roc.precision" title="Sequence.quality.roc.precision : Calculated precision if the GQ score threshold was set to &quot;score&quot; field value.">precision</a><a name="Sequence.quality.roc.precision"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Precision of the GQ score<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.roc.sensitivity" title="Sequence.quality.roc.sensitivity : Calculated sensitivity if the GQ score threshold was set to &quot;score&quot; field value.">sensitivity</a><a name="Sequence.quality.roc.sensitivity"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Sensitivity of the GQ score<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1000.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.roc.fMeasure" title="Sequence.quality.roc.fMeasure : Calculated fScore if the GQ score threshold was set to &quot;score&quot; field value.">fMeasure</a><a name="Sequence.quality.roc.fMeasure"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">FScore of the GQ score<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.readCoverage" title="Sequence.readCoverage : Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence.">readCoverage</a><a name="Sequence.readCoverage"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Average number of reads representing a given nucleotide in the reconstructed sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck11.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository" title="Sequence.repository : Configurations of the external repository. The repository shall store target's observedSeq or records related with target's observedSeq.">repository</a><a name="Sequence.repository"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">External repository which contains detailed report related with observedSeq in this resource<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.type" title="Sequence.repository.type : Click and see / RESTful API / Need login to see / RESTful API with authentication / Other ways to see resource.">type</a><a name="Sequence.repository.type"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#code">code</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">directlink | openapi | login | oauth | other<br/><a href="valueset-repository-type.html" title="Type for access of external URI.">repositoryType</a> (<a href="terminologies.html#required" title="To be conformant, the concept in this element SHALL be from the specified value set.">Required</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.url" title="Sequence.repository.url : URI of an external repository which contains further details about the genetics data.">url</a><a name="Sequence.repository.url"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#uri">uri</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">URI of the repository</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.name" title="Sequence.repository.name : URI of an external repository which contains further details about the genetics data.">name</a><a name="Sequence.repository.name"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Repository's name</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.datasetId" title="Sequence.repository.datasetId : Id of the variant in this external repository. The server will understand how to use this id to call for more info about datasets in external repository.">datasetId</a><a name="Sequence.repository.datasetId"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Id of the dataset that used to call for dataset in repository</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.variantsetId" title="Sequence.repository.variantsetId : Id of the variantset in this external repository. The server will understand how to use this id to call for more info about variantsets in external repository.">variantsetId</a><a name="Sequence.repository.variantsetId"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Id of the variantset that used to call for variantset in repository</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.readsetId" title="Sequence.repository.readsetId : Id of the read in this external repository.">readsetId</a><a name="Sequence.repository.readsetId"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Id of the read</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.pointer" title="Sequence.pointer : Pointer to next atomic sequence which at most contains one variant.">pointer</a><a name="Sequence.pointer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html#Reference">Reference</a>(<a href="sequence.html">Sequence</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Pointer to next atomic sequence<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck01.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant" title="Sequence.structureVariant : Information about chromosome structure variation.">structureVariant</a><a name="Sequence.structureVariant"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural variant<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.precision" title="Sequence.structureVariant.precision : Identify the exact boundaries of variant sequences. Each type of structure variant requires the DNA duplex to be broken and rejoined, and this creates a new sequence of bases at the rejoined sites, known as breakpoints or boundaries.">precision</a><a name="Sequence.structureVariant.precision"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Precision of boundaries</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio" title="Sequence.structureVariant.reportedaCGHRatio : Structural Variant reported aCGH ratio.">reportedaCGHRatio</a><a name="Sequence.structureVariant.reportedaCGHRatio"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant reported aCGH ratio</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.length" title="Sequence.structureVariant.length : Length of the variant choromosome.">length</a><a name="Sequence.structureVariant.length"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Length</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck011.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.outer" title="Sequence.structureVariant.outer : Structural variant outer.">outer</a><a name="Sequence.structureVariant.outer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural variant outer</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck0110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.outer.start" title="Sequence.structureVariant.outer.start : Structural Variant Outer Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive.">start</a><a name="Sequence.structureVariant.outer.start"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Outer Start</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck0100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.outer.end" title="Sequence.structureVariant.outer.end : Structural Variant Outer End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position.">end</a><a name="Sequence.structureVariant.outer.end"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Outer End</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck001.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.inner" title="Sequence.structureVariant.inner : Structural variant inner.">inner</a><a name="Sequence.structureVariant.inner"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural variant inner</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck0010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.inner.start" title="Sequence.structureVariant.inner.start : Structural Variant Inner Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive.">start</a><a name="Sequence.structureVariant.inner.start"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Inner Start</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck0000.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.inner.end" title="Sequence.structureVariant.inner.end : Structural Variant Inner End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position.">end</a><a name="Sequence.structureVariant.inner.end"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Inner End</td></tr>
<tr><td colspan="5" class="hierarchy"><br/><a href="formats.html#table" title="Legend for this format"><img src="help16.png" alt="doco" style="background-color: inherit"/> Documentation for this format</a></td></tr></table>
 </div>
</div>

  </div>
  <div id="tabs-uml">

<div id="uml">
<p><b>UML Diagram</b> (<a href="formats.html#uml">Legend</a>)</p>
 <div id="uml-inner">
 <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="1767.2" id="n206" version="1.1" height="652.0">  <defs>
    <filter x="0" width="200%" y="0" id="shadowA" height="200%">
      <feOffset result="offOut" dx="3" in="SourceGraphic" dy="3"></feOffset>
      <feColorMatrix result="matrixOut" in="offOut" values="0.2 0 0 0 0 0 0.2 0 0 0 0 0 0.2 0 0 0 0 0 1 0" type="matrix"></feColorMatrix>
      <feGaussianBlur result="blurOut" stdDeviation="2" in="matrixOut"></feGaussianBlur>
      <feBlend mode="normal" in="SourceGraphic" in2="blurOut"></feBlend>
    </filter>
  </defs><g id="n207"><rect filter="url(#shadowA)" rx="4" ry="4" x="410.40000000000003" width="198.00000000000003" y="242.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence" height="190.0"></rect><line y1="270.0" x1="410.40000000000003" y2="270.0" x2="608.4000000000001" style="stroke:dimgrey;stroke-width:1" id="n208"></line><text x="509.40000000000003" y="262.0" id="n209" fill="black" class="diagram-class-title  diagram-class-resource">Sequence<tspan class="diagram-class-title-link"> (<a id="n210" class="diagram-class-reference" xlink:href="domainresource.html">DomainResource</a>)</tspan></text><text x="416.40000000000003" y="284.0" id="n211" fill="black" class="diagram-class-detail"><a id="n212" xlink:href="sequence-definitions.html#Sequence.identifier"><title>A unique identifier for this particular sequence instance. This is a FHIR-defined id</title>identifier</a> : <a id="n213" xlink:href="datatypes.html#Identifier">Identifier</a> [0..*]</text><text x="416.40000000000003" y="298.0" id="n217" fill="black" class="diagram-class-detail"><a id="n218" xlink:href="sequence-definitions.html#Sequence.type"><title>Amino Acid Sequence/ DNA Sequence / RNA Sequence</title>type</a> : <a id="n219" xlink:href="datatypes.html#code">code</a> [0..1] « <a id="n220" xlink:href="valueset-sequence-type.html"><title id="n221">Type if a sequence -- DNA, RNA, or amino acid sequence. (Strength=Example)</title>sequenceType</a>?? »</text><text x="416.40000000000003" y="312.0" id="n227" fill="black" class="diagram-class-detail"><a id="n228" xlink:href="sequence-definitions.html#Sequence.coordinateSystem"><title>Whether the sequence is numbered starting at 0 (0-based numbering or coordinates, inclusive start, exclusive end) or starting at 1 (1-based numbering, inclusive start and inclusive end)</title>coordinateSystem</a> : <a id="n229" xlink:href="datatypes.html#integer">integer</a> [1..1]</text><text x="416.40000000000003" y="326.0" id="n233" fill="black" class="diagram-class-detail"><a id="n234" xlink:href="sequence-definitions.html#Sequence.patient"><title>The patient whose sequencing results are described by this resource</title>patient</a> : <a id="n235" xlink:href="references.html#Reference">Reference</a> [0..1] « <a id="n236" xlink:href="patient.html#Patient">Patient</a> »</text><text x="416.40000000000003" y="340.0" id="n241" fill="black" class="diagram-class-detail"><a id="n242" xlink:href="sequence-definitions.html#Sequence.specimen"><title>Specimen used for sequencing</title>specimen</a> : <a id="n243" xlink:href="references.html#Reference">Reference</a> [0..1] « <a id="n244" xlink:href="specimen.html#Specimen">Specimen</a> »</text><text x="416.40000000000003" y="354.0" id="n249" fill="black" class="diagram-class-detail"><a id="n250" xlink:href="sequence-definitions.html#Sequence.device"><title>The method for sequencing, for example, chip information</title>device</a> : <a id="n251" xlink:href="references.html#Reference">Reference</a> [0..1] « <a id="n252" xlink:href="device.html#Device">Device</a> »</text><text x="416.40000000000003" y="368.0" id="n257" fill="black" class="diagram-class-detail"><a id="n258" xlink:href="sequence-definitions.html#Sequence.performer"><title>The organization or lab that should be responsible for this result</title>performer</a> : <a id="n259" xlink:href="references.html#Reference">Reference</a> [0..1] « <a id="n260" xlink:href="organization.html#Organization">Organization</a> »</text><text x="416.40000000000003" y="382.0" id="n265" fill="black" class="diagram-class-detail"><a id="n266" xlink:href="sequence-definitions.html#Sequence.quantity"><title>The number of copies of the seqeunce of interest. (RNASeq)</title>quantity</a> : <a id="n267" xlink:href="datatypes.html#Quantity">Quantity</a> [0..1]</text><text x="416.40000000000003" y="396.0" id="n271" fill="black" class="diagram-class-detail"><a id="n272" xlink:href="sequence-definitions.html#Sequence.observedSeq"><title>Sequence that was observed. It is the result marked by referenceSeq along with variant records on referenceSeq. This shall starts from referenceSeq.windowStart and end by referenceSeq.windowEnd</title>observedSeq</a> : <a id="n273" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="416.40000000000003" y="410.0" id="n277" fill="black" class="diagram-class-detail"><a id="n278" xlink:href="sequence-definitions.html#Sequence.readCoverage"><title>Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence</title>readCoverage</a> : <a id="n279" xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="416.40000000000003" y="424.0" id="n283" fill="black" class="diagram-class-detail"><a id="n284" xlink:href="sequence-definitions.html#Sequence.pointer"><title>Pointer to next atomic sequence which at most contains one variant</title>pointer</a> : <a id="n285" xlink:href="references.html#Reference">Reference</a> [0..*] « <a id="n286" xlink:href="sequence.html#Sequence">Sequence</a> »</text></g><g id="n291"><rect filter="url(#shadowA)" rx="4" ry="4" x="386.20000000000005" width="246.40000000000003" y="0.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence.referenceSeq" height="162.0"></rect><line y1="28.0" x1="386.20000000000005" y2="28.0" x2="632.6000000000001" style="stroke:dimgrey;stroke-width:1" id="n292"></line><text x="509.4000000000001" y="20.0" id="n293" fill="black" class="diagram-class-title">ReferenceSeq</text><text x="392.20000000000005" y="42.0" id="n294" fill="black" class="diagram-class-detail"><a id="n295" xlink:href="sequence-definitions.html#Sequence.referenceSeq.chromosome"><title>Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340))</title>chromosome</a> : <a id="n296" xlink:href="datatypes.html#CodeableConcept">CodeableConcept</a> [0..1] « <a id="n297" xlink:href="valueset-chromosome-human.html"><title id="n298">Chromosome number for human. (Strength=Example)</title>chromosome-human</a>?? »</text><text x="392.20000000000005" y="56.0" id="n304" fill="black" class="diagram-class-detail"><a id="n305" xlink:href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild"><title>The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used</title>genomeBuild</a> : <a id="n306" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="392.20000000000005" y="70.0" id="n310" fill="black" class="diagram-class-detail"><a id="n311" xlink:href="sequence-definitions.html#Sequence.referenceSeq.orientation"><title>A relative reference to a DNA strand based on gene orientation. The strand that contains the open reading frame of the gene is the "sense" strand, and the opposite complementary strand is the "antisense" strand</title>orientation</a> : <a id="n312" xlink:href="datatypes.html#code">code</a> [0..1] « <a id="n313" xlink:href="valueset-orientation-type.html"><title id="n314">Type for orientation. (Strength=Required)</title>orientationType</a>! »</text><text x="392.20000000000005" y="84.0" id="n320" fill="black" class="diagram-class-detail"><a id="n321" xlink:href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId"><title>Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences</title>referenceSeqId</a> : <a id="n322" xlink:href="datatypes.html#CodeableConcept">CodeableConcept</a> [0..1] « <a id="n323" xlink:href="valueset-sequence-referenceSeq.html"><title id="n324">Reference identifier. (Strength=Example)</title>ENSEMBL</a>?? »</text><text x="392.20000000000005" y="98.0" id="n330" fill="black" class="diagram-class-detail"><a id="n331" xlink:href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer"><title>A Pointer to another Sequence entity as reference sequence</title>referenceSeqPointer</a> : <a id="n332" xlink:href="references.html#Reference">Reference</a> [0..1] « <a id="n333" xlink:href="sequence.html#Sequence">Sequence</a> »</text><text x="392.20000000000005" y="112.0" id="n338" fill="black" class="diagram-class-detail"><a id="n339" xlink:href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString"><title>A string like "ACGT"</title>referenceSeqString</a> : <a id="n340" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="392.20000000000005" y="126.0" id="n344" fill="black" class="diagram-class-detail"><a id="n345" xlink:href="sequence-definitions.html#Sequence.referenceSeq.strand"><title>An absolute reference to a strand. The Watson strand is the strand whose 5'-end is on the short arm of the chromosome, and the Crick strand as the one whose 5'-end is on the long arm</title>strand</a> : <a id="n346" xlink:href="datatypes.html#code">code</a> [0..1] « <a id="n347" xlink:href="valueset-strand-type.html"><title id="n348">Type for strand. (Strength=Required)</title>strandType</a>! »</text><text x="392.20000000000005" y="140.0" id="n354" fill="black" class="diagram-class-detail"><a id="n355" xlink:href="sequence-definitions.html#Sequence.referenceSeq.windowStart"><title>Start position of the window on the reference sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive</title>windowStart</a> : <a id="n356" xlink:href="datatypes.html#integer">integer</a> [1..1]</text><text x="392.20000000000005" y="154.0" id="n360" fill="black" class="diagram-class-detail"><a id="n361" xlink:href="sequence-definitions.html#Sequence.referenceSeq.windowEnd"><title>End position of the window on the reference sequence. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position</title>windowEnd</a> : <a id="n362" xlink:href="datatypes.html#integer">integer</a> [1..1]</text></g><g id="n366"><rect filter="url(#shadowA)" rx="4" ry="4" x="650.4000000000001" width="215.60000000000002" y="242.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence.variant" height="120.0"></rect><line y1="270.0" x1="650.4000000000001" y2="270.0" x2="866.0000000000001" style="stroke:dimgrey;stroke-width:1" id="n367"></line><text x="758.2" y="262.0" id="n368" fill="black" class="diagram-class-title">Variant</text><text x="656.4000000000001" y="284.0" id="n369" fill="black" class="diagram-class-detail"><a id="n370" xlink:href="sequence-definitions.html#Sequence.variant.start"><title>Start position of the variant on the  reference sequence.If the coordinate system is either 0-based or 1-based, then start position is inclusive</title>start</a> : <a id="n371" xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="656.4000000000001" y="298.0" id="n375" fill="black" class="diagram-class-detail"><a id="n376" xlink:href="sequence-definitions.html#Sequence.variant.end"><title>End position of the variant on the reference sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position</title>end</a> : <a id="n377" xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="656.4000000000001" y="312.0" id="n381" fill="black" class="diagram-class-detail"><a id="n382" xlink:href="sequence-definitions.html#Sequence.variant.observedAllele"><title>An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)).  Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed  sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end</title>observedAllele</a> : <a id="n383" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="656.4000000000001" y="326.0" id="n387" fill="black" class="diagram-class-detail"><a id="n388" xlink:href="sequence-definitions.html#Sequence.variant.referenceAllele"><title>An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end</title>referenceAllele</a> : <a id="n389" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="656.4000000000001" y="340.0" id="n393" fill="black" class="diagram-class-detail"><a id="n394" xlink:href="sequence-definitions.html#Sequence.variant.cigar"><title>Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm)</title>cigar</a> : <a id="n395" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="656.4000000000001" y="354.0" id="n399" fill="black" class="diagram-class-detail"><a id="n400" xlink:href="sequence-definitions.html#Sequence.variant.variantPointer"><title>A pointer to an Observation containing variant information</title>variantPointer</a> : <a id="n401" xlink:href="references.html#Reference">Reference</a> [0..1] « <a id="n402" xlink:href="observation.html#Observation">Observation</a> »</text></g><g id="n407"><rect filter="url(#shadowA)" rx="4" ry="4" x="0.0" width="290.40000000000003" y="221.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence.quality" height="232.0"></rect><line y1="249.0" x1="0.0" y2="249.0" x2="290.40000000000003" style="stroke:dimgrey;stroke-width:1" id="n408"></line><text x="145.20000000000002" y="241.0" id="n409" fill="black" class="diagram-class-title">Quality</text><text x="6.0" y="263.0" id="n410" fill="black" class="diagram-class-detail"><a id="n411" xlink:href="sequence-definitions.html#Sequence.quality.type"><title>INDEL / SNP / Undefined variant</title>type</a> : <a id="n412" xlink:href="datatypes.html#code">code</a> [1..1] « <a id="n413" xlink:href="valueset-quality-type.html"><title id="n414">Type for quality report. (Strength=Required)</title>qualityType</a>! »</text><text x="6.0" y="277.0" id="n420" fill="black" class="diagram-class-detail"><a id="n421" xlink:href="sequence-definitions.html#Sequence.quality.standardSequence"><title>Gold standard sequence used for comparing against</title>standardSequence</a> : <a id="n422" xlink:href="datatypes.html#CodeableConcept">CodeableConcept</a> [0..1] « <a id="n423" xlink:href="valueset-sequence-quality-standardSequence.html"><title id="n424">Reference identifier of the sequence that used to mark the quality of tested samples. (Strength=Example)</title>FDA-StandardSequence</a>?? »</text><text x="6.0" y="291.0" id="n430" fill="black" class="diagram-class-detail"><a id="n431" xlink:href="sequence-definitions.html#Sequence.quality.start"><title>Start position of the sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive</title>start</a> : <a id="n432" xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="6.0" y="305.0" id="n436" fill="black" class="diagram-class-detail"><a id="n437" xlink:href="sequence-definitions.html#Sequence.quality.end"><title>End position of the sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position</title>end</a> : <a id="n438" xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="6.0" y="319.0" id="n442" fill="black" class="diagram-class-detail"><a id="n443" xlink:href="sequence-definitions.html#Sequence.quality.score"><title>The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685))</title>score</a> : <a id="n444" xlink:href="datatypes.html#Quantity">Quantity</a> [0..1]</text><text x="6.0" y="333.0" id="n448" fill="black" class="diagram-class-detail"><a id="n449" xlink:href="sequence-definitions.html#Sequence.quality.method"><title>Which method is used to get sequence quality</title>method</a> : <a id="n450" xlink:href="datatypes.html#CodeableConcept">CodeableConcept</a> [0..1] « <a id="n451" xlink:href="valueset-sequence-quality-method.html"><title id="n452">The method used to evaluate the numerical quality of the observed sequence. (Strength=Example)</title>FDA-Method</a>?? »</text><text x="6.0" y="347.0" id="n458" fill="black" class="diagram-class-detail"><a id="n459" xlink:href="sequence-definitions.html#Sequence.quality.truthTP"><title>True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event</title>truthTP</a> : <a id="n460" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="361.0" id="n464" fill="black" class="diagram-class-detail"><a id="n465" xlink:href="sequence-definitions.html#Sequence.quality.queryTP"><title>True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event</title>queryTP</a> : <a id="n466" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="375.0" id="n470" fill="black" class="diagram-class-detail"><a id="n471" xlink:href="sequence-definitions.html#Sequence.quality.truthFN"><title>False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here</title>truthFN</a> : <a id="n472" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="389.0" id="n476" fill="black" class="diagram-class-detail"><a id="n477" xlink:href="sequence-definitions.html#Sequence.quality.queryFP"><title>False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here</title>queryFP</a> : <a id="n478" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="403.0" id="n482" fill="black" class="diagram-class-detail"><a id="n483" xlink:href="sequence-definitions.html#Sequence.quality.gtFP"><title>The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar)</title>gtFP</a> : <a id="n484" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="417.0" id="n488" fill="black" class="diagram-class-detail"><a id="n489" xlink:href="sequence-definitions.html#Sequence.quality.precision"><title>QUERY.TP / (QUERY.TP + QUERY.FP)</title>precision</a> : <a id="n490" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="431.0" id="n494" fill="black" class="diagram-class-detail"><a id="n495" xlink:href="sequence-definitions.html#Sequence.quality.recall"><title>TRUTH.TP / (TRUTH.TP + TRUTH.FN)</title>recall</a> : <a id="n496" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="445.0" id="n500" fill="black" class="diagram-class-detail"><a id="n501" xlink:href="sequence-definitions.html#Sequence.quality.fScore"><title>Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall)</title>fScore</a> : <a id="n502" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text></g><g id="n506"><rect filter="url(#shadowA)" rx="4" ry="4" x="890.4000000000001" width="123.20000000000002" y="242.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence.quality.roc" height="134.0"></rect><line y1="270.0" x1="890.4000000000001" y2="270.0" x2="1013.6000000000001" style="stroke:dimgrey;stroke-width:1" id="n507"></line><text x="952.0000000000001" y="262.0" id="n508" fill="black" class="diagram-class-title">Roc</text><text x="896.4000000000001" y="284.0" id="n509" fill="black" class="diagram-class-detail"><a id="n510" xlink:href="sequence-definitions.html#Sequence.quality.roc.score"><title>Invidual data point representing the GQ (genotype quality) score threshold</title>score</a> : <a id="n511" xlink:href="datatypes.html#integer">integer</a> [0..*]</text><text x="896.4000000000001" y="298.0" id="n515" fill="black" class="diagram-class-detail"><a id="n516" xlink:href="sequence-definitions.html#Sequence.quality.roc.numTP"><title>The number of true positives if the GQ score threshold was set to "score" field value</title>numTP</a> : <a id="n517" xlink:href="datatypes.html#integer">integer</a> [0..*]</text><text x="896.4000000000001" y="312.0" id="n521" fill="black" class="diagram-class-detail"><a id="n522" xlink:href="sequence-definitions.html#Sequence.quality.roc.numFP"><title>The number of false positives if the GQ score threshold was set to "score" field value</title>numFP</a> : <a id="n523" xlink:href="datatypes.html#integer">integer</a> [0..*]</text><text x="896.4000000000001" y="326.0" id="n527" fill="black" class="diagram-class-detail"><a id="n528" xlink:href="sequence-definitions.html#Sequence.quality.roc.numFN"><title>The number of false negatives if the GQ score threshold was set to "score" field value</title>numFN</a> : <a id="n529" xlink:href="datatypes.html#integer">integer</a> [0..*]</text><text x="896.4000000000001" y="340.0" id="n533" fill="black" class="diagram-class-detail"><a id="n534" xlink:href="sequence-definitions.html#Sequence.quality.roc.precision"><title>Calculated precision if the GQ score threshold was set to "score" field value</title>precision</a> : <a id="n535" xlink:href="datatypes.html#decimal">decimal</a> [0..*]</text><text x="896.4000000000001" y="354.0" id="n539" fill="black" class="diagram-class-detail"><a id="n540" xlink:href="sequence-definitions.html#Sequence.quality.roc.sensitivity"><title>Calculated sensitivity if the GQ score threshold was set to "score" field value</title>sensitivity</a> : <a id="n541" xlink:href="datatypes.html#decimal">decimal</a> [0..*]</text><text x="896.4000000000001" y="368.0" id="n545" fill="black" class="diagram-class-detail"><a id="n546" xlink:href="sequence-definitions.html#Sequence.quality.roc.fMeasure"><title>Calculated fScore if the GQ score threshold was set to "score" field value</title>fMeasure</a> : <a id="n547" xlink:href="datatypes.html#decimal">decimal</a> [0..*]</text></g><g id="n551"><rect filter="url(#shadowA)" rx="4" ry="4" x="428.00000000000006" width="162.8" y="512.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence.repository" height="120.0"></rect><line y1="540.0" x1="428.00000000000006" y2="540.0" x2="590.8000000000001" style="stroke:dimgrey;stroke-width:1" id="n552"></line><text x="509.4000000000001" y="532.0" id="n553" fill="black" class="diagram-class-title">Repository</text><text x="434.00000000000006" y="554.0" id="n554" fill="black" class="diagram-class-detail"><a id="n555" xlink:href="sequence-definitions.html#Sequence.repository.type"><title>Click and see / RESTful API / Need login to see / RESTful API with authentication / Other ways to see resource</title>type</a> : <a id="n556" xlink:href="datatypes.html#code">code</a> [1..1] « <a id="n557" xlink:href="valueset-repository-type.html"><title id="n558">Type for access of external URI. (Strength=Required)</title>repositoryType</a>! »</text><text x="434.00000000000006" y="568.0" id="n564" fill="black" class="diagram-class-detail"><a id="n565" xlink:href="sequence-definitions.html#Sequence.repository.url"><title>URI of an external repository which contains further details about the genetics data</title>url</a> : <a id="n566" xlink:href="datatypes.html#uri">uri</a> [0..1]</text><text x="434.00000000000006" y="582.0" id="n570" fill="black" class="diagram-class-detail"><a id="n571" xlink:href="sequence-definitions.html#Sequence.repository.name"><title>URI of an external repository which contains further details about the genetics data</title>name</a> : <a id="n572" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="434.00000000000006" y="596.0" id="n576" fill="black" class="diagram-class-detail"><a id="n577" xlink:href="sequence-definitions.html#Sequence.repository.datasetId"><title>Id of the variant in this external repository. The server will understand how to use this id to call for more info about datasets in external repository</title>datasetId</a> : <a id="n578" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="434.00000000000006" y="610.0" id="n582" fill="black" class="diagram-class-detail"><a id="n583" xlink:href="sequence-definitions.html#Sequence.repository.variantsetId"><title>Id of the variantset in this external repository. The server will understand how to use this id to call for more info about variantsets in external repository</title>variantsetId</a> : <a id="n584" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="434.00000000000006" y="624.0" id="n588" fill="black" class="diagram-class-detail"><a id="n589" xlink:href="sequence-definitions.html#Sequence.repository.readsetId"><title>Id of the read in this external repository</title>readsetId</a> : <a id="n590" xlink:href="datatypes.html#string">string</a> [0..1]</text></g><g id="n594"><rect filter="url(#shadowA)" rx="4" ry="4" x="1050.4" width="149.60000000000002" y="242.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence.structureVariant" height="78.0"></rect><line y1="270.0" x1="1050.4" y2="270.0" x2="1200.0" style="stroke:dimgrey;stroke-width:1" id="n595"></line><text x="1125.2" y="262.0" id="n596" fill="black" class="diagram-class-title">StructureVariant</text><text x="1056.4" y="284.0" id="n597" fill="black" class="diagram-class-detail"><a id="n598" xlink:href="sequence-definitions.html#Sequence.structureVariant.precision"><title>Identify the exact boundaries of variant sequences. Each type of structure variant requires the DNA duplex to be broken and rejoined, and this creates a new sequence of bases at the rejoined sites, known as breakpoints or boundaries</title>precision</a> : <a id="n599" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="1056.4" y="298.0" id="n603" fill="black" class="diagram-class-detail"><a id="n604" xlink:href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio"><title>Structural Variant reported aCGH ratio</title>reportedaCGHRatio</a> : <a id="n605" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="1056.4" y="312.0" id="n609" fill="black" class="diagram-class-detail"><a id="n610" xlink:href="sequence-definitions.html#Sequence.structureVariant.length"><title>Length of the variant choromosome</title>length</a> : <a id="n611" xlink:href="datatypes.html#integer">integer</a> [0..1]</text></g><g id="n615"><rect filter="url(#shadowA)" rx="4" ry="4" x="1210.4" width="96.80000000000001" y="242.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence.structureVariant.outer" height="64.0"></rect><line y1="270.0" x1="1210.4" y2="270.0" x2="1307.2" style="stroke:dimgrey;stroke-width:1" id="n616"></line><text x="1258.8000000000002" y="262.0" id="n617" fill="black" class="diagram-class-title">Outer</text><text x="1216.4" y="284.0" id="n618" fill="black" class="diagram-class-detail"><a id="n619" xlink:href="sequence-definitions.html#Sequence.structureVariant.outer.start"><title>Structural Variant Outer Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive</title>start</a> : <a id="n620" xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="1216.4" y="298.0" id="n624" fill="black" class="diagram-class-detail"><a id="n625" xlink:href="sequence-definitions.html#Sequence.structureVariant.outer.end"><title>Structural Variant Outer End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position</title>end</a> : <a id="n626" xlink:href="datatypes.html#integer">integer</a> [0..1]</text></g><g id="n630"><rect filter="url(#shadowA)" rx="4" ry="4" x="1370.4" width="96.80000000000001" y="242.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence.structureVariant.inner" height="64.0"></rect><line y1="270.0" x1="1370.4" y2="270.0" x2="1467.2" style="stroke:dimgrey;stroke-width:1" id="n631"></line><text x="1418.8000000000002" y="262.0" id="n632" fill="black" class="diagram-class-title">Inner</text><text x="1376.4" y="284.0" id="n633" fill="black" class="diagram-class-detail"><a id="n634" xlink:href="sequence-definitions.html#Sequence.structureVariant.inner.start"><title>Structural Variant Inner Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive</title>start</a> : <a id="n635" xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="1376.4" y="298.0" id="n639" fill="black" class="diagram-class-detail"><a id="n640" xlink:href="sequence-definitions.html#Sequence.structureVariant.inner.end"><title>Structural Variant Inner End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position</title>end</a> : <a id="n641" xlink:href="datatypes.html#integer">integer</a> [0..1]</text></g><line y1="256.0" x1="509.40000000000003" y2="0.0" x2="509.4000000000001" style="stroke:navy;stroke-width:1" id="n645"></line><polygon transform="rotate(-89.99999999999999 509.40000000000003 256.0)" style="fill:navy;stroke:navy;stroke-width:1" id="n646" points="509.40000000000003,256.0 515.4000000000001,260.0 521.4000000000001,256.0 515.4000000000001,252.0 509.40000000000003,256.0"></polygon><rect x="483.0" width="52.0" y="121.0" style="fill:white;stroke:black;stroke-width:0" id="n647" height="18.0"></rect><text x="509.0" y="131.0" id="n648" fill="black" class="diagram-class-linkage"><a id="n649" xlink:href="sequence-definitions.html#Sequence.referenceSeq"><title>A sequence that is used as a reference to describe variants that are present in a sequence analyzed</title>referenceSeq</a></text><text x="509.4000000000001" y="-4.0" id="n650" fill="black" class="diagram-class-linkage">[0..1]</text><line y1="323.073151125402" x1="608.4000000000001" y2="317.16479099678463" x2="650.4000000000001" style="stroke:navy;stroke-width:1" id="n651"></line><polygon transform="rotate(-8.00755162235136 608.4000000000001 323.073151125402)" style="fill:navy;stroke:navy;stroke-width:1" id="n652" points="608.4000000000001,323.073151125402 614.4000000000001,327.073151125402 620.4000000000001,323.073151125402 614.4000000000001,319.073151125402 608.4000000000001,323.073151125402"></polygon><rect x="614.0" width="30.0" y="313.0" style="fill:white;stroke:black;stroke-width:0" id="n653" height="18.0"></rect><text x="629.0" y="323.0" id="n654" fill="black" class="diagram-class-linkage"><a id="n655" xlink:href="sequence-definitions.html#Sequence.variant"><title>The definition of variant here originates from Sequence ontology ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of)). This element can represent amino acid or nucleic sequence change(including insertion,deletion,SNP,etc.)  It can represent some complex mutation or segment variation with the assist of CIGAR string</title>variant</a></text><text x="630.4000000000001" y="313.16479099678463" id="n656" fill="black" class="diagram-class-linkage">[0..*]</text><line y1="331.9608329201786" x1="290.40000000000003" y2="311.1378284581062" x2="890.4000000000001" style="stroke:navy;stroke-width:1" id="n657"></line><polygon transform="rotate(-1.9876527090606158 290.40000000000003 331.9608329201786)" style="fill:navy;stroke:navy;stroke-width:1" id="n658" points="290.40000000000003,331.9608329201786 296.40000000000003,335.9608329201786 302.40000000000003,331.9608329201786 296.40000000000003,327.9608329201786 290.40000000000003,331.9608329201786"></polygon><rect x="583.5" width="13.0" y="314.0" style="fill:white;stroke:black;stroke-width:0" id="n659" height="18.0"></rect><text x="590.0" y="324.0" id="n660" fill="black" class="diagram-class-linkage"><a id="n661" xlink:href="sequence-definitions.html#Sequence.quality.roc"><title>Receiver Operator Characteristic (ROC) Curve  to give sensitivity/specificity tradeoff</title>roc</a></text><text x="870.4000000000001" y="307.1378284581062" id="n662" fill="black" class="diagram-class-linkage">[0..1]</text><line y1="337.0" x1="410.40000000000003" y2="337.0" x2="290.40000000000003" style="stroke:navy;stroke-width:1" id="n663"></line><polygon transform="rotate(180.0 410.40000000000003 337.0)" style="fill:navy;stroke:navy;stroke-width:1" id="n664" points="410.40000000000003,337.0 416.40000000000003,341.0 422.40000000000003,337.0 416.40000000000003,333.0 410.40000000000003,337.0"></polygon><rect x="335.0" width="30.0" y="330.0" style="fill:white;stroke:black;stroke-width:0" id="n665" height="18.0"></rect><text x="350.0" y="340.0" id="n666" fill="black" class="diagram-class-linkage"><a id="n667" xlink:href="sequence-definitions.html#Sequence.quality"><title>An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686))</title>quality</a></text><text x="305.40000000000003" y="333.0" id="n668" fill="black" class="diagram-class-linkage">[0..*]</text><line y1="277.0808383233534" x1="1200.0" y2="276.53592814371274" x2="1210.4" style="stroke:navy;stroke-width:1" id="n669"></line><polygon transform="rotate(-2.9992817812825256 1200.0 277.0808383233534)" style="fill:navy;stroke:navy;stroke-width:1" id="n670" points="1200.0,277.0808383233534 1206.0,281.0808383233534 1212.0,277.0808383233534 1206.0,273.0808383233534 1200.0,277.0808383233534"></polygon><rect x="1194.0" width="22.0" y="269.0" style="fill:white;stroke:black;stroke-width:0" id="n671" height="18.0"></rect><text x="1205.0" y="279.0" id="n672" fill="black" class="diagram-class-linkage"><a id="n673" xlink:href="sequence-definitions.html#Sequence.structureVariant.outer"><title>Structural variant outer</title>outer</a></text><text x="1190.4" y="272.53592814371274" id="n674" fill="black" class="diagram-class-linkage">[0..1]</text><line y1="279.21662125340606" x1="1200.0" y2="275.15395095367853" x2="1370.4" style="stroke:navy;stroke-width:1" id="n675"></line><polygon transform="rotate(-1.3657850401371643 1200.0 279.21662125340606)" style="fill:navy;stroke:navy;stroke-width:1" id="n676" points="1200.0,279.21662125340606 1206.0,283.21662125340606 1212.0,279.21662125340606 1206.0,275.21662125340606 1200.0,279.21662125340606"></polygon><rect x="1274.0" width="22.0" y="270.0" style="fill:white;stroke:black;stroke-width:0" id="n677" height="18.0"></rect><text x="1285.0" y="280.0" id="n678" fill="black" class="diagram-class-linkage"><a id="n679" xlink:href="sequence-definitions.html#Sequence.structureVariant.inner"><title>Structural variant inner</title>inner</a></text><text x="1350.4" y="271.15395095367853" id="n680" fill="black" class="diagram-class-linkage">[0..1]</text><line y1="327.9970769730432" x1="608.4000000000001" y2="287.8022085092563" x2="1050.4" style="stroke:navy;stroke-width:1" id="n681"></line><polygon transform="rotate(-5.196106724932867 608.4000000000001 327.9970769730432)" style="fill:navy;stroke:navy;stroke-width:1" id="n682" points="608.4000000000001,327.9970769730432 614.4000000000001,331.9970769730432 620.4000000000001,327.9970769730432 614.4000000000001,323.9970769730432 608.4000000000001,327.9970769730432"></polygon><rect x="794.0" width="70.0" y="300.0" style="fill:white;stroke:black;stroke-width:0" id="n683" height="18.0"></rect><text x="829.0" y="310.0" id="n684" fill="black" class="diagram-class-linkage"><a id="n685" xlink:href="sequence-definitions.html#Sequence.structureVariant"><title>Information about chromosome structure variation</title>structureVariant</a></text><text x="1030.4" y="283.8022085092563" id="n686" fill="black" class="diagram-class-linkage">[0..*]</text></svg>
 </div>
</div>

  </div>
  <div id="tabs-xml">

<div id="xml">
<p><b>XML Template</b></p>
 <div id="xml-inner">
  <pre class="spec">
&lt;<a href="sequence-definitions.html#Sequence" title="Raw data describing a biological sequence." class="dict"><b>Sequence</b></a> xmlns="http://hl7.org/fhir"&gt; <span style="float: right"><a href="xml.html" title="Documentation for this format"><img src="help.png" alt="doco"/></a></span>
 &lt;!-- from <a href="resource.html">Resource</a>: <a href="resource.html#id">id</a>, <a href="resource.html#meta">meta</a>, <a href="resource.html#implicitRules">implicitRules</a>, and <a href="resource.html#language">language</a> --&gt;
 &lt;!-- from <a href="domainresource.html">DomainResource</a>: <a href="narrative.html#Narrative">text</a>, <a href="references.html#contained">contained</a>, <a href="extensibility.html">extension</a>, and <a href="extensibility.html#modifierExtension">modifierExtension</a> --&gt;
 &lt;<a href="sequence-definitions.html#Sequence.identifier" title="A unique identifier for this particular sequence instance. This is a FHIR-defined id." class="dict"><b>identifier</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: darkgreen"><a href="datatypes.html#Identifier">Identifier</a></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Unique ID for this particular sequence. This is a FHIR-defined id</a></span><span style="color: Gray"> --&gt;</span>&lt;/identifier&gt;
 &lt;<a href="sequence-definitions.html#Sequence.type" title="Amino Acid Sequence/ DNA Sequence / RNA Sequence." class="dict"><b>type</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="valueset-sequence-type.html">aa | dna | rna</a></span><span style="color: Gray"> --&gt;</span>
 &lt;<a href="sequence-definitions.html#Sequence.coordinateSystem" title="Whether the sequence is numbered starting at 0 (0-based numbering or coordinates, inclusive start, exclusive end) or starting at 1 (1-based numbering, inclusive start and inclusive end)." class="dict"><b>coordinateSystem</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Base number of coordinate system (0 for 0-based numbering or coordinates, inclusive start, exclusive end, 1 for 1-based numbering, inclusive start, inclusive end)</a></span><span style="color: Gray"> --&gt;</span>
 &lt;<a href="sequence-definitions.html#Sequence.patient" title="The patient whose sequencing results are described by this resource." class="dict"><b>patient</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="patient.html#Patient">Patient</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Who and/or what this is about</a></span><span style="color: Gray"> --&gt;</span>&lt;/patient&gt;
 &lt;<a href="sequence-definitions.html#Sequence.specimen" title="Specimen used for sequencing." class="dict"><b>specimen</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="specimen.html#Specimen">Specimen</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Specimen used for sequencing</a></span><span style="color: Gray"> --&gt;</span>&lt;/specimen&gt;
 &lt;<a href="sequence-definitions.html#Sequence.device" title="The method for sequencing, for example, chip information." class="dict"><b>device</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="device.html#Device">Device</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">The method for sequencing</a></span><span style="color: Gray"> --&gt;</span>&lt;/device&gt;
 &lt;<a href="sequence-definitions.html#Sequence.performer" title="The organization or lab that should be responsible for this result." class="dict"><b>performer</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="organization.html#Organization">Organization</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Who should be responsible for test result</a></span><span style="color: Gray"> --&gt;</span>&lt;/performer&gt;
 &lt;<a href="sequence-definitions.html#Sequence.quantity" title="The number of copies of the seqeunce of interest. (RNASeq)." class="dict"><b>quantity</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#Quantity">Quantity</a></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">The number of copies of the seqeunce of interest.  (RNASeq)</a></span><span style="color: Gray"> --&gt;</span>&lt;/quantity&gt;
 &lt;<a href="sequence-definitions.html#Sequence.referenceSeq" title="A sequence that is used as a reference to describe variants that are present in a sequence analyzed." class="dict"><b>referenceSeq</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..1</b></span> A sequence used as reference --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.chromosome" title="Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340))." class="dict"><b>chromosome</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> <span style="color: navy"><a style="color: navy" href="valueset-chromosome-human.html">Chromosome containing genetic finding</a></span><span style="color: Gray"> --&gt;</span>&lt;/chromosome&gt;
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild" title="The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used." class="dict"><b>genomeBuild</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.orientation" title="A relative reference to a DNA strand based on gene orientation. The strand that contains the open reading frame of the gene is the &quot;sense&quot; strand, and the opposite complementary strand is the &quot;antisense&quot; strand." class="dict"><b>orientation</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="valueset-orientation-type.html">sense | antisense</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId" title="Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences." class="dict"><b>referenceSeqId</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> <span style="color: navy"><a style="color: navy" href="valueset-sequence-referenceSeq.html">Reference identifier</a></span><span style="color: Gray"> --&gt;</span>&lt;/referenceSeqId&gt;
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer" title="A Pointer to another Sequence entity as reference sequence." class="dict"><b>referenceSeqPointer</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">A Pointer to another Sequence entity as reference sequence</a></span><span style="color: Gray"> --&gt;</span>&lt;/referenceSeqPointer&gt;
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString" title="A string like &quot;ACGT&quot;." class="dict"><b>referenceSeqString</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">A string to represent reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.strand" title="An absolute reference to a strand. The Watson strand is the strand whose 5'-end is on the short arm of the chromosome, and the Crick strand as the one whose 5'-end is on the long arm." class="dict"><b>strand</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="valueset-strand-type.html">watson | crick</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.windowStart" title="Start position of the window on the reference sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict"><b>windowStart</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Start position of the window on the  reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.windowEnd" title="End position of the window on the reference sequence. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict"><b>windowEnd</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">End position of the window on the reference sequence</a></span><span style="color: Gray"> --&gt;</span>
 &lt;/referenceSeq&gt;
 &lt;<a href="sequence-definitions.html#Sequence.variant" title="The definition of variant here originates from Sequence ontology ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of)). This element can represent amino acid or nucleic sequence change(including insertion,deletion,SNP,etc.)  It can represent some complex mutation or segment variation with the assist of CIGAR string." class="dict"><b>variant</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..*</b></span> Variant in sequence --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.start" title="Start position of the variant on the  reference sequence.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict"><b>start</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Start position of the variant on the  reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.end" title="End position of the variant on the reference sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict"><b>end</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">End position of the variant on the reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.observedAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)).  Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed  sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end." class="dict"><b>observedAllele</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Allele that was observed</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.referenceAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end." class="dict"><b>referenceAllele</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Allele in the reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.cigar" title="Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm)." class="dict"><b>cigar</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Extended CIGAR string for aligning the sequence with reference bases</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.variantPointer" title="A pointer to an Observation containing variant information." class="dict"><b>variantPointer</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="observation.html#Observation">Observation</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Pointer to observed variant information</a></span><span style="color: Gray"> --&gt;</span>&lt;/variantPointer&gt;
 &lt;/variant&gt;
 &lt;<a href="sequence-definitions.html#Sequence.observedSeq" title="Sequence that was observed. It is the result marked by referenceSeq along with variant records on referenceSeq. This shall starts from referenceSeq.windowStart and end by referenceSeq.windowEnd." class="dict"><b>observedSeq</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Sequence that was observed</a></span><span style="color: Gray"> --&gt;</span>
 &lt;<a href="sequence-definitions.html#Sequence.quality" title="An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686))." class="dict"><b>quality</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..*</b></span> An set of value as quality of sequence --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.type" title="INDEL / SNP / Undefined variant." class="dict"><b>type</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="valueset-quality-type.html">indel | snp | unknown</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.standardSequence" title="Gold standard sequence used for comparing against." class="dict"><b>standardSequence</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> <span style="color: navy"><a style="color: navy" href="valueset-sequence-quality-standardSequence.html">Standard sequence for comparison</a></span><span style="color: Gray"> --&gt;</span>&lt;/standardSequence&gt;
  &lt;<a href="sequence-definitions.html#Sequence.quality.start" title="Start position of the sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict"><b>start</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Start position of the sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.end" title="End position of the sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict"><b>end</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">End position of the sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.score" title="The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685))." class="dict"><b>score</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#Quantity">Quantity</a></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Quality score for the comparison</a></span><span style="color: Gray"> --&gt;</span>&lt;/score&gt;
  &lt;<a href="sequence-definitions.html#Sequence.quality.method" title="Which method is used to get sequence quality." class="dict"><b>method</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> <span style="color: navy"><a style="color: navy" href="valueset-sequence-quality-method.html">Method to get quality</a></span><span style="color: Gray"> --&gt;</span>&lt;/method&gt;
  &lt;<a href="sequence-definitions.html#Sequence.quality.truthTP" title="True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict"><b>truthTP</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">True positives from the perspective of the truth data</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.queryTP" title="True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict"><b>queryTP</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">True positives from the perspective of the query data</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.truthFN" title="False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here." class="dict"><b>truthFN</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">False negatives</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.queryFP" title="False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here." class="dict"><b>queryFP</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">False positives</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.gtFP" title="The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar)." class="dict"><b>gtFP</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">False positives where the non-REF alleles in the Truth and Query Call Sets match</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.precision" title="QUERY.TP / (QUERY.TP + QUERY.FP)." class="dict"><b>precision</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Precision of comparison</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.recall" title="TRUTH.TP / (TRUTH.TP + TRUTH.FN)." class="dict"><b>recall</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Recall of comparison</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.fScore" title="Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall)." class="dict"><b>fScore</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">F-score</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.roc" title="Receiver Operator Characteristic (ROC) Curve  to give sensitivity/specificity tradeoff." class="dict"><b>roc</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..1</b></span> Receiver Operator Characteristic (ROC) Curve --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.quality.roc.score" title="Invidual data point representing the GQ (genotype quality) score threshold." class="dict"><b>score</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Genotype quality score</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.quality.roc.numTP" title="The number of true positives if the GQ score threshold was set to &quot;score&quot; field value." class="dict"><b>numTP</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Roc score true positive numbers</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.quality.roc.numFP" title="The number of false positives if the GQ score threshold was set to &quot;score&quot; field value." class="dict"><b>numFP</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Roc score false positive numbers</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.quality.roc.numFN" title="The number of false negatives if the GQ score threshold was set to &quot;score&quot; field value." class="dict"><b>numFN</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Roc score false negative numbers</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.quality.roc.precision" title="Calculated precision if the GQ score threshold was set to &quot;score&quot; field value." class="dict"><b>precision</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Precision of the GQ score</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.quality.roc.sensitivity" title="Calculated sensitivity if the GQ score threshold was set to &quot;score&quot; field value." class="dict"><b>sensitivity</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Sensitivity of the GQ score</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.quality.roc.fMeasure" title="Calculated fScore if the GQ score threshold was set to &quot;score&quot; field value." class="dict"><b>fMeasure</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">FScore of the GQ score</a></span><span style="color: Gray"> --&gt;</span>
  &lt;/roc&gt;
 &lt;/quality&gt;
 &lt;<a href="sequence-definitions.html#Sequence.readCoverage" title="Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence." class="dict"><b>readCoverage</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Average number of reads representing a given nucleotide in the reconstructed sequence</a></span><span style="color: Gray"> --&gt;</span>
 &lt;<a href="sequence-definitions.html#Sequence.repository" title="Configurations of the external repository. The repository shall store target's observedSeq or records related with target's observedSeq." class="dict"><b>repository</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..*</b></span> External repository which contains detailed report related with observedSeq in this resource --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.type" title="Click and see / RESTful API / Need login to see / RESTful API with authentication / Other ways to see resource." class="dict"><b>type</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="valueset-repository-type.html">directlink | openapi | login | oauth | other</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.url" title="URI of an external repository which contains further details about the genetics data." class="dict"><b>url</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#uri">uri</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">URI of the repository</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.name" title="URI of an external repository which contains further details about the genetics data." class="dict"><b>name</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Repository's name</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.datasetId" title="Id of the variant in this external repository. The server will understand how to use this id to call for more info about datasets in external repository." class="dict"><b>datasetId</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Id of the dataset that used to call for dataset in repository</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.variantsetId" title="Id of the variantset in this external repository. The server will understand how to use this id to call for more info about variantsets in external repository." class="dict"><b>variantsetId</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Id of the variantset that used to call for variantset in repository</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.readsetId" title="Id of the read in this external repository." class="dict"><b>readsetId</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Id of the read</a></span><span style="color: Gray"> --&gt;</span>
 &lt;/repository&gt;
 &lt;<a href="sequence-definitions.html#Sequence.pointer" title="Pointer to next atomic sequence which at most contains one variant." class="dict"><b>pointer</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Pointer to next atomic sequence</a></span><span style="color: Gray"> --&gt;</span>&lt;/pointer&gt;
 &lt;<a href="sequence-definitions.html#Sequence.structureVariant" title="Information about chromosome structure variation." class="dict"><b>structureVariant</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..*</b></span> Structural variant --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.precision" title="Identify the exact boundaries of variant sequences. Each type of structure variant requires the DNA duplex to be broken and rejoined, and this creates a new sequence of bases at the rejoined sites, known as breakpoints or boundaries." class="dict"><b>precision</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Precision of boundaries</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio" title="Structural Variant reported aCGH ratio." class="dict"><b>reportedaCGHRatio</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant reported aCGH ratio</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.length" title="Length of the variant choromosome." class="dict"><b>length</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Length</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.outer" title="Structural variant outer." class="dict"><b>outer</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..1</b></span> Structural variant outer --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.structureVariant.outer.start" title="Structural Variant Outer Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict"><b>start</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Outer Start</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.structureVariant.outer.end" title="Structural Variant Outer End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict"><b>end</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Outer End</a></span><span style="color: Gray"> --&gt;</span>
  &lt;/outer&gt;
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.inner" title="Structural variant inner." class="dict"><b>inner</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..1</b></span> Structural variant inner --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.structureVariant.inner.start" title="Structural Variant Inner Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict"><b>start</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Inner Start</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.structureVariant.inner.end" title="Structural Variant Inner End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict"><b>end</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Inner End</a></span><span style="color: Gray"> --&gt;</span>
  &lt;/inner&gt;
 &lt;/structureVariant&gt;
&lt;/Sequence&gt;
</pre>

 </div>
</div>

  </div>
  <div id="tabs-json">

<div id="json">
<p><b>JSON Template</b></p>
 <div id="json-inner">
  <pre class="spec">
{<span style="float: right"><a href="json.html" title="Documentation for this format"><img src="help.png" alt="doco"/></a></span>
  "resourceType" : "<a href="sequence-definitions.html#Sequence" title="Raw data describing a biological sequence." class="dict"><b>Sequence</b></a>",
  // from <a href="resource.html">Resource</a>: <a href="resource.html#id">id</a>, <a href="resource.html#meta">meta</a>, <a href="resource.html#implicitRules">implicitRules</a>, and <a href="resource.html#language">language</a>
  // from <a href="domainresource.html">DomainResource</a>: <a href="narrative.html#Narrative">text</a>, <a href="references.html#contained">contained</a>, <a href="extensibility.html">extension</a>, and <a href="extensibility.html#modifierExtension">modifierExtension</a>
  "<a href="sequence-definitions.html#Sequence.identifier" title="A unique identifier for this particular sequence instance. This is a FHIR-defined id." class="dict">identifier</a>" : [{ <span style="color: darkgreen"><a href="datatypes.html#Identifier">Identifier</a></span> }], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Unique ID for this particular sequence. This is a FHIR-defined id</a></span>
  "<a href="sequence-definitions.html#Sequence.type" title="Amino Acid Sequence/ DNA Sequence / RNA Sequence." class="dict">type</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-sequence-type.html">aa | dna | rna</a></span>
  "<a href="sequence-definitions.html#Sequence.coordinateSystem" title="Whether the sequence is numbered starting at 0 (0-based numbering or coordinates, inclusive start, exclusive end) or starting at 1 (1-based numbering, inclusive start and inclusive end)." class="dict">coordinateSystem</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Base number of coordinate system (0 for 0-based numbering or coordinates, inclusive start, exclusive end, 1 for 1-based numbering, inclusive start, inclusive end)</a></span>
  "<a href="sequence-definitions.html#Sequence.patient" title="The patient whose sequencing results are described by this resource." class="dict">patient</a>" : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="patient.html#Patient">Patient</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Who and/or what this is about</a></span>
  "<a href="sequence-definitions.html#Sequence.specimen" title="Specimen used for sequencing." class="dict">specimen</a>" : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="specimen.html#Specimen">Specimen</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Specimen used for sequencing</a></span>
  "<a href="sequence-definitions.html#Sequence.device" title="The method for sequencing, for example, chip information." class="dict">device</a>" : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="device.html#Device">Device</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">The method for sequencing</a></span>
  "<a href="sequence-definitions.html#Sequence.performer" title="The organization or lab that should be responsible for this result." class="dict">performer</a>" : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="organization.html#Organization">Organization</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Who should be responsible for test result</a></span>
  "<a href="sequence-definitions.html#Sequence.quantity" title="The number of copies of the seqeunce of interest. (RNASeq)." class="dict">quantity</a>" : { <span style="color: darkgreen"><a href="datatypes.html#Quantity">Quantity</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">The number of copies of the seqeunce of interest.  (RNASeq)</a></span>
  "<a href="sequence-definitions.html#Sequence.referenceSeq" title="A sequence that is used as a reference to describe variants that are present in a sequence analyzed." class="dict">referenceSeq</a>" : { <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">A sequence used as reference</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.chromosome" title="Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340))." class="dict">chromosome</a>" : { <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-chromosome-human.html">Chromosome containing genetic finding</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild" title="The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used." class="dict">genomeBuild</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.orientation" title="A relative reference to a DNA strand based on gene orientation. The strand that contains the open reading frame of the gene is the &quot;sense&quot; strand, and the opposite complementary strand is the &quot;antisense&quot; strand." class="dict">orientation</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-orientation-type.html">sense | antisense</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId" title="Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences." class="dict">referenceSeqId</a>" : { <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-sequence-referenceSeq.html">Reference identifier</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer" title="A Pointer to another Sequence entity as reference sequence." class="dict">referenceSeqPointer</a>" : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">A Pointer to another Sequence entity as reference sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString" title="A string like &quot;ACGT&quot;." class="dict">referenceSeqString</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">A string to represent reference sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.strand" title="An absolute reference to a strand. The Watson strand is the strand whose 5'-end is on the short arm of the chromosome, and the Crick strand as the one whose 5'-end is on the long arm." class="dict">strand</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-strand-type.html">watson | crick</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.windowStart" title="Start position of the window on the reference sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">windowStart</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Start position of the window on the  reference sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.windowEnd" title="End position of the window on the reference sequence. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">windowEnd</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt; <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">End position of the window on the reference sequence</a></span>
  },
  "<a href="sequence-definitions.html#Sequence.variant" title="The definition of variant here originates from Sequence ontology ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of)). This element can represent amino acid or nucleic sequence change(including insertion,deletion,SNP,etc.)  It can represent some complex mutation or segment variation with the assist of CIGAR string." class="dict">variant</a>" : [{ <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Variant in sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.variant.start" title="Start position of the variant on the  reference sequence.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">start</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Start position of the variant on the  reference sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.variant.end" title="End position of the variant on the reference sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">end</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">End position of the variant on the reference sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.variant.observedAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)).  Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed  sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end." class="dict">observedAllele</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Allele that was observed</a></span>
    "<a href="sequence-definitions.html#Sequence.variant.referenceAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end." class="dict">referenceAllele</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Allele in the reference sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.variant.cigar" title="Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm)." class="dict">cigar</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Extended CIGAR string for aligning the sequence with reference bases</a></span>
    "<a href="sequence-definitions.html#Sequence.variant.variantPointer" title="A pointer to an Observation containing variant information." class="dict">variantPointer</a>" : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="observation.html#Observation">Observation</a>)</span> } <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Pointer to observed variant information</a></span>
  }],
  "<a href="sequence-definitions.html#Sequence.observedSeq" title="Sequence that was observed. It is the result marked by referenceSeq along with variant records on referenceSeq. This shall starts from referenceSeq.windowStart and end by referenceSeq.windowEnd." class="dict">observedSeq</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Sequence that was observed</a></span>
  "<a href="sequence-definitions.html#Sequence.quality" title="An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686))." class="dict">quality</a>" : [{ <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">An set of value as quality of sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.type" title="INDEL / SNP / Undefined variant." class="dict">type</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-quality-type.html">indel | snp | unknown</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.standardSequence" title="Gold standard sequence used for comparing against." class="dict">standardSequence</a>" : { <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-sequence-quality-standardSequence.html">Standard sequence for comparison</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.start" title="Start position of the sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">start</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Start position of the sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.end" title="End position of the sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">end</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">End position of the sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.score" title="The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685))." class="dict">score</a>" : { <span style="color: darkgreen"><a href="datatypes.html#Quantity">Quantity</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Quality score for the comparison</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.method" title="Which method is used to get sequence quality." class="dict">method</a>" : { <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-sequence-quality-method.html">Method to get quality</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.truthTP" title="True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict">truthTP</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">True positives from the perspective of the truth data</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.queryTP" title="True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict">queryTP</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">True positives from the perspective of the query data</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.truthFN" title="False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here." class="dict">truthFN</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">False negatives</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.queryFP" title="False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here." class="dict">queryFP</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">False positives</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.gtFP" title="The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar)." class="dict">gtFP</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">False positives where the non-REF alleles in the Truth and Query Call Sets match</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.precision" title="QUERY.TP / (QUERY.TP + QUERY.FP)." class="dict">precision</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Precision of comparison</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.recall" title="TRUTH.TP / (TRUTH.TP + TRUTH.FN)." class="dict">recall</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Recall of comparison</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.fScore" title="Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall)." class="dict">fScore</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">F-score</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.roc" title="Receiver Operator Characteristic (ROC) Curve  to give sensitivity/specificity tradeoff." class="dict">roc</a>" : { <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Receiver Operator Characteristic (ROC) Curve</a></span>
      "<a href="sequence-definitions.html#Sequence.quality.roc.score" title="Invidual data point representing the GQ (genotype quality) score threshold." class="dict">score</a>" : [&lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Genotype quality score</a></span>
      "<a href="sequence-definitions.html#Sequence.quality.roc.numTP" title="The number of true positives if the GQ score threshold was set to &quot;score&quot; field value." class="dict">numTP</a>" : [&lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Roc score true positive numbers</a></span>
      "<a href="sequence-definitions.html#Sequence.quality.roc.numFP" title="The number of false positives if the GQ score threshold was set to &quot;score&quot; field value." class="dict">numFP</a>" : [&lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Roc score false positive numbers</a></span>
      "<a href="sequence-definitions.html#Sequence.quality.roc.numFN" title="The number of false negatives if the GQ score threshold was set to &quot;score&quot; field value." class="dict">numFN</a>" : [&lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Roc score false negative numbers</a></span>
      "<a href="sequence-definitions.html#Sequence.quality.roc.precision" title="Calculated precision if the GQ score threshold was set to &quot;score&quot; field value." class="dict">precision</a>" : [&lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Precision of the GQ score</a></span>
      "<a href="sequence-definitions.html#Sequence.quality.roc.sensitivity" title="Calculated sensitivity if the GQ score threshold was set to &quot;score&quot; field value." class="dict">sensitivity</a>" : [&lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Sensitivity of the GQ score</a></span>
      "<a href="sequence-definitions.html#Sequence.quality.roc.fMeasure" title="Calculated fScore if the GQ score threshold was set to &quot;score&quot; field value." class="dict">fMeasure</a>" : [&lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;] <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">FScore of the GQ score</a></span>
    }
  }],
  "<a href="sequence-definitions.html#Sequence.readCoverage" title="Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence." class="dict">readCoverage</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Average number of reads representing a given nucleotide in the reconstructed sequence</a></span>
  "<a href="sequence-definitions.html#Sequence.repository" title="Configurations of the external repository. The repository shall store target's observedSeq or records related with target's observedSeq." class="dict">repository</a>" : [{ <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">External repository which contains detailed report related with observedSeq in this resource</a></span>
    "<a href="sequence-definitions.html#Sequence.repository.type" title="Click and see / RESTful API / Need login to see / RESTful API with authentication / Other ways to see resource." class="dict">type</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-repository-type.html">directlink | openapi | login | oauth | other</a></span>
    "<a href="sequence-definitions.html#Sequence.repository.url" title="URI of an external repository which contains further details about the genetics data." class="dict">url</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#uri">uri</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">URI of the repository</a></span>
    "<a href="sequence-definitions.html#Sequence.repository.name" title="URI of an external repository which contains further details about the genetics data." class="dict">name</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Repository's name</a></span>
    "<a href="sequence-definitions.html#Sequence.repository.datasetId" title="Id of the variant in this external repository. The server will understand how to use this id to call for more info about datasets in external repository." class="dict">datasetId</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Id of the dataset that used to call for dataset in repository</a></span>
    "<a href="sequence-definitions.html#Sequence.repository.variantsetId" title="Id of the variantset in this external repository. The server will understand how to use this id to call for more info about variantsets in external repository." class="dict">variantsetId</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Id of the variantset that used to call for variantset in repository</a></span>
    "<a href="sequence-definitions.html#Sequence.repository.readsetId" title="Id of the read in this external repository." class="dict">readsetId</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;" <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Id of the read</a></span>
  }],
  "<a href="sequence-definitions.html#Sequence.pointer" title="Pointer to next atomic sequence which at most contains one variant." class="dict">pointer</a>" : [{ <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>)</span> }], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Pointer to next atomic sequence</a></span>
  "<a href="sequence-definitions.html#Sequence.structureVariant" title="Information about chromosome structure variation." class="dict">structureVariant</a>" : [{ <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural variant</a></span>
    "<a href="sequence-definitions.html#Sequence.structureVariant.precision" title="Identify the exact boundaries of variant sequences. Each type of structure variant requires the DNA duplex to be broken and rejoined, and this creates a new sequence of bases at the rejoined sites, known as breakpoints or boundaries." class="dict">precision</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Precision of boundaries</a></span>
    "<a href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio" title="Structural Variant reported aCGH ratio." class="dict">reportedaCGHRatio</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant reported aCGH ratio</a></span>
    "<a href="sequence-definitions.html#Sequence.structureVariant.length" title="Length of the variant choromosome." class="dict">length</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Length</a></span>
    "<a href="sequence-definitions.html#Sequence.structureVariant.outer" title="Structural variant outer." class="dict">outer</a>" : { <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural variant outer</a></span>
      "<a href="sequence-definitions.html#Sequence.structureVariant.outer.start" title="Structural Variant Outer Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">start</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Outer Start</a></span>
      "<a href="sequence-definitions.html#Sequence.structureVariant.outer.end" title="Structural Variant Outer End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">end</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt; <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Outer End</a></span>
    },
    "<a href="sequence-definitions.html#Sequence.structureVariant.inner" title="Structural variant inner." class="dict">inner</a>" : { <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural variant inner</a></span>
      "<a href="sequence-definitions.html#Sequence.structureVariant.inner.start" title="Structural Variant Inner Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">start</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Inner Start</a></span>
      "<a href="sequence-definitions.html#Sequence.structureVariant.inner.end" title="Structural Variant Inner End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">end</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt; <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Inner End</a></span>
    }
  }]
}
</pre>

 </div>
</div>

  </div>
  <div id="tabs-ttl">

<div id="ttl">
<p><b>Turtle Template</b></p>
 <div id="json-inner">
  <pre class="spec">
@prefix fhir: &lt;http://hl7.org/fhir/&gt; .<span style="float: right"><a href="rdf.html" title="Documentation for this format"><img src="help.png" alt="doco"/></a></span>


[ a fhir:<a href="sequence-definitions.html#Sequence" title="Raw data describing a biological sequence." class="dict"><b>Sequence</b></a>;
  fhir:nodeRole fhir:treeRoot; # if this is the parser root

  # from <a href="resource.html">Resource</a>: <a href="resource.html#id">.id</a>, <a href="resource.html#meta">.meta</a>, <a href="resource.html#implicitRules">.implicitRules</a>, and <a href="resource.html#language">.language</a>
  # from <a href="domainresource.html">DomainResource</a>: <a href="narrative.html#Narrative">.text</a>, <a href="references.html#contained">.contained</a>, <a href="extensibility.html">.extension</a>, and <a href="extensibility.html#modifierExtension">.modifierExtension</a>
  fhir:<a href="sequence-definitions.html#Sequence.identifier" title="A unique identifier for this particular sequence instance. This is a FHIR-defined id." class="dict">Sequence.identifier</a><a name="ttl-identifier"> </a>[ <a href="datatypes.html#Identifier">Identifier</a> ], ... ; # 0..* <span style="color: navy">Unique ID for this particular sequence. This is a FHIR-defined id</span>
  fhir:<a href="sequence-definitions.html#Sequence.type" title="Amino Acid Sequence/ DNA Sequence / RNA Sequence." class="dict">Sequence.type</a><a name="ttl-type"> </a>[ <a href="datatypes.html#code">code</a> ]; # 0..1 <span style="color: navy">aa | dna | rna</span>
  fhir:<a href="sequence-definitions.html#Sequence.coordinateSystem" title="Whether the sequence is numbered starting at 0 (0-based numbering or coordinates, inclusive start, exclusive end) or starting at 1 (1-based numbering, inclusive start and inclusive end)." class="dict">Sequence.coordinateSystem</a><a name="ttl-coordinateSystem"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 1..1 <span style="color: navy">Base number of coordinate system (0 for 0-based numbering or coordinates, inclusive start, exclusive end, 1 for 1-based numbering, inclusive start, inclusive end)</span>
  fhir:<a href="sequence-definitions.html#Sequence.patient" title="The patient whose sequencing results are described by this resource." class="dict">Sequence.patient</a><a name="ttl-patient"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="patient.html#Patient">Patient</a>) ]; # 0..1 <span style="color: navy">Who and/or what this is about</span>
  fhir:<a href="sequence-definitions.html#Sequence.specimen" title="Specimen used for sequencing." class="dict">Sequence.specimen</a><a name="ttl-specimen"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="specimen.html#Specimen">Specimen</a>) ]; # 0..1 <span style="color: navy">Specimen used for sequencing</span>
  fhir:<a href="sequence-definitions.html#Sequence.device" title="The method for sequencing, for example, chip information." class="dict">Sequence.device</a><a name="ttl-device"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="device.html#Device">Device</a>) ]; # 0..1 <span style="color: navy">The method for sequencing</span>
  fhir:<a href="sequence-definitions.html#Sequence.performer" title="The organization or lab that should be responsible for this result." class="dict">Sequence.performer</a><a name="ttl-performer"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="organization.html#Organization">Organization</a>) ]; # 0..1 <span style="color: navy">Who should be responsible for test result</span>
  fhir:<a href="sequence-definitions.html#Sequence.quantity" title="The number of copies of the seqeunce of interest. (RNASeq)." class="dict">Sequence.quantity</a><a name="ttl-quantity"> </a>[ <a href="datatypes.html#Quantity">Quantity</a> ]; # 0..1 <span style="color: navy">The number of copies of the seqeunce of interest.  (RNASeq)</span>
  fhir:<a href="sequence-definitions.html#Sequence.referenceSeq" title="A sequence that is used as a reference to describe variants that are present in a sequence analyzed." class="dict">Sequence.referenceSeq</a><a name="ttl-referenceSeq"> </a>[ # 0..1 <span style="color: navy">A sequence used as reference</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.chromosome" title="Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340))." class="dict">Sequence.referenceSeq.chromosome</a><a name="ttl-chromosome"> </a>[ <a href="datatypes.html#CodeableConcept">CodeableConcept</a> ]; # 0..1 <span style="color: navy">Chromosome containing genetic finding</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild" title="The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used." class="dict">Sequence.referenceSeq.genomeBuild</a><a name="ttl-genomeBuild"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.orientation" title="A relative reference to a DNA strand based on gene orientation. The strand that contains the open reading frame of the gene is the &quot;sense&quot; strand, and the opposite complementary strand is the &quot;antisense&quot; strand." class="dict">Sequence.referenceSeq.orientation</a><a name="ttl-orientation"> </a>[ <a href="datatypes.html#code">code</a> ]; # 0..1 <span style="color: navy">sense | antisense</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId" title="Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences." class="dict">Sequence.referenceSeq.referenceSeqId</a><a name="ttl-referenceSeqId"> </a>[ <a href="datatypes.html#CodeableConcept">CodeableConcept</a> ]; # 0..1 <span style="color: navy">Reference identifier</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer" title="A Pointer to another Sequence entity as reference sequence." class="dict">Sequence.referenceSeq.referenceSeqPointer</a><a name="ttl-referenceSeqPointer"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>) ]; # 0..1 <span style="color: navy">A Pointer to another Sequence entity as reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString" title="A string like &quot;ACGT&quot;." class="dict">Sequence.referenceSeq.referenceSeqString</a><a name="ttl-referenceSeqString"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">A string to represent reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.strand" title="An absolute reference to a strand. The Watson strand is the strand whose 5'-end is on the short arm of the chromosome, and the Crick strand as the one whose 5'-end is on the long arm." class="dict">Sequence.referenceSeq.strand</a><a name="ttl-strand"> </a>[ <a href="datatypes.html#code">code</a> ]; # 0..1 <span style="color: navy">watson | crick</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.windowStart" title="Start position of the window on the reference sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">Sequence.referenceSeq.windowStart</a><a name="ttl-windowStart"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 1..1 <span style="color: navy">Start position of the window on the  reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.windowEnd" title="End position of the window on the reference sequence. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">Sequence.referenceSeq.windowEnd</a><a name="ttl-windowEnd"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 1..1 <span style="color: navy">End position of the window on the reference sequence</span>
  ];
  fhir:<a href="sequence-definitions.html#Sequence.variant" title="The definition of variant here originates from Sequence ontology ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of)). This element can represent amino acid or nucleic sequence change(including insertion,deletion,SNP,etc.)  It can represent some complex mutation or segment variation with the assist of CIGAR string." class="dict">Sequence.variant</a><a name="ttl-variant"> </a>[ # 0..* <span style="color: navy">Variant in sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.start" title="Start position of the variant on the  reference sequence.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">Sequence.variant.start</a><a name="ttl-start"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Start position of the variant on the  reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.end" title="End position of the variant on the reference sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">Sequence.variant.end</a><a name="ttl-end"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">End position of the variant on the reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.observedAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)).  Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed  sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end." class="dict">Sequence.variant.observedAllele</a><a name="ttl-observedAllele"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Allele that was observed</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.referenceAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end." class="dict">Sequence.variant.referenceAllele</a><a name="ttl-referenceAllele"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Allele in the reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.cigar" title="Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm)." class="dict">Sequence.variant.cigar</a><a name="ttl-cigar"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Extended CIGAR string for aligning the sequence with reference bases</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.variantPointer" title="A pointer to an Observation containing variant information." class="dict">Sequence.variant.variantPointer</a><a name="ttl-variantPointer"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="observation.html#Observation">Observation</a>) ]; # 0..1 <span style="color: navy">Pointer to observed variant information</span>
  ], ...;
  fhir:<a href="sequence-definitions.html#Sequence.observedSeq" title="Sequence that was observed. It is the result marked by referenceSeq along with variant records on referenceSeq. This shall starts from referenceSeq.windowStart and end by referenceSeq.windowEnd." class="dict">Sequence.observedSeq</a><a name="ttl-observedSeq"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Sequence that was observed</span>
  fhir:<a href="sequence-definitions.html#Sequence.quality" title="An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686))." class="dict">Sequence.quality</a><a name="ttl-quality"> </a>[ # 0..* <span style="color: navy">An set of value as quality of sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.type" title="INDEL / SNP / Undefined variant." class="dict">Sequence.quality.type</a><a name="ttl-type"> </a>[ <a href="datatypes.html#code">code</a> ]; # 1..1 <span style="color: navy">indel | snp | unknown</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.standardSequence" title="Gold standard sequence used for comparing against." class="dict">Sequence.quality.standardSequence</a><a name="ttl-standardSequence"> </a>[ <a href="datatypes.html#CodeableConcept">CodeableConcept</a> ]; # 0..1 <span style="color: navy">Standard sequence for comparison</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.start" title="Start position of the sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">Sequence.quality.start</a><a name="ttl-start"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Start position of the sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.end" title="End position of the sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">Sequence.quality.end</a><a name="ttl-end"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">End position of the sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.score" title="The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685))." class="dict">Sequence.quality.score</a><a name="ttl-score"> </a>[ <a href="datatypes.html#Quantity">Quantity</a> ]; # 0..1 <span style="color: navy">Quality score for the comparison</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.method" title="Which method is used to get sequence quality." class="dict">Sequence.quality.method</a><a name="ttl-method"> </a>[ <a href="datatypes.html#CodeableConcept">CodeableConcept</a> ]; # 0..1 <span style="color: navy">Method to get quality</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.truthTP" title="True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict">Sequence.quality.truthTP</a><a name="ttl-truthTP"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">True positives from the perspective of the truth data</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.queryTP" title="True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict">Sequence.quality.queryTP</a><a name="ttl-queryTP"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">True positives from the perspective of the query data</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.truthFN" title="False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here." class="dict">Sequence.quality.truthFN</a><a name="ttl-truthFN"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">False negatives</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.queryFP" title="False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here." class="dict">Sequence.quality.queryFP</a><a name="ttl-queryFP"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">False positives</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.gtFP" title="The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar)." class="dict">Sequence.quality.gtFP</a><a name="ttl-gtFP"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">False positives where the non-REF alleles in the Truth and Query Call Sets match</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.precision" title="QUERY.TP / (QUERY.TP + QUERY.FP)." class="dict">Sequence.quality.precision</a><a name="ttl-precision"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">Precision of comparison</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.recall" title="TRUTH.TP / (TRUTH.TP + TRUTH.FN)." class="dict">Sequence.quality.recall</a><a name="ttl-recall"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">Recall of comparison</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.fScore" title="Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall)." class="dict">Sequence.quality.fScore</a><a name="ttl-fScore"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">F-score</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.roc" title="Receiver Operator Characteristic (ROC) Curve  to give sensitivity/specificity tradeoff." class="dict">Sequence.quality.roc</a><a name="ttl-roc"> </a>[ # 0..1 <span style="color: navy">Receiver Operator Characteristic (ROC) Curve</span>
      fhir:<a href="sequence-definitions.html#Sequence.quality.roc.score" title="Invidual data point representing the GQ (genotype quality) score threshold." class="dict">Sequence.quality.roc.score</a><a name="ttl-score"> </a>[ <a href="datatypes.html#integer">integer</a> ], ... ; # 0..* <span style="color: navy">Genotype quality score</span>
      fhir:<a href="sequence-definitions.html#Sequence.quality.roc.numTP" title="The number of true positives if the GQ score threshold was set to &quot;score&quot; field value." class="dict">Sequence.quality.roc.numTP</a><a name="ttl-numTP"> </a>[ <a href="datatypes.html#integer">integer</a> ], ... ; # 0..* <span style="color: navy">Roc score true positive numbers</span>
      fhir:<a href="sequence-definitions.html#Sequence.quality.roc.numFP" title="The number of false positives if the GQ score threshold was set to &quot;score&quot; field value." class="dict">Sequence.quality.roc.numFP</a><a name="ttl-numFP"> </a>[ <a href="datatypes.html#integer">integer</a> ], ... ; # 0..* <span style="color: navy">Roc score false positive numbers</span>
      fhir:<a href="sequence-definitions.html#Sequence.quality.roc.numFN" title="The number of false negatives if the GQ score threshold was set to &quot;score&quot; field value." class="dict">Sequence.quality.roc.numFN</a><a name="ttl-numFN"> </a>[ <a href="datatypes.html#integer">integer</a> ], ... ; # 0..* <span style="color: navy">Roc score false negative numbers</span>
      fhir:<a href="sequence-definitions.html#Sequence.quality.roc.precision" title="Calculated precision if the GQ score threshold was set to &quot;score&quot; field value." class="dict">Sequence.quality.roc.precision</a><a name="ttl-precision"> </a>[ <a href="datatypes.html#decimal">decimal</a> ], ... ; # 0..* <span style="color: navy">Precision of the GQ score</span>
      fhir:<a href="sequence-definitions.html#Sequence.quality.roc.sensitivity" title="Calculated sensitivity if the GQ score threshold was set to &quot;score&quot; field value." class="dict">Sequence.quality.roc.sensitivity</a><a name="ttl-sensitivity"> </a>[ <a href="datatypes.html#decimal">decimal</a> ], ... ; # 0..* <span style="color: navy">Sensitivity of the GQ score</span>
      fhir:<a href="sequence-definitions.html#Sequence.quality.roc.fMeasure" title="Calculated fScore if the GQ score threshold was set to &quot;score&quot; field value." class="dict">Sequence.quality.roc.fMeasure</a><a name="ttl-fMeasure"> </a>[ <a href="datatypes.html#decimal">decimal</a> ], ... ; # 0..* <span style="color: navy">FScore of the GQ score</span>
    ];
  ], ...;
  fhir:<a href="sequence-definitions.html#Sequence.readCoverage" title="Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence." class="dict">Sequence.readCoverage</a><a name="ttl-readCoverage"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Average number of reads representing a given nucleotide in the reconstructed sequence</span>
  fhir:<a href="sequence-definitions.html#Sequence.repository" title="Configurations of the external repository. The repository shall store target's observedSeq or records related with target's observedSeq." class="dict">Sequence.repository</a><a name="ttl-repository"> </a>[ # 0..* <span style="color: navy">External repository which contains detailed report related with observedSeq in this resource</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.type" title="Click and see / RESTful API / Need login to see / RESTful API with authentication / Other ways to see resource." class="dict">Sequence.repository.type</a><a name="ttl-type"> </a>[ <a href="datatypes.html#code">code</a> ]; # 1..1 <span style="color: navy">directlink | openapi | login | oauth | other</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.url" title="URI of an external repository which contains further details about the genetics data." class="dict">Sequence.repository.url</a><a name="ttl-url"> </a>[ <a href="datatypes.html#uri">uri</a> ]; # 0..1 <span style="color: navy">URI of the repository</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.name" title="URI of an external repository which contains further details about the genetics data." class="dict">Sequence.repository.name</a><a name="ttl-name"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Repository's name</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.datasetId" title="Id of the variant in this external repository. The server will understand how to use this id to call for more info about datasets in external repository." class="dict">Sequence.repository.datasetId</a><a name="ttl-datasetId"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Id of the dataset that used to call for dataset in repository</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.variantsetId" title="Id of the variantset in this external repository. The server will understand how to use this id to call for more info about variantsets in external repository." class="dict">Sequence.repository.variantsetId</a><a name="ttl-variantsetId"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Id of the variantset that used to call for variantset in repository</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.readsetId" title="Id of the read in this external repository." class="dict">Sequence.repository.readsetId</a><a name="ttl-readsetId"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Id of the read</span>
  ], ...;
  fhir:<a href="sequence-definitions.html#Sequence.pointer" title="Pointer to next atomic sequence which at most contains one variant." class="dict">Sequence.pointer</a><a name="ttl-pointer"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>) ], ... ; # 0..* <span style="color: navy">Pointer to next atomic sequence</span>
  fhir:<a href="sequence-definitions.html#Sequence.structureVariant" title="Information about chromosome structure variation." class="dict">Sequence.structureVariant</a><a name="ttl-structureVariant"> </a>[ # 0..* <span style="color: navy">Structural variant</span>
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.precision" title="Identify the exact boundaries of variant sequences. Each type of structure variant requires the DNA duplex to be broken and rejoined, and this creates a new sequence of bases at the rejoined sites, known as breakpoints or boundaries." class="dict">Sequence.structureVariant.precision</a><a name="ttl-precision"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Precision of boundaries</span>
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio" title="Structural Variant reported aCGH ratio." class="dict">Sequence.structureVariant.reportedaCGHRatio</a><a name="ttl-reportedaCGHRatio"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">Structural Variant reported aCGH ratio</span>
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.length" title="Length of the variant choromosome." class="dict">Sequence.structureVariant.length</a><a name="ttl-length"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Length</span>
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.outer" title="Structural variant outer." class="dict">Sequence.structureVariant.outer</a><a name="ttl-outer"> </a>[ # 0..1 <span style="color: navy">Structural variant outer</span>
      fhir:<a href="sequence-definitions.html#Sequence.structureVariant.outer.start" title="Structural Variant Outer Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">Sequence.structureVariant.outer.start</a><a name="ttl-start"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Outer Start</span>
      fhir:<a href="sequence-definitions.html#Sequence.structureVariant.outer.end" title="Structural Variant Outer End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">Sequence.structureVariant.outer.end</a><a name="ttl-end"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Outer End</span>
    ];
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.inner" title="Structural variant inner." class="dict">Sequence.structureVariant.inner</a><a name="ttl-inner"> </a>[ # 0..1 <span style="color: navy">Structural variant inner</span>
      fhir:<a href="sequence-definitions.html#Sequence.structureVariant.inner.start" title="Structural Variant Inner Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">Sequence.structureVariant.inner.start</a><a name="ttl-start"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Inner Start</span>
      fhir:<a href="sequence-definitions.html#Sequence.structureVariant.inner.end" title="Structural Variant Inner End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">Sequence.structureVariant.inner.end</a><a name="ttl-end"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Inner End</span>
    ];
  ], ...;
]
</pre>

 </div>
</div>

  </div>
  <div id="tabs-diff">

<div id="vdiff">
<p><b>Changes since R3</b></p>
 <div id="diff-inner">
  <table class="grid">
  <tr class="diff-item">
    <td class="diff-left">
      <a href="sequence.html#Sequence">Sequence</a>
    </td>
    <td class="diff-right"></td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.referenceSeq.orientation</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-entry">
    <td class="diff-left">Sequence.referenceSeq.strand</td>
    <td class="diff-right">
      <ul>
        <li>Type changed from integer to code</li>
        <li>Add Binding http://hl7.org/fhir/ValueSet/strand-type (required)</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.quality.roc</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.quality.roc.score</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.quality.roc.numTP</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.quality.roc.numFP</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.quality.roc.numFN</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.quality.roc.precision</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.quality.roc.sensitivity</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.quality.roc.fMeasure</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.precision</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.reportedaCGHRatio</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.length</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.outer</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.outer.start</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.outer.end</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.inner</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.inner.start</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.inner.end</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
</table>

<p>See the <a href="diff.html">Full Difference</a> for further information</p>

  <p>
  This analysis is available as <a href="sequence.diff.xml">XML</a> or <a href="sequence.diff.json">JSON</a>.
  </p>
  <p>See <a href="sequence-version-maps.html">R3 &lt;--&gt; R4 Conversion Maps</a> (status = 16 tests that all execute ok. <span style="background-color: #ccffcc">16 fail round-trip testing</span> and <span style="background-color: #E0B0FF">16 r3 resources are invalid (0 errors).</span>)</p>

 </div>
</div>

  </div>
  <div id="tabs-all">

<div id="tbl">
<a name="tbl"></a>
<p><b>Structure</b></p>
 <div id="tbl-inner">
  <table border="0" cellpadding="0" cellspacing="0" style="border: 0px #F0F0F0 solid; font-size: 11px; font-family: verdana; vertical-align: top;"><tr style="border: 1px #F0F0F0 solid; font-size: 11px; font-family: verdana; vertical-align: top;"><th style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="formats.html#table" title="The logical name of the element">Name</a></th><th style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="formats.html#table" title="Information about the use of the element">Flags</a></th><th style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="formats.html#table" title="Minimum and Maximum # of times the the element can appear in the instance">Card.</a></th><th style="width: 100px" class="hierarchy"><a href="formats.html#table" title="Reference to the type of the element">Type</a></th><th style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="formats.html#table" title="Additional information about the element">Description &amp; Constraints</a><span style="float: right"><a href="formats.html#table" title="Legend for this format"><img src="help16.png" alt="doco" style="background-color: inherit"/></a></span></th></tr><tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_resource.png" alt="." style="background-color: white; background-color: inherit" title="Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence" title="Sequence : Raw data describing a biological sequence.">Sequence</a><a name="Sequence"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="conformance-rules.html#constraints" title="This element has or is affected by some invariants">I</a><a style="padding-left: 3px; padding-right: 3px; border: 1px grey solid; font-weight: bold; color: black; background-color: #fff5e6" href="versions.html#std-process" title="Ballot Status = Trial Use">TU</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="domainresource.html">DomainResource</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Information about a biological sequence<br/><span style="font-style: italic" title="seq-3">+ Rule: Only 0 and 1 are valid for coordinateSystem</span><br/>Elements defined in Ancestors: <a href="resource.html#Resource" title="The logical id of the resource, as used in the URL for the resource. Once assigned, this value never changes.">id</a>, <a href="resource.html#Resource" title="The metadata about the resource. This is content that is maintained by the infrastructure. Changes to the content might not always be associated with version changes to the resource.">meta</a>, <a href="resource.html#Resource" title="A reference to a set of rules that were followed when the resource was constructed, and which must be understood when processing the content. Often, this is a reference to an implementation guide that defines the special rules along with other profiles etc.">implicitRules</a>, <a href="resource.html#Resource" title="The base language in which the resource is written.">language</a>, <a href="domainresource.html#DomainResource" title="A human-readable narrative that contains a summary of the resource and can be used to represent the content of the resource to a human. The narrative need not encode all the structured data, but is required to contain sufficient detail to make it &quot;clinically safe&quot; for a human to just read the narrative. Resource definitions may define what content should be represented in the narrative to ensure clinical safety.">text</a>, <a href="domainresource.html#DomainResource" title="These resources do not have an independent existence apart from the resource that contains them - they cannot be identified independently, and nor can they have their own independent transaction scope.">contained</a>, <a href="domainresource.html#DomainResource" title="May be used to represent additional information that is not part of the basic definition of the resource. To make the use of extensions safe and manageable, there is a strict set of governance  applied to the definition and use of extensions. Though any implementer can define an extension, there is a set of requirements that SHALL be met as part of the definition of the extension.">extension</a>, <a href="domainresource.html#DomainResource" title="May be used to represent additional information that is not part of the basic definition of the resource, and that modifies the understanding of the element that contains it. Usually modifier elements provide negation or qualification. To make the use of extensions safe and manageable, there is a strict set of governance applied to the definition and use of extensions. Though any implementer is allowed to define an extension, there is a set of requirements that SHALL be met as part of the definition of the extension. Applications processing a resource are required to check for modifier extensions.

Modifier extensions SHALL NOT change the meaning of any elements on Resource or DomainResource (including cannot change the meaning of modifierExtension itself).">modifierExtension</a></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.identifier" title="Sequence.identifier : A unique identifier for this particular sequence instance. This is a FHIR-defined id.">identifier</a><a name="Sequence.identifier"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#Identifier">Identifier</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Unique ID for this particular sequence. This is a FHIR-defined id<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.type" title="Sequence.type : Amino Acid Sequence/ DNA Sequence / RNA Sequence.">type</a><a name="Sequence.type"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#code">code</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">aa | dna | rna<br/><a href="valueset-sequence-type.html" title="Type if a sequence -- DNA, RNA, or amino acid sequence.">sequenceType</a> (<a href="terminologies.html#example" title="Instances are not expected or even encouraged to draw from the specified value set.  The value set merely provides examples of the types of concepts intended to be included.">Example</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.coordinateSystem" title="Sequence.coordinateSystem : Whether the sequence is numbered starting at 0 (0-based numbering or coordinates, inclusive start, exclusive end) or starting at 1 (1-based numbering, inclusive start and inclusive end).">coordinateSystem</a><a name="Sequence.coordinateSystem"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Base number of coordinate system (0 for 0-based numbering or coordinates, inclusive start, exclusive end, 1 for 1-based numbering, inclusive start, inclusive end)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.patient" title="Sequence.patient : The patient whose sequencing results are described by this resource.">patient</a><a name="Sequence.patient"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html#Reference">Reference</a>(<a href="patient.html">Patient</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Who and/or what this is about</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.specimen" title="Sequence.specimen : Specimen used for sequencing.">specimen</a><a name="Sequence.specimen"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html#Reference">Reference</a>(<a href="specimen.html">Specimen</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Specimen used for sequencing</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.device" title="Sequence.device : The method for sequencing, for example, chip information.">device</a><a name="Sequence.device"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html#Reference">Reference</a>(<a href="device.html">Device</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">The method for sequencing</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.performer" title="Sequence.performer : The organization or lab that should be responsible for this result.">performer</a><a name="Sequence.performer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html#Reference">Reference</a>(<a href="organization.html">Organization</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Who should be responsible for test result</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quantity" title="Sequence.quantity : The number of copies of the seqeunce of interest. (RNASeq).">quantity</a><a name="Sequence.quantity"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#Quantity">Quantity</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">The number of copies of the seqeunce of interest.  (RNASeq)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck11.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq" title="Sequence.referenceSeq : A sequence that is used as a reference to describe variants that are present in a sequence analyzed.">referenceSeq</a><a name="Sequence.referenceSeq"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="conformance-rules.html#constraints" title="This element has or is affected by some invariants">I</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">A sequence used as reference<br/><span style="font-style: italic" title="seq-5">+ Rule: GenomeBuild and chromosome must be both contained if either one of them is contained</span><br/><span style="font-style: italic" title="seq-6">+ Rule: Have and only have one of the following elements in referenceSeq : 1. genomeBuild ; 2 referenceSeqId; 3. referenceSeqPointer;  4. referenceSeqString;</span></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.chromosome" title="Sequence.referenceSeq.chromosome : Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340)).">chromosome</a><a name="Sequence.referenceSeq.chromosome"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Chromosome containing genetic finding<br/><a href="valueset-chromosome-human.html" title="Chromosome number for human.">chromosome-human</a> (<a href="terminologies.html#example" title="Instances are not expected or even encouraged to draw from the specified value set.  The value set merely provides examples of the types of concepts intended to be included.">Example</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild" title="Sequence.referenceSeq.genomeBuild : The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used.">genomeBuild</a><a name="Sequence.referenceSeq.genomeBuild"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.orientation" title="Sequence.referenceSeq.orientation : A relative reference to a DNA strand based on gene orientation. The strand that contains the open reading frame of the gene is the &quot;sense&quot; strand, and the opposite complementary strand is the &quot;antisense&quot; strand.">orientation</a><a name="Sequence.referenceSeq.orientation"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#code">code</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">sense | antisense<br/><a href="valueset-orientation-type.html" title="Type for orientation.">orientationType</a> (<a href="terminologies.html#required" title="To be conformant, the concept in this element SHALL be from the specified value set.">Required</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId" title="Sequence.referenceSeq.referenceSeqId : Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences.">referenceSeqId</a><a name="Sequence.referenceSeq.referenceSeqId"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Reference identifier<br/><a href="valueset-sequence-referenceSeq.html" title="Reference identifier.">E n s e m b l</a> (<a href="terminologies.html#example" title="Instances are not expected or even encouraged to draw from the specified value set.  The value set merely provides examples of the types of concepts intended to be included.">Example</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer" title="Sequence.referenceSeq.referenceSeqPointer : A Pointer to another Sequence entity as reference sequence.">referenceSeqPointer</a><a name="Sequence.referenceSeq.referenceSeqPointer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html#Reference">Reference</a>(<a href="sequence.html">Sequence</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">A Pointer to another Sequence entity as reference sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString" title="Sequence.referenceSeq.referenceSeqString : A string like &quot;ACGT&quot;.">referenceSeqString</a><a name="Sequence.referenceSeq.referenceSeqString"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">A string to represent reference sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.strand" title="Sequence.referenceSeq.strand : An absolute reference to a strand. The Watson strand is the strand whose 5'-end is on the short arm of the chromosome, and the Crick strand as the one whose 5'-end is on the long arm.">strand</a><a name="Sequence.referenceSeq.strand"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#code">code</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">watson | crick<br/><a href="valueset-strand-type.html" title="Type for strand.">strandType</a> (<a href="terminologies.html#required" title="To be conformant, the concept in this element SHALL be from the specified value set.">Required</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.windowStart" title="Sequence.referenceSeq.windowStart : Start position of the window on the reference sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive.">windowStart</a><a name="Sequence.referenceSeq.windowStart"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Start position of the window on the  reference sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.referenceSeq.windowEnd" title="Sequence.referenceSeq.windowEnd : End position of the window on the reference sequence. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position.">windowEnd</a><a name="Sequence.referenceSeq.windowEnd"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">End position of the window on the reference sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck11.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant" title="Sequence.variant : The definition of variant here originates from Sequence ontology ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of)). This element can represent amino acid or nucleic sequence change(including insertion,deletion,SNP,etc.)  It can represent some complex mutation or segment variation with the assist of CIGAR string.">variant</a><a name="Sequence.variant"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Variant in sequence<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.start" title="Sequence.variant.start : Start position of the variant on the  reference sequence.If the coordinate system is either 0-based or 1-based, then start position is inclusive.">start</a><a name="Sequence.variant.start"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Start position of the variant on the  reference sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.end" title="Sequence.variant.end : End position of the variant on the reference sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position.">end</a><a name="Sequence.variant.end"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">End position of the variant on the reference sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.observedAllele" title="Sequence.variant.observedAllele : An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)).  Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed  sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end.">observedAllele</a><a name="Sequence.variant.observedAllele"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Allele that was observed</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.referenceAllele" title="Sequence.variant.referenceAllele : An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end.">referenceAllele</a><a name="Sequence.variant.referenceAllele"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Allele in the reference sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.cigar" title="Sequence.variant.cigar : Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm).">cigar</a><a name="Sequence.variant.cigar"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Extended CIGAR string for aligning the sequence with reference bases</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.variant.variantPointer" title="Sequence.variant.variantPointer : A pointer to an Observation containing variant information.">variantPointer</a><a name="Sequence.variant.variantPointer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html#Reference">Reference</a>(<a href="observation.html">Observation</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Pointer to observed variant information</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.observedSeq" title="Sequence.observedSeq : Sequence that was observed. It is the result marked by referenceSeq along with variant records on referenceSeq. This shall starts from referenceSeq.windowStart and end by referenceSeq.windowEnd.">observedSeq</a><a name="Sequence.observedSeq"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Sequence that was observed</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck11.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality" title="Sequence.quality : An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686)).">quality</a><a name="Sequence.quality"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">An set of value as quality of sequence<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.type" title="Sequence.quality.type : INDEL / SNP / Undefined variant.">type</a><a name="Sequence.quality.type"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#code">code</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">indel | snp | unknown<br/><a href="valueset-quality-type.html" title="Type for quality report.">qualityType</a> (<a href="terminologies.html#required" title="To be conformant, the concept in this element SHALL be from the specified value set.">Required</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.standardSequence" title="Sequence.quality.standardSequence : Gold standard sequence used for comparing against.">standardSequence</a><a name="Sequence.quality.standardSequence"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Standard sequence for comparison<br/><a href="valueset-sequence-quality-standardSequence.html" title="Reference identifier of the sequence that used to mark the quality of tested samples.">F d a- standard sequence</a> (<a href="terminologies.html#example" title="Instances are not expected or even encouraged to draw from the specified value set.  The value set merely provides examples of the types of concepts intended to be included.">Example</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.start" title="Sequence.quality.start : Start position of the sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive.">start</a><a name="Sequence.quality.start"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Start position of the sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.end" title="Sequence.quality.end : End position of the sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position.">end</a><a name="Sequence.quality.end"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">End position of the sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.score" title="Sequence.quality.score : The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685)).">score</a><a name="Sequence.quality.score"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#Quantity">Quantity</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Quality score for the comparison</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_datatype.gif" alt="." style="background-color: white; background-color: inherit" title="Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.method" title="Sequence.quality.method : Which method is used to get sequence quality.">method</a><a name="Sequence.quality.method"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Method to get quality<br/><a href="valueset-sequence-quality-method.html" title="The method used to evaluate the numerical quality of the observed sequence.">F d a- method</a> (<a href="terminologies.html#example" title="Instances are not expected or even encouraged to draw from the specified value set.  The value set merely provides examples of the types of concepts intended to be included.">Example</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.truthTP" title="Sequence.quality.truthTP : True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event.">truthTP</a><a name="Sequence.quality.truthTP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">True positives from the perspective of the truth data</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.queryTP" title="Sequence.quality.queryTP : True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event.">queryTP</a><a name="Sequence.quality.queryTP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">True positives from the perspective of the query data</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.truthFN" title="Sequence.quality.truthFN : False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here.">truthFN</a><a name="Sequence.quality.truthFN"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">False negatives</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.queryFP" title="Sequence.quality.queryFP : False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here.">queryFP</a><a name="Sequence.quality.queryFP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">False positives</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.gtFP" title="Sequence.quality.gtFP : The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar).">gtFP</a><a name="Sequence.quality.gtFP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">False positives where the non-REF alleles in the Truth and Query Call Sets match</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.precision" title="Sequence.quality.precision : QUERY.TP / (QUERY.TP + QUERY.FP).">precision</a><a name="Sequence.quality.precision"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Precision of comparison</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.recall" title="Sequence.quality.recall : TRUTH.TP / (TRUTH.TP + TRUTH.FN).">recall</a><a name="Sequence.quality.recall"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Recall of comparison</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.fScore" title="Sequence.quality.fScore : Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall).">fScore</a><a name="Sequence.quality.fScore"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">F-score</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck101.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.roc" title="Sequence.quality.roc : Receiver Operator Characteristic (ROC) Curve  to give sensitivity/specificity tradeoff.">roc</a><a name="Sequence.quality.roc"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Receiver Operator Characteristic (ROC) Curve</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.roc.score" title="Sequence.quality.roc.score : Invidual data point representing the GQ (genotype quality) score threshold.">score</a><a name="Sequence.quality.roc.score"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Genotype quality score<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.roc.numTP" title="Sequence.quality.roc.numTP : The number of true positives if the GQ score threshold was set to &quot;score&quot; field value.">numTP</a><a name="Sequence.quality.roc.numTP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Roc score true positive numbers<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.roc.numFP" title="Sequence.quality.roc.numFP : The number of false positives if the GQ score threshold was set to &quot;score&quot; field value.">numFP</a><a name="Sequence.quality.roc.numFP"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Roc score false positive numbers<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.roc.numFN" title="Sequence.quality.roc.numFN : The number of false negatives if the GQ score threshold was set to &quot;score&quot; field value.">numFN</a><a name="Sequence.quality.roc.numFN"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Roc score false negative numbers<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.roc.precision" title="Sequence.quality.roc.precision : Calculated precision if the GQ score threshold was set to &quot;score&quot; field value.">precision</a><a name="Sequence.quality.roc.precision"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Precision of the GQ score<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.roc.sensitivity" title="Sequence.quality.roc.sensitivity : Calculated sensitivity if the GQ score threshold was set to &quot;score&quot; field value.">sensitivity</a><a name="Sequence.quality.roc.sensitivity"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Sensitivity of the GQ score<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck1000.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.quality.roc.fMeasure" title="Sequence.quality.roc.fMeasure : Calculated fScore if the GQ score threshold was set to &quot;score&quot; field value.">fMeasure</a><a name="Sequence.quality.roc.fMeasure"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">FScore of the GQ score<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.readCoverage" title="Sequence.readCoverage : Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence.">readCoverage</a><a name="Sequence.readCoverage"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Average number of reads representing a given nucleotide in the reconstructed sequence</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck11.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository" title="Sequence.repository : Configurations of the external repository. The repository shall store target's observedSeq or records related with target's observedSeq.">repository</a><a name="Sequence.repository"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">External repository which contains detailed report related with observedSeq in this resource<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.type" title="Sequence.repository.type : Click and see / RESTful API / Need login to see / RESTful API with authentication / Other ways to see resource.">type</a><a name="Sequence.repository.type"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">1..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#code">code</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">directlink | openapi | login | oauth | other<br/><a href="valueset-repository-type.html" title="Type for access of external URI.">repositoryType</a> (<a href="terminologies.html#required" title="To be conformant, the concept in this element SHALL be from the specified value set.">Required</a>)</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.url" title="Sequence.repository.url : URI of an external repository which contains further details about the genetics data.">url</a><a name="Sequence.repository.url"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#uri">uri</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">URI of the repository</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.name" title="Sequence.repository.name : URI of an external repository which contains further details about the genetics data.">name</a><a name="Sequence.repository.name"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Repository's name</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.datasetId" title="Sequence.repository.datasetId : Id of the variant in this external repository. The server will understand how to use this id to call for more info about datasets in external repository.">datasetId</a><a name="Sequence.repository.datasetId"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Id of the dataset that used to call for dataset in repository</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.variantsetId" title="Sequence.repository.variantsetId : Id of the variantset in this external repository. The server will understand how to use this id to call for more info about variantsets in external repository.">variantsetId</a><a name="Sequence.repository.variantsetId"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Id of the variantset that used to call for variantset in repository</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.repository.readsetId" title="Sequence.repository.readsetId : Id of the read in this external repository.">readsetId</a><a name="Sequence.repository.readsetId"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Id of the read</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck10.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_reference.png" alt="." style="background-color: white; background-color: inherit" title="Reference to another Resource" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.pointer" title="Sequence.pointer : Pointer to next atomic sequence which at most contains one variant.">pointer</a><a name="Sequence.pointer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="references.html#Reference">Reference</a>(<a href="sequence.html">Sequence</a>)</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Pointer to next atomic sequence<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck01.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant" title="Sequence.structureVariant : Information about chromosome structure variation.">structureVariant</a><a name="Sequence.structureVariant"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..*</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural variant<br/></td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.precision" title="Sequence.structureVariant.precision : Identify the exact boundaries of variant sequences. Each type of structure variant requires the DNA duplex to be broken and rejoined, and this creates a new sequence of bases at the rejoined sites, known as breakpoints or boundaries.">precision</a><a name="Sequence.structureVariant.precision"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#string">string</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Precision of boundaries</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio" title="Sequence.structureVariant.reportedaCGHRatio : Structural Variant reported aCGH ratio.">reportedaCGHRatio</a><a name="Sequence.structureVariant.reportedaCGHRatio"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#decimal">decimal</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant reported aCGH ratio</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.length" title="Sequence.structureVariant.length : Length of the variant choromosome.">length</a><a name="Sequence.structureVariant.length"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Length</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck011.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.outer" title="Sequence.structureVariant.outer : Structural variant outer.">outer</a><a name="Sequence.structureVariant.outer"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural variant outer</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck0110.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.outer.start" title="Sequence.structureVariant.outer.start : Structural Variant Outer Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive.">start</a><a name="Sequence.structureVariant.outer.start"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Outer Start</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck0100.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vline.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.outer.end" title="Sequence.structureVariant.outer.end : Structural Variant Outer End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position.">end</a><a name="Sequence.structureVariant.outer.end"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Outer End</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck001.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_element.gif" alt="." style="background-color: white; background-color: inherit" title="Element" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.inner" title="Sequence.structureVariant.inner : Structural variant inner.">inner</a><a name="Sequence.structureVariant.inner"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="backboneelement.html">BackboneElement</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural variant inner</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck0010.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.inner.start" title="Sequence.structureVariant.inner.start : Structural Variant Inner Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive.">start</a><a name="Sequence.structureVariant.inner.start"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Inner Start</td></tr>
<tr style="border: 0px #F0F0F0 solid; padding:0px; vertical-align: top; background-color: white;"><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px; white-space: nowrap; background-image: url(tbl_bck0000.png)" class="hierarchy"><img src="tbl_spacer.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_blank.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="tbl_vjoin_end.png" alt="." style="background-color: inherit" class="hierarchy"/><img src="icon_primitive.png" alt="." style="background-color: white; background-color: inherit" title="Primitive Data Type" class="hierarchy"/> <a href="sequence-definitions.html#Sequence.structureVariant.inner.end" title="Sequence.structureVariant.inner.end : Structural Variant Inner End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position.">end</a><a name="Sequence.structureVariant.inner.end"> </a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a style="padding-left: 3px; padding-right: 3px; color: black; background-color: white" href="elementdefinition-definitions.html#ElementDefinition.isSummary" title="This element is included in summaries">Σ</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">0..1</td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy"><a href="datatypes.html#integer">integer</a></td><td style="vertical-align: top; text-align : left; background-color: white; border: 0px #F0F0F0 solid; padding:0px 4px 0px 4px" class="hierarchy">Structural Variant Inner End</td></tr>
<tr><td colspan="5" class="hierarchy"><br/><a href="formats.html#table" title="Legend for this format"><img src="help16.png" alt="doco" style="background-color: inherit"/> Documentation for this format</a></td></tr></table>
 </div>
</div>


<div id="uml">
<a name="uml"></a>
<p><b>UML Diagram</b> (<a href="formats.html#uml">Legend</a>)</p>
 <div id="uml-inner">
 <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="1767.2" id="n206" version="1.1" height="652.0">  <defs>
    <filter x="0" width="200%" y="0" id="shadowB" height="200%">
      <feOffset result="offOut" dx="3" in="SourceGraphic" dy="3"></feOffset>
      <feColorMatrix result="matrixOut" in="offOut" values="0.2 0 0 0 0 0 0.2 0 0 0 0 0 0.2 0 0 0 0 0 1 0" type="matrix"></feColorMatrix>
      <feGaussianBlur result="blurOut" stdDeviation="2" in="matrixOut"></feGaussianBlur>
      <feBlend mode="normal" in="SourceGraphic" in2="blurOut"></feBlend>
    </filter>
  </defs><g id="n207"><rect filter="url(#shadowB)" rx="4" ry="4" x="410.40000000000003" width="198.00000000000003" y="242.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence" height="190.0"></rect><line y1="270.0" x1="410.40000000000003" y2="270.0" x2="608.4000000000001" style="stroke:dimgrey;stroke-width:1" id="n208"></line><text x="509.40000000000003" y="262.0" id="n209" fill="black" class="diagram-class-title  diagram-class-resource">Sequence<tspan class="diagram-class-title-link"> (<a id="n210" class="diagram-class-reference" xlink:href="domainresource.html">DomainResource</a>)</tspan></text><text x="416.40000000000003" y="284.0" id="n211" fill="black" class="diagram-class-detail"><a id="n212" xlink:href="sequence-definitions.html#Sequence.identifier"><title>A unique identifier for this particular sequence instance. This is a FHIR-defined id</title>identifier</a> : <a id="n213" xlink:href="datatypes.html#Identifier">Identifier</a> [0..*]</text><text x="416.40000000000003" y="298.0" id="n217" fill="black" class="diagram-class-detail"><a id="n218" xlink:href="sequence-definitions.html#Sequence.type"><title>Amino Acid Sequence/ DNA Sequence / RNA Sequence</title>type</a> : <a id="n219" xlink:href="datatypes.html#code">code</a> [0..1] « <a id="n220" xlink:href="valueset-sequence-type.html"><title id="n221">Type if a sequence -- DNA, RNA, or amino acid sequence. (Strength=Example)</title>sequenceType</a>?? »</text><text x="416.40000000000003" y="312.0" id="n227" fill="black" class="diagram-class-detail"><a id="n228" xlink:href="sequence-definitions.html#Sequence.coordinateSystem"><title>Whether the sequence is numbered starting at 0 (0-based numbering or coordinates, inclusive start, exclusive end) or starting at 1 (1-based numbering, inclusive start and inclusive end)</title>coordinateSystem</a> : <a id="n229" xlink:href="datatypes.html#integer">integer</a> [1..1]</text><text x="416.40000000000003" y="326.0" id="n233" fill="black" class="diagram-class-detail"><a id="n234" xlink:href="sequence-definitions.html#Sequence.patient"><title>The patient whose sequencing results are described by this resource</title>patient</a> : <a id="n235" xlink:href="references.html#Reference">Reference</a> [0..1] « <a id="n236" xlink:href="patient.html#Patient">Patient</a> »</text><text x="416.40000000000003" y="340.0" id="n241" fill="black" class="diagram-class-detail"><a id="n242" xlink:href="sequence-definitions.html#Sequence.specimen"><title>Specimen used for sequencing</title>specimen</a> : <a id="n243" xlink:href="references.html#Reference">Reference</a> [0..1] « <a id="n244" xlink:href="specimen.html#Specimen">Specimen</a> »</text><text x="416.40000000000003" y="354.0" id="n249" fill="black" class="diagram-class-detail"><a id="n250" xlink:href="sequence-definitions.html#Sequence.device"><title>The method for sequencing, for example, chip information</title>device</a> : <a id="n251" xlink:href="references.html#Reference">Reference</a> [0..1] « <a id="n252" xlink:href="device.html#Device">Device</a> »</text><text x="416.40000000000003" y="368.0" id="n257" fill="black" class="diagram-class-detail"><a id="n258" xlink:href="sequence-definitions.html#Sequence.performer"><title>The organization or lab that should be responsible for this result</title>performer</a> : <a id="n259" xlink:href="references.html#Reference">Reference</a> [0..1] « <a id="n260" xlink:href="organization.html#Organization">Organization</a> »</text><text x="416.40000000000003" y="382.0" id="n265" fill="black" class="diagram-class-detail"><a id="n266" xlink:href="sequence-definitions.html#Sequence.quantity"><title>The number of copies of the seqeunce of interest. (RNASeq)</title>quantity</a> : <a id="n267" xlink:href="datatypes.html#Quantity">Quantity</a> [0..1]</text><text x="416.40000000000003" y="396.0" id="n271" fill="black" class="diagram-class-detail"><a id="n272" xlink:href="sequence-definitions.html#Sequence.observedSeq"><title>Sequence that was observed. It is the result marked by referenceSeq along with variant records on referenceSeq. This shall starts from referenceSeq.windowStart and end by referenceSeq.windowEnd</title>observedSeq</a> : <a id="n273" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="416.40000000000003" y="410.0" id="n277" fill="black" class="diagram-class-detail"><a id="n278" xlink:href="sequence-definitions.html#Sequence.readCoverage"><title>Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence</title>readCoverage</a> : <a id="n279" xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="416.40000000000003" y="424.0" id="n283" fill="black" class="diagram-class-detail"><a id="n284" xlink:href="sequence-definitions.html#Sequence.pointer"><title>Pointer to next atomic sequence which at most contains one variant</title>pointer</a> : <a id="n285" xlink:href="references.html#Reference">Reference</a> [0..*] « <a id="n286" xlink:href="sequence.html#Sequence">Sequence</a> »</text></g><g id="n291"><rect filter="url(#shadowB)" rx="4" ry="4" x="386.20000000000005" width="246.40000000000003" y="0.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence.referenceSeq" height="162.0"></rect><line y1="28.0" x1="386.20000000000005" y2="28.0" x2="632.6000000000001" style="stroke:dimgrey;stroke-width:1" id="n292"></line><text x="509.4000000000001" y="20.0" id="n293" fill="black" class="diagram-class-title">ReferenceSeq</text><text x="392.20000000000005" y="42.0" id="n294" fill="black" class="diagram-class-detail"><a id="n295" xlink:href="sequence-definitions.html#Sequence.referenceSeq.chromosome"><title>Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340))</title>chromosome</a> : <a id="n296" xlink:href="datatypes.html#CodeableConcept">CodeableConcept</a> [0..1] « <a id="n297" xlink:href="valueset-chromosome-human.html"><title id="n298">Chromosome number for human. (Strength=Example)</title>chromosome-human</a>?? »</text><text x="392.20000000000005" y="56.0" id="n304" fill="black" class="diagram-class-detail"><a id="n305" xlink:href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild"><title>The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used</title>genomeBuild</a> : <a id="n306" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="392.20000000000005" y="70.0" id="n310" fill="black" class="diagram-class-detail"><a id="n311" xlink:href="sequence-definitions.html#Sequence.referenceSeq.orientation"><title>A relative reference to a DNA strand based on gene orientation. The strand that contains the open reading frame of the gene is the "sense" strand, and the opposite complementary strand is the "antisense" strand</title>orientation</a> : <a id="n312" xlink:href="datatypes.html#code">code</a> [0..1] « <a id="n313" xlink:href="valueset-orientation-type.html"><title id="n314">Type for orientation. (Strength=Required)</title>orientationType</a>! »</text><text x="392.20000000000005" y="84.0" id="n320" fill="black" class="diagram-class-detail"><a id="n321" xlink:href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId"><title>Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences</title>referenceSeqId</a> : <a id="n322" xlink:href="datatypes.html#CodeableConcept">CodeableConcept</a> [0..1] « <a id="n323" xlink:href="valueset-sequence-referenceSeq.html"><title id="n324">Reference identifier. (Strength=Example)</title>ENSEMBL</a>?? »</text><text x="392.20000000000005" y="98.0" id="n330" fill="black" class="diagram-class-detail"><a id="n331" xlink:href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer"><title>A Pointer to another Sequence entity as reference sequence</title>referenceSeqPointer</a> : <a id="n332" xlink:href="references.html#Reference">Reference</a> [0..1] « <a id="n333" xlink:href="sequence.html#Sequence">Sequence</a> »</text><text x="392.20000000000005" y="112.0" id="n338" fill="black" class="diagram-class-detail"><a id="n339" xlink:href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString"><title>A string like "ACGT"</title>referenceSeqString</a> : <a id="n340" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="392.20000000000005" y="126.0" id="n344" fill="black" class="diagram-class-detail"><a id="n345" xlink:href="sequence-definitions.html#Sequence.referenceSeq.strand"><title>An absolute reference to a strand. The Watson strand is the strand whose 5'-end is on the short arm of the chromosome, and the Crick strand as the one whose 5'-end is on the long arm</title>strand</a> : <a id="n346" xlink:href="datatypes.html#code">code</a> [0..1] « <a id="n347" xlink:href="valueset-strand-type.html"><title id="n348">Type for strand. (Strength=Required)</title>strandType</a>! »</text><text x="392.20000000000005" y="140.0" id="n354" fill="black" class="diagram-class-detail"><a id="n355" xlink:href="sequence-definitions.html#Sequence.referenceSeq.windowStart"><title>Start position of the window on the reference sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive</title>windowStart</a> : <a id="n356" xlink:href="datatypes.html#integer">integer</a> [1..1]</text><text x="392.20000000000005" y="154.0" id="n360" fill="black" class="diagram-class-detail"><a id="n361" xlink:href="sequence-definitions.html#Sequence.referenceSeq.windowEnd"><title>End position of the window on the reference sequence. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position</title>windowEnd</a> : <a id="n362" xlink:href="datatypes.html#integer">integer</a> [1..1]</text></g><g id="n366"><rect filter="url(#shadowB)" rx="4" ry="4" x="650.4000000000001" width="215.60000000000002" y="242.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence.variant" height="120.0"></rect><line y1="270.0" x1="650.4000000000001" y2="270.0" x2="866.0000000000001" style="stroke:dimgrey;stroke-width:1" id="n367"></line><text x="758.2" y="262.0" id="n368" fill="black" class="diagram-class-title">Variant</text><text x="656.4000000000001" y="284.0" id="n369" fill="black" class="diagram-class-detail"><a id="n370" xlink:href="sequence-definitions.html#Sequence.variant.start"><title>Start position of the variant on the  reference sequence.If the coordinate system is either 0-based or 1-based, then start position is inclusive</title>start</a> : <a id="n371" xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="656.4000000000001" y="298.0" id="n375" fill="black" class="diagram-class-detail"><a id="n376" xlink:href="sequence-definitions.html#Sequence.variant.end"><title>End position of the variant on the reference sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position</title>end</a> : <a id="n377" xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="656.4000000000001" y="312.0" id="n381" fill="black" class="diagram-class-detail"><a id="n382" xlink:href="sequence-definitions.html#Sequence.variant.observedAllele"><title>An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)).  Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed  sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end</title>observedAllele</a> : <a id="n383" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="656.4000000000001" y="326.0" id="n387" fill="black" class="diagram-class-detail"><a id="n388" xlink:href="sequence-definitions.html#Sequence.variant.referenceAllele"><title>An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end</title>referenceAllele</a> : <a id="n389" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="656.4000000000001" y="340.0" id="n393" fill="black" class="diagram-class-detail"><a id="n394" xlink:href="sequence-definitions.html#Sequence.variant.cigar"><title>Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm)</title>cigar</a> : <a id="n395" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="656.4000000000001" y="354.0" id="n399" fill="black" class="diagram-class-detail"><a id="n400" xlink:href="sequence-definitions.html#Sequence.variant.variantPointer"><title>A pointer to an Observation containing variant information</title>variantPointer</a> : <a id="n401" xlink:href="references.html#Reference">Reference</a> [0..1] « <a id="n402" xlink:href="observation.html#Observation">Observation</a> »</text></g><g id="n407"><rect filter="url(#shadowB)" rx="4" ry="4" x="0.0" width="290.40000000000003" y="221.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence.quality" height="232.0"></rect><line y1="249.0" x1="0.0" y2="249.0" x2="290.40000000000003" style="stroke:dimgrey;stroke-width:1" id="n408"></line><text x="145.20000000000002" y="241.0" id="n409" fill="black" class="diagram-class-title">Quality</text><text x="6.0" y="263.0" id="n410" fill="black" class="diagram-class-detail"><a id="n411" xlink:href="sequence-definitions.html#Sequence.quality.type"><title>INDEL / SNP / Undefined variant</title>type</a> : <a id="n412" xlink:href="datatypes.html#code">code</a> [1..1] « <a id="n413" xlink:href="valueset-quality-type.html"><title id="n414">Type for quality report. (Strength=Required)</title>qualityType</a>! »</text><text x="6.0" y="277.0" id="n420" fill="black" class="diagram-class-detail"><a id="n421" xlink:href="sequence-definitions.html#Sequence.quality.standardSequence"><title>Gold standard sequence used for comparing against</title>standardSequence</a> : <a id="n422" xlink:href="datatypes.html#CodeableConcept">CodeableConcept</a> [0..1] « <a id="n423" xlink:href="valueset-sequence-quality-standardSequence.html"><title id="n424">Reference identifier of the sequence that used to mark the quality of tested samples. (Strength=Example)</title>FDA-StandardSequence</a>?? »</text><text x="6.0" y="291.0" id="n430" fill="black" class="diagram-class-detail"><a id="n431" xlink:href="sequence-definitions.html#Sequence.quality.start"><title>Start position of the sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive</title>start</a> : <a id="n432" xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="6.0" y="305.0" id="n436" fill="black" class="diagram-class-detail"><a id="n437" xlink:href="sequence-definitions.html#Sequence.quality.end"><title>End position of the sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position</title>end</a> : <a id="n438" xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="6.0" y="319.0" id="n442" fill="black" class="diagram-class-detail"><a id="n443" xlink:href="sequence-definitions.html#Sequence.quality.score"><title>The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685))</title>score</a> : <a id="n444" xlink:href="datatypes.html#Quantity">Quantity</a> [0..1]</text><text x="6.0" y="333.0" id="n448" fill="black" class="diagram-class-detail"><a id="n449" xlink:href="sequence-definitions.html#Sequence.quality.method"><title>Which method is used to get sequence quality</title>method</a> : <a id="n450" xlink:href="datatypes.html#CodeableConcept">CodeableConcept</a> [0..1] « <a id="n451" xlink:href="valueset-sequence-quality-method.html"><title id="n452">The method used to evaluate the numerical quality of the observed sequence. (Strength=Example)</title>FDA-Method</a>?? »</text><text x="6.0" y="347.0" id="n458" fill="black" class="diagram-class-detail"><a id="n459" xlink:href="sequence-definitions.html#Sequence.quality.truthTP"><title>True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event</title>truthTP</a> : <a id="n460" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="361.0" id="n464" fill="black" class="diagram-class-detail"><a id="n465" xlink:href="sequence-definitions.html#Sequence.quality.queryTP"><title>True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event</title>queryTP</a> : <a id="n466" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="375.0" id="n470" fill="black" class="diagram-class-detail"><a id="n471" xlink:href="sequence-definitions.html#Sequence.quality.truthFN"><title>False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here</title>truthFN</a> : <a id="n472" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="389.0" id="n476" fill="black" class="diagram-class-detail"><a id="n477" xlink:href="sequence-definitions.html#Sequence.quality.queryFP"><title>False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here</title>queryFP</a> : <a id="n478" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="403.0" id="n482" fill="black" class="diagram-class-detail"><a id="n483" xlink:href="sequence-definitions.html#Sequence.quality.gtFP"><title>The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar)</title>gtFP</a> : <a id="n484" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="417.0" id="n488" fill="black" class="diagram-class-detail"><a id="n489" xlink:href="sequence-definitions.html#Sequence.quality.precision"><title>QUERY.TP / (QUERY.TP + QUERY.FP)</title>precision</a> : <a id="n490" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="431.0" id="n494" fill="black" class="diagram-class-detail"><a id="n495" xlink:href="sequence-definitions.html#Sequence.quality.recall"><title>TRUTH.TP / (TRUTH.TP + TRUTH.FN)</title>recall</a> : <a id="n496" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="6.0" y="445.0" id="n500" fill="black" class="diagram-class-detail"><a id="n501" xlink:href="sequence-definitions.html#Sequence.quality.fScore"><title>Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall)</title>fScore</a> : <a id="n502" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text></g><g id="n506"><rect filter="url(#shadowB)" rx="4" ry="4" x="890.4000000000001" width="123.20000000000002" y="242.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence.quality.roc" height="134.0"></rect><line y1="270.0" x1="890.4000000000001" y2="270.0" x2="1013.6000000000001" style="stroke:dimgrey;stroke-width:1" id="n507"></line><text x="952.0000000000001" y="262.0" id="n508" fill="black" class="diagram-class-title">Roc</text><text x="896.4000000000001" y="284.0" id="n509" fill="black" class="diagram-class-detail"><a id="n510" xlink:href="sequence-definitions.html#Sequence.quality.roc.score"><title>Invidual data point representing the GQ (genotype quality) score threshold</title>score</a> : <a id="n511" xlink:href="datatypes.html#integer">integer</a> [0..*]</text><text x="896.4000000000001" y="298.0" id="n515" fill="black" class="diagram-class-detail"><a id="n516" xlink:href="sequence-definitions.html#Sequence.quality.roc.numTP"><title>The number of true positives if the GQ score threshold was set to "score" field value</title>numTP</a> : <a id="n517" xlink:href="datatypes.html#integer">integer</a> [0..*]</text><text x="896.4000000000001" y="312.0" id="n521" fill="black" class="diagram-class-detail"><a id="n522" xlink:href="sequence-definitions.html#Sequence.quality.roc.numFP"><title>The number of false positives if the GQ score threshold was set to "score" field value</title>numFP</a> : <a id="n523" xlink:href="datatypes.html#integer">integer</a> [0..*]</text><text x="896.4000000000001" y="326.0" id="n527" fill="black" class="diagram-class-detail"><a id="n528" xlink:href="sequence-definitions.html#Sequence.quality.roc.numFN"><title>The number of false negatives if the GQ score threshold was set to "score" field value</title>numFN</a> : <a id="n529" xlink:href="datatypes.html#integer">integer</a> [0..*]</text><text x="896.4000000000001" y="340.0" id="n533" fill="black" class="diagram-class-detail"><a id="n534" xlink:href="sequence-definitions.html#Sequence.quality.roc.precision"><title>Calculated precision if the GQ score threshold was set to "score" field value</title>precision</a> : <a id="n535" xlink:href="datatypes.html#decimal">decimal</a> [0..*]</text><text x="896.4000000000001" y="354.0" id="n539" fill="black" class="diagram-class-detail"><a id="n540" xlink:href="sequence-definitions.html#Sequence.quality.roc.sensitivity"><title>Calculated sensitivity if the GQ score threshold was set to "score" field value</title>sensitivity</a> : <a id="n541" xlink:href="datatypes.html#decimal">decimal</a> [0..*]</text><text x="896.4000000000001" y="368.0" id="n545" fill="black" class="diagram-class-detail"><a id="n546" xlink:href="sequence-definitions.html#Sequence.quality.roc.fMeasure"><title>Calculated fScore if the GQ score threshold was set to "score" field value</title>fMeasure</a> : <a id="n547" xlink:href="datatypes.html#decimal">decimal</a> [0..*]</text></g><g id="n551"><rect filter="url(#shadowB)" rx="4" ry="4" x="428.00000000000006" width="162.8" y="512.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence.repository" height="120.0"></rect><line y1="540.0" x1="428.00000000000006" y2="540.0" x2="590.8000000000001" style="stroke:dimgrey;stroke-width:1" id="n552"></line><text x="509.4000000000001" y="532.0" id="n553" fill="black" class="diagram-class-title">Repository</text><text x="434.00000000000006" y="554.0" id="n554" fill="black" class="diagram-class-detail"><a id="n555" xlink:href="sequence-definitions.html#Sequence.repository.type"><title>Click and see / RESTful API / Need login to see / RESTful API with authentication / Other ways to see resource</title>type</a> : <a id="n556" xlink:href="datatypes.html#code">code</a> [1..1] « <a id="n557" xlink:href="valueset-repository-type.html"><title id="n558">Type for access of external URI. (Strength=Required)</title>repositoryType</a>! »</text><text x="434.00000000000006" y="568.0" id="n564" fill="black" class="diagram-class-detail"><a id="n565" xlink:href="sequence-definitions.html#Sequence.repository.url"><title>URI of an external repository which contains further details about the genetics data</title>url</a> : <a id="n566" xlink:href="datatypes.html#uri">uri</a> [0..1]</text><text x="434.00000000000006" y="582.0" id="n570" fill="black" class="diagram-class-detail"><a id="n571" xlink:href="sequence-definitions.html#Sequence.repository.name"><title>URI of an external repository which contains further details about the genetics data</title>name</a> : <a id="n572" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="434.00000000000006" y="596.0" id="n576" fill="black" class="diagram-class-detail"><a id="n577" xlink:href="sequence-definitions.html#Sequence.repository.datasetId"><title>Id of the variant in this external repository. The server will understand how to use this id to call for more info about datasets in external repository</title>datasetId</a> : <a id="n578" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="434.00000000000006" y="610.0" id="n582" fill="black" class="diagram-class-detail"><a id="n583" xlink:href="sequence-definitions.html#Sequence.repository.variantsetId"><title>Id of the variantset in this external repository. The server will understand how to use this id to call for more info about variantsets in external repository</title>variantsetId</a> : <a id="n584" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="434.00000000000006" y="624.0" id="n588" fill="black" class="diagram-class-detail"><a id="n589" xlink:href="sequence-definitions.html#Sequence.repository.readsetId"><title>Id of the read in this external repository</title>readsetId</a> : <a id="n590" xlink:href="datatypes.html#string">string</a> [0..1]</text></g><g id="n594"><rect filter="url(#shadowB)" rx="4" ry="4" x="1050.4" width="149.60000000000002" y="242.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence.structureVariant" height="78.0"></rect><line y1="270.0" x1="1050.4" y2="270.0" x2="1200.0" style="stroke:dimgrey;stroke-width:1" id="n595"></line><text x="1125.2" y="262.0" id="n596" fill="black" class="diagram-class-title">StructureVariant</text><text x="1056.4" y="284.0" id="n597" fill="black" class="diagram-class-detail"><a id="n598" xlink:href="sequence-definitions.html#Sequence.structureVariant.precision"><title>Identify the exact boundaries of variant sequences. Each type of structure variant requires the DNA duplex to be broken and rejoined, and this creates a new sequence of bases at the rejoined sites, known as breakpoints or boundaries</title>precision</a> : <a id="n599" xlink:href="datatypes.html#string">string</a> [0..1]</text><text x="1056.4" y="298.0" id="n603" fill="black" class="diagram-class-detail"><a id="n604" xlink:href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio"><title>Structural Variant reported aCGH ratio</title>reportedaCGHRatio</a> : <a id="n605" xlink:href="datatypes.html#decimal">decimal</a> [0..1]</text><text x="1056.4" y="312.0" id="n609" fill="black" class="diagram-class-detail"><a id="n610" xlink:href="sequence-definitions.html#Sequence.structureVariant.length"><title>Length of the variant choromosome</title>length</a> : <a id="n611" xlink:href="datatypes.html#integer">integer</a> [0..1]</text></g><g id="n615"><rect filter="url(#shadowB)" rx="4" ry="4" x="1210.4" width="96.80000000000001" y="242.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence.structureVariant.outer" height="64.0"></rect><line y1="270.0" x1="1210.4" y2="270.0" x2="1307.2" style="stroke:dimgrey;stroke-width:1" id="n616"></line><text x="1258.8000000000002" y="262.0" id="n617" fill="black" class="diagram-class-title">Outer</text><text x="1216.4" y="284.0" id="n618" fill="black" class="diagram-class-detail"><a id="n619" xlink:href="sequence-definitions.html#Sequence.structureVariant.outer.start"><title>Structural Variant Outer Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive</title>start</a> : <a id="n620" xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="1216.4" y="298.0" id="n624" fill="black" class="diagram-class-detail"><a id="n625" xlink:href="sequence-definitions.html#Sequence.structureVariant.outer.end"><title>Structural Variant Outer End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position</title>end</a> : <a id="n626" xlink:href="datatypes.html#integer">integer</a> [0..1]</text></g><g id="n630"><rect filter="url(#shadowB)" rx="4" ry="4" x="1370.4" width="96.80000000000001" y="242.0" style="fill:#fff9ec;stroke:black;stroke-width:1" id="Sequence.structureVariant.inner" height="64.0"></rect><line y1="270.0" x1="1370.4" y2="270.0" x2="1467.2" style="stroke:dimgrey;stroke-width:1" id="n631"></line><text x="1418.8000000000002" y="262.0" id="n632" fill="black" class="diagram-class-title">Inner</text><text x="1376.4" y="284.0" id="n633" fill="black" class="diagram-class-detail"><a id="n634" xlink:href="sequence-definitions.html#Sequence.structureVariant.inner.start"><title>Structural Variant Inner Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive</title>start</a> : <a id="n635" xlink:href="datatypes.html#integer">integer</a> [0..1]</text><text x="1376.4" y="298.0" id="n639" fill="black" class="diagram-class-detail"><a id="n640" xlink:href="sequence-definitions.html#Sequence.structureVariant.inner.end"><title>Structural Variant Inner End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position</title>end</a> : <a id="n641" xlink:href="datatypes.html#integer">integer</a> [0..1]</text></g><line y1="256.0" x1="509.40000000000003" y2="0.0" x2="509.4000000000001" style="stroke:navy;stroke-width:1" id="n645"></line><polygon transform="rotate(-89.99999999999999 509.40000000000003 256.0)" style="fill:navy;stroke:navy;stroke-width:1" id="n646" points="509.40000000000003,256.0 515.4000000000001,260.0 521.4000000000001,256.0 515.4000000000001,252.0 509.40000000000003,256.0"></polygon><rect x="483.0" width="52.0" y="121.0" style="fill:white;stroke:black;stroke-width:0" id="n647" height="18.0"></rect><text x="509.0" y="131.0" id="n648" fill="black" class="diagram-class-linkage"><a id="n649" xlink:href="sequence-definitions.html#Sequence.referenceSeq"><title>A sequence that is used as a reference to describe variants that are present in a sequence analyzed</title>referenceSeq</a></text><text x="509.4000000000001" y="-4.0" id="n650" fill="black" class="diagram-class-linkage">[0..1]</text><line y1="323.073151125402" x1="608.4000000000001" y2="317.16479099678463" x2="650.4000000000001" style="stroke:navy;stroke-width:1" id="n651"></line><polygon transform="rotate(-8.00755162235136 608.4000000000001 323.073151125402)" style="fill:navy;stroke:navy;stroke-width:1" id="n652" points="608.4000000000001,323.073151125402 614.4000000000001,327.073151125402 620.4000000000001,323.073151125402 614.4000000000001,319.073151125402 608.4000000000001,323.073151125402"></polygon><rect x="614.0" width="30.0" y="313.0" style="fill:white;stroke:black;stroke-width:0" id="n653" height="18.0"></rect><text x="629.0" y="323.0" id="n654" fill="black" class="diagram-class-linkage"><a id="n655" xlink:href="sequence-definitions.html#Sequence.variant"><title>The definition of variant here originates from Sequence ontology ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of)). This element can represent amino acid or nucleic sequence change(including insertion,deletion,SNP,etc.)  It can represent some complex mutation or segment variation with the assist of CIGAR string</title>variant</a></text><text x="630.4000000000001" y="313.16479099678463" id="n656" fill="black" class="diagram-class-linkage">[0..*]</text><line y1="331.9608329201786" x1="290.40000000000003" y2="311.1378284581062" x2="890.4000000000001" style="stroke:navy;stroke-width:1" id="n657"></line><polygon transform="rotate(-1.9876527090606158 290.40000000000003 331.9608329201786)" style="fill:navy;stroke:navy;stroke-width:1" id="n658" points="290.40000000000003,331.9608329201786 296.40000000000003,335.9608329201786 302.40000000000003,331.9608329201786 296.40000000000003,327.9608329201786 290.40000000000003,331.9608329201786"></polygon><rect x="583.5" width="13.0" y="314.0" style="fill:white;stroke:black;stroke-width:0" id="n659" height="18.0"></rect><text x="590.0" y="324.0" id="n660" fill="black" class="diagram-class-linkage"><a id="n661" xlink:href="sequence-definitions.html#Sequence.quality.roc"><title>Receiver Operator Characteristic (ROC) Curve  to give sensitivity/specificity tradeoff</title>roc</a></text><text x="870.4000000000001" y="307.1378284581062" id="n662" fill="black" class="diagram-class-linkage">[0..1]</text><line y1="337.0" x1="410.40000000000003" y2="337.0" x2="290.40000000000003" style="stroke:navy;stroke-width:1" id="n663"></line><polygon transform="rotate(180.0 410.40000000000003 337.0)" style="fill:navy;stroke:navy;stroke-width:1" id="n664" points="410.40000000000003,337.0 416.40000000000003,341.0 422.40000000000003,337.0 416.40000000000003,333.0 410.40000000000003,337.0"></polygon><rect x="335.0" width="30.0" y="330.0" style="fill:white;stroke:black;stroke-width:0" id="n665" height="18.0"></rect><text x="350.0" y="340.0" id="n666" fill="black" class="diagram-class-linkage"><a id="n667" xlink:href="sequence-definitions.html#Sequence.quality"><title>An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686))</title>quality</a></text><text x="305.40000000000003" y="333.0" id="n668" fill="black" class="diagram-class-linkage">[0..*]</text><line y1="277.0808383233534" x1="1200.0" y2="276.53592814371274" x2="1210.4" style="stroke:navy;stroke-width:1" id="n669"></line><polygon transform="rotate(-2.9992817812825256 1200.0 277.0808383233534)" style="fill:navy;stroke:navy;stroke-width:1" id="n670" points="1200.0,277.0808383233534 1206.0,281.0808383233534 1212.0,277.0808383233534 1206.0,273.0808383233534 1200.0,277.0808383233534"></polygon><rect x="1194.0" width="22.0" y="269.0" style="fill:white;stroke:black;stroke-width:0" id="n671" height="18.0"></rect><text x="1205.0" y="279.0" id="n672" fill="black" class="diagram-class-linkage"><a id="n673" xlink:href="sequence-definitions.html#Sequence.structureVariant.outer"><title>Structural variant outer</title>outer</a></text><text x="1190.4" y="272.53592814371274" id="n674" fill="black" class="diagram-class-linkage">[0..1]</text><line y1="279.21662125340606" x1="1200.0" y2="275.15395095367853" x2="1370.4" style="stroke:navy;stroke-width:1" id="n675"></line><polygon transform="rotate(-1.3657850401371643 1200.0 279.21662125340606)" style="fill:navy;stroke:navy;stroke-width:1" id="n676" points="1200.0,279.21662125340606 1206.0,283.21662125340606 1212.0,279.21662125340606 1206.0,275.21662125340606 1200.0,279.21662125340606"></polygon><rect x="1274.0" width="22.0" y="270.0" style="fill:white;stroke:black;stroke-width:0" id="n677" height="18.0"></rect><text x="1285.0" y="280.0" id="n678" fill="black" class="diagram-class-linkage"><a id="n679" xlink:href="sequence-definitions.html#Sequence.structureVariant.inner"><title>Structural variant inner</title>inner</a></text><text x="1350.4" y="271.15395095367853" id="n680" fill="black" class="diagram-class-linkage">[0..1]</text><line y1="327.9970769730432" x1="608.4000000000001" y2="287.8022085092563" x2="1050.4" style="stroke:navy;stroke-width:1" id="n681"></line><polygon transform="rotate(-5.196106724932867 608.4000000000001 327.9970769730432)" style="fill:navy;stroke:navy;stroke-width:1" id="n682" points="608.4000000000001,327.9970769730432 614.4000000000001,331.9970769730432 620.4000000000001,327.9970769730432 614.4000000000001,323.9970769730432 608.4000000000001,327.9970769730432"></polygon><rect x="794.0" width="70.0" y="300.0" style="fill:white;stroke:black;stroke-width:0" id="n683" height="18.0"></rect><text x="829.0" y="310.0" id="n684" fill="black" class="diagram-class-linkage"><a id="n685" xlink:href="sequence-definitions.html#Sequence.structureVariant"><title>Information about chromosome structure variation</title>structureVariant</a></text><text x="1030.4" y="283.8022085092563" id="n686" fill="black" class="diagram-class-linkage">[0..*]</text></svg>
 </div>
</div>

<div id="xml">
<a name="xml"></a>
<p><b>XML Template</b></p>
 <div id="xml-inner">
  <pre class="spec">
&lt;<a href="sequence-definitions.html#Sequence" title="Raw data describing a biological sequence." class="dict"><b>Sequence</b></a> xmlns="http://hl7.org/fhir"&gt; <span style="float: right"><a href="xml.html" title="Documentation for this format"><img src="help.png" alt="doco"/></a></span>
 &lt;!-- from <a href="resource.html">Resource</a>: <a href="resource.html#id">id</a>, <a href="resource.html#meta">meta</a>, <a href="resource.html#implicitRules">implicitRules</a>, and <a href="resource.html#language">language</a> --&gt;
 &lt;!-- from <a href="domainresource.html">DomainResource</a>: <a href="narrative.html#Narrative">text</a>, <a href="references.html#contained">contained</a>, <a href="extensibility.html">extension</a>, and <a href="extensibility.html#modifierExtension">modifierExtension</a> --&gt;
 &lt;<a href="sequence-definitions.html#Sequence.identifier" title="A unique identifier for this particular sequence instance. This is a FHIR-defined id." class="dict"><b>identifier</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: darkgreen"><a href="datatypes.html#Identifier">Identifier</a></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Unique ID for this particular sequence. This is a FHIR-defined id</a></span><span style="color: Gray"> --&gt;</span>&lt;/identifier&gt;
 &lt;<a href="sequence-definitions.html#Sequence.type" title="Amino Acid Sequence/ DNA Sequence / RNA Sequence." class="dict"><b>type</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="valueset-sequence-type.html">aa | dna | rna</a></span><span style="color: Gray"> --&gt;</span>
 &lt;<a href="sequence-definitions.html#Sequence.coordinateSystem" title="Whether the sequence is numbered starting at 0 (0-based numbering or coordinates, inclusive start, exclusive end) or starting at 1 (1-based numbering, inclusive start and inclusive end)." class="dict"><b>coordinateSystem</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Base number of coordinate system (0 for 0-based numbering or coordinates, inclusive start, exclusive end, 1 for 1-based numbering, inclusive start, inclusive end)</a></span><span style="color: Gray"> --&gt;</span>
 &lt;<a href="sequence-definitions.html#Sequence.patient" title="The patient whose sequencing results are described by this resource." class="dict"><b>patient</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="patient.html#Patient">Patient</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Who and/or what this is about</a></span><span style="color: Gray"> --&gt;</span>&lt;/patient&gt;
 &lt;<a href="sequence-definitions.html#Sequence.specimen" title="Specimen used for sequencing." class="dict"><b>specimen</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="specimen.html#Specimen">Specimen</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Specimen used for sequencing</a></span><span style="color: Gray"> --&gt;</span>&lt;/specimen&gt;
 &lt;<a href="sequence-definitions.html#Sequence.device" title="The method for sequencing, for example, chip information." class="dict"><b>device</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="device.html#Device">Device</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">The method for sequencing</a></span><span style="color: Gray"> --&gt;</span>&lt;/device&gt;
 &lt;<a href="sequence-definitions.html#Sequence.performer" title="The organization or lab that should be responsible for this result." class="dict"><b>performer</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="organization.html#Organization">Organization</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Who should be responsible for test result</a></span><span style="color: Gray"> --&gt;</span>&lt;/performer&gt;
 &lt;<a href="sequence-definitions.html#Sequence.quantity" title="The number of copies of the seqeunce of interest. (RNASeq)." class="dict"><b>quantity</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#Quantity">Quantity</a></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">The number of copies of the seqeunce of interest.  (RNASeq)</a></span><span style="color: Gray"> --&gt;</span>&lt;/quantity&gt;
 &lt;<a href="sequence-definitions.html#Sequence.referenceSeq" title="A sequence that is used as a reference to describe variants that are present in a sequence analyzed." class="dict"><b>referenceSeq</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..1</b></span> A sequence used as reference --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.chromosome" title="Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340))." class="dict"><b>chromosome</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> <span style="color: navy"><a style="color: navy" href="valueset-chromosome-human.html">Chromosome containing genetic finding</a></span><span style="color: Gray"> --&gt;</span>&lt;/chromosome&gt;
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild" title="The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used." class="dict"><b>genomeBuild</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.orientation" title="A relative reference to a DNA strand based on gene orientation. The strand that contains the open reading frame of the gene is the &quot;sense&quot; strand, and the opposite complementary strand is the &quot;antisense&quot; strand." class="dict"><b>orientation</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="valueset-orientation-type.html">sense | antisense</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId" title="Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences." class="dict"><b>referenceSeqId</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> <span style="color: navy"><a style="color: navy" href="valueset-sequence-referenceSeq.html">Reference identifier</a></span><span style="color: Gray"> --&gt;</span>&lt;/referenceSeqId&gt;
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer" title="A Pointer to another Sequence entity as reference sequence." class="dict"><b>referenceSeqPointer</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">A Pointer to another Sequence entity as reference sequence</a></span><span style="color: Gray"> --&gt;</span>&lt;/referenceSeqPointer&gt;
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString" title="A string like &quot;ACGT&quot;." class="dict"><b>referenceSeqString</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">A string to represent reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.strand" title="An absolute reference to a strand. The Watson strand is the strand whose 5'-end is on the short arm of the chromosome, and the Crick strand as the one whose 5'-end is on the long arm." class="dict"><b>strand</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="valueset-strand-type.html">watson | crick</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.windowStart" title="Start position of the window on the reference sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict"><b>windowStart</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Start position of the window on the  reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.referenceSeq.windowEnd" title="End position of the window on the reference sequence. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict"><b>windowEnd</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">End position of the window on the reference sequence</a></span><span style="color: Gray"> --&gt;</span>
 &lt;/referenceSeq&gt;
 &lt;<a href="sequence-definitions.html#Sequence.variant" title="The definition of variant here originates from Sequence ontology ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of)). This element can represent amino acid or nucleic sequence change(including insertion,deletion,SNP,etc.)  It can represent some complex mutation or segment variation with the assist of CIGAR string." class="dict"><b>variant</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..*</b></span> Variant in sequence --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.start" title="Start position of the variant on the  reference sequence.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict"><b>start</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Start position of the variant on the  reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.end" title="End position of the variant on the reference sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict"><b>end</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">End position of the variant on the reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.observedAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)).  Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed  sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end." class="dict"><b>observedAllele</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Allele that was observed</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.referenceAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end." class="dict"><b>referenceAllele</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Allele in the reference sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.cigar" title="Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm)." class="dict"><b>cigar</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Extended CIGAR string for aligning the sequence with reference bases</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.variant.variantPointer" title="A pointer to an Observation containing variant information." class="dict"><b>variantPointer</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="observation.html#Observation">Observation</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Pointer to observed variant information</a></span><span style="color: Gray"> --&gt;</span>&lt;/variantPointer&gt;
 &lt;/variant&gt;
 &lt;<a href="sequence-definitions.html#Sequence.observedSeq" title="Sequence that was observed. It is the result marked by referenceSeq along with variant records on referenceSeq. This shall starts from referenceSeq.windowStart and end by referenceSeq.windowEnd." class="dict"><b>observedSeq</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Sequence that was observed</a></span><span style="color: Gray"> --&gt;</span>
 &lt;<a href="sequence-definitions.html#Sequence.quality" title="An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686))." class="dict"><b>quality</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..*</b></span> An set of value as quality of sequence --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.type" title="INDEL / SNP / Undefined variant." class="dict"><b>type</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="valueset-quality-type.html">indel | snp | unknown</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.standardSequence" title="Gold standard sequence used for comparing against." class="dict"><b>standardSequence</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> <span style="color: navy"><a style="color: navy" href="valueset-sequence-quality-standardSequence.html">Standard sequence for comparison</a></span><span style="color: Gray"> --&gt;</span>&lt;/standardSequence&gt;
  &lt;<a href="sequence-definitions.html#Sequence.quality.start" title="Start position of the sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict"><b>start</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Start position of the sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.end" title="End position of the sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict"><b>end</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">End position of the sequence</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.score" title="The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685))." class="dict"><b>score</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#Quantity">Quantity</a></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Quality score for the comparison</a></span><span style="color: Gray"> --&gt;</span>&lt;/score&gt;
  &lt;<a href="sequence-definitions.html#Sequence.quality.method" title="Which method is used to get sequence quality." class="dict"><b>method</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> <span style="color: navy"><a style="color: navy" href="valueset-sequence-quality-method.html">Method to get quality</a></span><span style="color: Gray"> --&gt;</span>&lt;/method&gt;
  &lt;<a href="sequence-definitions.html#Sequence.quality.truthTP" title="True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict"><b>truthTP</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">True positives from the perspective of the truth data</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.queryTP" title="True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict"><b>queryTP</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">True positives from the perspective of the query data</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.truthFN" title="False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here." class="dict"><b>truthFN</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">False negatives</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.queryFP" title="False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here." class="dict"><b>queryFP</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">False positives</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.gtFP" title="The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar)." class="dict"><b>gtFP</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">False positives where the non-REF alleles in the Truth and Query Call Sets match</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.precision" title="QUERY.TP / (QUERY.TP + QUERY.FP)." class="dict"><b>precision</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Precision of comparison</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.recall" title="TRUTH.TP / (TRUTH.TP + TRUTH.FN)." class="dict"><b>recall</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Recall of comparison</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.fScore" title="Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall)." class="dict"><b>fScore</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">F-score</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.quality.roc" title="Receiver Operator Characteristic (ROC) Curve  to give sensitivity/specificity tradeoff." class="dict"><b>roc</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..1</b></span> Receiver Operator Characteristic (ROC) Curve --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.quality.roc.score" title="Invidual data point representing the GQ (genotype quality) score threshold." class="dict"><b>score</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Genotype quality score</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.quality.roc.numTP" title="The number of true positives if the GQ score threshold was set to &quot;score&quot; field value." class="dict"><b>numTP</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Roc score true positive numbers</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.quality.roc.numFP" title="The number of false positives if the GQ score threshold was set to &quot;score&quot; field value." class="dict"><b>numFP</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Roc score false positive numbers</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.quality.roc.numFN" title="The number of false negatives if the GQ score threshold was set to &quot;score&quot; field value." class="dict"><b>numFN</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Roc score false negative numbers</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.quality.roc.precision" title="Calculated precision if the GQ score threshold was set to &quot;score&quot; field value." class="dict"><b>precision</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Precision of the GQ score</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.quality.roc.sensitivity" title="Calculated sensitivity if the GQ score threshold was set to &quot;score&quot; field value." class="dict"><b>sensitivity</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Sensitivity of the GQ score</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.quality.roc.fMeasure" title="Calculated fScore if the GQ score threshold was set to &quot;score&quot; field value." class="dict"><b>fMeasure</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">FScore of the GQ score</a></span><span style="color: Gray"> --&gt;</span>
  &lt;/roc&gt;
 &lt;/quality&gt;
 &lt;<a href="sequence-definitions.html#Sequence.readCoverage" title="Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence." class="dict"><b>readCoverage</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Average number of reads representing a given nucleotide in the reconstructed sequence</a></span><span style="color: Gray"> --&gt;</span>
 &lt;<a href="sequence-definitions.html#Sequence.repository" title="Configurations of the external repository. The repository shall store target's observedSeq or records related with target's observedSeq." class="dict"><b>repository</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..*</b></span> External repository which contains detailed report related with observedSeq in this resource --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.type" title="Click and see / RESTful API / Need login to see / RESTful API with authentication / Other ways to see resource." class="dict"><b>type</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>1..1</b></span> <span style="color: navy"><a style="color: navy" href="valueset-repository-type.html">directlink | openapi | login | oauth | other</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.url" title="URI of an external repository which contains further details about the genetics data." class="dict"><b>url</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#uri">uri</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">URI of the repository</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.name" title="URI of an external repository which contains further details about the genetics data." class="dict"><b>name</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Repository's name</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.datasetId" title="Id of the variant in this external repository. The server will understand how to use this id to call for more info about datasets in external repository." class="dict"><b>datasetId</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Id of the dataset that used to call for dataset in repository</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.variantsetId" title="Id of the variantset in this external repository. The server will understand how to use this id to call for more info about variantsets in external repository." class="dict"><b>variantsetId</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Id of the variantset that used to call for variantset in repository</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.repository.readsetId" title="Id of the read in this external repository." class="dict"><b>readsetId</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Id of the read</a></span><span style="color: Gray"> --&gt;</span>
 &lt;/repository&gt;
 &lt;<a href="sequence-definitions.html#Sequence.pointer" title="Pointer to next atomic sequence which at most contains one variant." class="dict"><b>pointer</b></a>&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..*</b></span> <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>)</span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Pointer to next atomic sequence</a></span><span style="color: Gray"> --&gt;</span>&lt;/pointer&gt;
 &lt;<a href="sequence-definitions.html#Sequence.structureVariant" title="Information about chromosome structure variation." class="dict"><b>structureVariant</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..*</b></span> Structural variant --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.precision" title="Identify the exact boundaries of variant sequences. Each type of structure variant requires the DNA duplex to be broken and rejoined, and this creates a new sequence of bases at the rejoined sites, known as breakpoints or boundaries." class="dict"><b>precision</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Precision of boundaries</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio" title="Structural Variant reported aCGH ratio." class="dict"><b>reportedaCGHRatio</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant reported aCGH ratio</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.length" title="Length of the variant choromosome." class="dict"><b>length</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Length</a></span><span style="color: Gray"> --&gt;</span>
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.outer" title="Structural variant outer." class="dict"><b>outer</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..1</b></span> Structural variant outer --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.structureVariant.outer.start" title="Structural Variant Outer Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict"><b>start</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Outer Start</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.structureVariant.outer.end" title="Structural Variant Outer End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict"><b>end</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Outer End</a></span><span style="color: Gray"> --&gt;</span>
  &lt;/outer&gt;
  &lt;<a href="sequence-definitions.html#Sequence.structureVariant.inner" title="Structural variant inner." class="dict"><b>inner</b></a>&gt;  <span style="color: Gray">&lt;!-- <span style="color: brown"><b>0..1</b></span> Structural variant inner --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.structureVariant.inner.start" title="Structural Variant Inner Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict"><b>start</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Inner Start</a></span><span style="color: Gray"> --&gt;</span>
   &lt;<a href="sequence-definitions.html#Sequence.structureVariant.inner.end" title="Structural Variant Inner End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict"><b>end</b></a> value="[<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>]"/&gt;<span style="color: Gray">&lt;!--</span> <span style="color: brown"><b>0..1</b></span> <span style="color: navy"><a style="color: navy" href="terminologies.html#unbound">Structural Variant Inner End</a></span><span style="color: Gray"> --&gt;</span>
  &lt;/inner&gt;
 &lt;/structureVariant&gt;
&lt;/Sequence&gt;
</pre>

 </div>
</div>

<div id="json2">
<a name="json"></a>
<p><b>JSON Template</b></p>
 <div id="json-inner">
  <pre class="spec">
{<span style="float: right"><a href="json.html" title="Documentation for this format"><img src="help.png" alt="doco"/></a></span>
  "resourceType" : "<a href="sequence-definitions.html#Sequence" title="Raw data describing a biological sequence." class="dict"><b>Sequence</b></a>",
  // from <a href="resource.html">Resource</a>: <a href="resource.html#id">id</a>, <a href="resource.html#meta">meta</a>, <a href="resource.html#implicitRules">implicitRules</a>, and <a href="resource.html#language">language</a>
  // from <a href="domainresource.html">DomainResource</a>: <a href="narrative.html#Narrative">text</a>, <a href="references.html#contained">contained</a>, <a href="extensibility.html">extension</a>, and <a href="extensibility.html#modifierExtension">modifierExtension</a>
  "<a href="sequence-definitions.html#Sequence.identifier" title="A unique identifier for this particular sequence instance. This is a FHIR-defined id." class="dict">identifier</a>" : [{ <span style="color: darkgreen"><a href="datatypes.html#Identifier">Identifier</a></span> }], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Unique ID for this particular sequence. This is a FHIR-defined id</a></span>
  "<a href="sequence-definitions.html#Sequence.type" title="Amino Acid Sequence/ DNA Sequence / RNA Sequence." class="dict">type</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-sequence-type.html">aa | dna | rna</a></span>
  "<a href="sequence-definitions.html#Sequence.coordinateSystem" title="Whether the sequence is numbered starting at 0 (0-based numbering or coordinates, inclusive start, exclusive end) or starting at 1 (1-based numbering, inclusive start and inclusive end)." class="dict">coordinateSystem</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Base number of coordinate system (0 for 0-based numbering or coordinates, inclusive start, exclusive end, 1 for 1-based numbering, inclusive start, inclusive end)</a></span>
  "<a href="sequence-definitions.html#Sequence.patient" title="The patient whose sequencing results are described by this resource." class="dict">patient</a>" : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="patient.html#Patient">Patient</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Who and/or what this is about</a></span>
  "<a href="sequence-definitions.html#Sequence.specimen" title="Specimen used for sequencing." class="dict">specimen</a>" : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="specimen.html#Specimen">Specimen</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Specimen used for sequencing</a></span>
  "<a href="sequence-definitions.html#Sequence.device" title="The method for sequencing, for example, chip information." class="dict">device</a>" : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="device.html#Device">Device</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">The method for sequencing</a></span>
  "<a href="sequence-definitions.html#Sequence.performer" title="The organization or lab that should be responsible for this result." class="dict">performer</a>" : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="organization.html#Organization">Organization</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Who should be responsible for test result</a></span>
  "<a href="sequence-definitions.html#Sequence.quantity" title="The number of copies of the seqeunce of interest. (RNASeq)." class="dict">quantity</a>" : { <span style="color: darkgreen"><a href="datatypes.html#Quantity">Quantity</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">The number of copies of the seqeunce of interest.  (RNASeq)</a></span>
  "<a href="sequence-definitions.html#Sequence.referenceSeq" title="A sequence that is used as a reference to describe variants that are present in a sequence analyzed." class="dict">referenceSeq</a>" : { <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">A sequence used as reference</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.chromosome" title="Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340))." class="dict">chromosome</a>" : { <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-chromosome-human.html">Chromosome containing genetic finding</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild" title="The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used." class="dict">genomeBuild</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.orientation" title="A relative reference to a DNA strand based on gene orientation. The strand that contains the open reading frame of the gene is the &quot;sense&quot; strand, and the opposite complementary strand is the &quot;antisense&quot; strand." class="dict">orientation</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-orientation-type.html">sense | antisense</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId" title="Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences." class="dict">referenceSeqId</a>" : { <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-sequence-referenceSeq.html">Reference identifier</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer" title="A Pointer to another Sequence entity as reference sequence." class="dict">referenceSeqPointer</a>" : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>)</span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">A Pointer to another Sequence entity as reference sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString" title="A string like &quot;ACGT&quot;." class="dict">referenceSeqString</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">A string to represent reference sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.strand" title="An absolute reference to a strand. The Watson strand is the strand whose 5'-end is on the short arm of the chromosome, and the Crick strand as the one whose 5'-end is on the long arm." class="dict">strand</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-strand-type.html">watson | crick</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.windowStart" title="Start position of the window on the reference sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">windowStart</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Start position of the window on the  reference sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.referenceSeq.windowEnd" title="End position of the window on the reference sequence. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">windowEnd</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt; <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">End position of the window on the reference sequence</a></span>
  },
  "<a href="sequence-definitions.html#Sequence.variant" title="The definition of variant here originates from Sequence ontology ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of)). This element can represent amino acid or nucleic sequence change(including insertion,deletion,SNP,etc.)  It can represent some complex mutation or segment variation with the assist of CIGAR string." class="dict">variant</a>" : [{ <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Variant in sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.variant.start" title="Start position of the variant on the  reference sequence.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">start</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Start position of the variant on the  reference sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.variant.end" title="End position of the variant on the reference sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">end</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">End position of the variant on the reference sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.variant.observedAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)).  Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed  sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end." class="dict">observedAllele</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Allele that was observed</a></span>
    "<a href="sequence-definitions.html#Sequence.variant.referenceAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end." class="dict">referenceAllele</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Allele in the reference sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.variant.cigar" title="Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm)." class="dict">cigar</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Extended CIGAR string for aligning the sequence with reference bases</a></span>
    "<a href="sequence-definitions.html#Sequence.variant.variantPointer" title="A pointer to an Observation containing variant information." class="dict">variantPointer</a>" : { <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="observation.html#Observation">Observation</a>)</span> } <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Pointer to observed variant information</a></span>
  }],
  "<a href="sequence-definitions.html#Sequence.observedSeq" title="Sequence that was observed. It is the result marked by referenceSeq along with variant records on referenceSeq. This shall starts from referenceSeq.windowStart and end by referenceSeq.windowEnd." class="dict">observedSeq</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Sequence that was observed</a></span>
  "<a href="sequence-definitions.html#Sequence.quality" title="An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686))." class="dict">quality</a>" : [{ <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">An set of value as quality of sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.type" title="INDEL / SNP / Undefined variant." class="dict">type</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-quality-type.html">indel | snp | unknown</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.standardSequence" title="Gold standard sequence used for comparing against." class="dict">standardSequence</a>" : { <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-sequence-quality-standardSequence.html">Standard sequence for comparison</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.start" title="Start position of the sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">start</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Start position of the sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.end" title="End position of the sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">end</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">End position of the sequence</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.score" title="The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685))." class="dict">score</a>" : { <span style="color: darkgreen"><a href="datatypes.html#Quantity">Quantity</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Quality score for the comparison</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.method" title="Which method is used to get sequence quality." class="dict">method</a>" : { <span style="color: darkgreen"><a href="datatypes.html#CodeableConcept">CodeableConcept</a></span> }, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-sequence-quality-method.html">Method to get quality</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.truthTP" title="True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict">truthTP</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">True positives from the perspective of the truth data</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.queryTP" title="True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict">queryTP</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">True positives from the perspective of the query data</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.truthFN" title="False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here." class="dict">truthFN</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">False negatives</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.queryFP" title="False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here." class="dict">queryFP</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">False positives</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.gtFP" title="The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar)." class="dict">gtFP</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">False positives where the non-REF alleles in the Truth and Query Call Sets match</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.precision" title="QUERY.TP / (QUERY.TP + QUERY.FP)." class="dict">precision</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Precision of comparison</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.recall" title="TRUTH.TP / (TRUTH.TP + TRUTH.FN)." class="dict">recall</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Recall of comparison</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.fScore" title="Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall)." class="dict">fScore</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">F-score</a></span>
    "<a href="sequence-definitions.html#Sequence.quality.roc" title="Receiver Operator Characteristic (ROC) Curve  to give sensitivity/specificity tradeoff." class="dict">roc</a>" : { <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Receiver Operator Characteristic (ROC) Curve</a></span>
      "<a href="sequence-definitions.html#Sequence.quality.roc.score" title="Invidual data point representing the GQ (genotype quality) score threshold." class="dict">score</a>" : [&lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Genotype quality score</a></span>
      "<a href="sequence-definitions.html#Sequence.quality.roc.numTP" title="The number of true positives if the GQ score threshold was set to &quot;score&quot; field value." class="dict">numTP</a>" : [&lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Roc score true positive numbers</a></span>
      "<a href="sequence-definitions.html#Sequence.quality.roc.numFP" title="The number of false positives if the GQ score threshold was set to &quot;score&quot; field value." class="dict">numFP</a>" : [&lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Roc score false positive numbers</a></span>
      "<a href="sequence-definitions.html#Sequence.quality.roc.numFN" title="The number of false negatives if the GQ score threshold was set to &quot;score&quot; field value." class="dict">numFN</a>" : [&lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Roc score false negative numbers</a></span>
      "<a href="sequence-definitions.html#Sequence.quality.roc.precision" title="Calculated precision if the GQ score threshold was set to &quot;score&quot; field value." class="dict">precision</a>" : [&lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Precision of the GQ score</a></span>
      "<a href="sequence-definitions.html#Sequence.quality.roc.sensitivity" title="Calculated sensitivity if the GQ score threshold was set to &quot;score&quot; field value." class="dict">sensitivity</a>" : [&lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Sensitivity of the GQ score</a></span>
      "<a href="sequence-definitions.html#Sequence.quality.roc.fMeasure" title="Calculated fScore if the GQ score threshold was set to &quot;score&quot; field value." class="dict">fMeasure</a>" : [&lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;] <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">FScore of the GQ score</a></span>
    }
  }],
  "<a href="sequence-definitions.html#Sequence.readCoverage" title="Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence." class="dict">readCoverage</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Average number of reads representing a given nucleotide in the reconstructed sequence</a></span>
  "<a href="sequence-definitions.html#Sequence.repository" title="Configurations of the external repository. The repository shall store target's observedSeq or records related with target's observedSeq." class="dict">repository</a>" : [{ <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">External repository which contains detailed report related with observedSeq in this resource</a></span>
    "<a href="sequence-definitions.html#Sequence.repository.type" title="Click and see / RESTful API / Need login to see / RESTful API with authentication / Other ways to see resource." class="dict">type</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#code">code</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: brown" title="This element is required"><b>R!</b></span>  <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="valueset-repository-type.html">directlink | openapi | login | oauth | other</a></span>
    "<a href="sequence-definitions.html#Sequence.repository.url" title="URI of an external repository which contains further details about the genetics data." class="dict">url</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#uri">uri</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">URI of the repository</a></span>
    "<a href="sequence-definitions.html#Sequence.repository.name" title="URI of an external repository which contains further details about the genetics data." class="dict">name</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Repository's name</a></span>
    "<a href="sequence-definitions.html#Sequence.repository.datasetId" title="Id of the variant in this external repository. The server will understand how to use this id to call for more info about datasets in external repository." class="dict">datasetId</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Id of the dataset that used to call for dataset in repository</a></span>
    "<a href="sequence-definitions.html#Sequence.repository.variantsetId" title="Id of the variantset in this external repository. The server will understand how to use this id to call for more info about variantsets in external repository." class="dict">variantsetId</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Id of the variantset that used to call for variantset in repository</a></span>
    "<a href="sequence-definitions.html#Sequence.repository.readsetId" title="Id of the read in this external repository." class="dict">readsetId</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;" <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Id of the read</a></span>
  }],
  "<a href="sequence-definitions.html#Sequence.pointer" title="Pointer to next atomic sequence which at most contains one variant." class="dict">pointer</a>" : [{ <span style="color: darkgreen"><a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>)</span> }], <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Pointer to next atomic sequence</a></span>
  "<a href="sequence-definitions.html#Sequence.structureVariant" title="Information about chromosome structure variation." class="dict">structureVariant</a>" : [{ <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural variant</a></span>
    "<a href="sequence-definitions.html#Sequence.structureVariant.precision" title="Identify the exact boundaries of variant sequences. Each type of structure variant requires the DNA duplex to be broken and rejoined, and this creates a new sequence of bases at the rejoined sites, known as breakpoints or boundaries." class="dict">precision</a>" : "&lt;<span style="color: darkgreen"><a href="datatypes.html#string">string</a></span>&gt;", <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Precision of boundaries</a></span>
    "<a href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio" title="Structural Variant reported aCGH ratio." class="dict">reportedaCGHRatio</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#decimal">decimal</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant reported aCGH ratio</a></span>
    "<a href="sequence-definitions.html#Sequence.structureVariant.length" title="Length of the variant choromosome." class="dict">length</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Length</a></span>
    "<a href="sequence-definitions.html#Sequence.structureVariant.outer" title="Structural variant outer." class="dict">outer</a>" : { <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural variant outer</a></span>
      "<a href="sequence-definitions.html#Sequence.structureVariant.outer.start" title="Structural Variant Outer Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">start</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Outer Start</a></span>
      "<a href="sequence-definitions.html#Sequence.structureVariant.outer.end" title="Structural Variant Outer End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">end</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt; <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Outer End</a></span>
    },
    "<a href="sequence-definitions.html#Sequence.structureVariant.inner" title="Structural variant inner." class="dict">inner</a>" : { <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural variant inner</a></span>
      "<a href="sequence-definitions.html#Sequence.structureVariant.inner.start" title="Structural Variant Inner Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">start</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt;, <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Inner Start</a></span>
      "<a href="sequence-definitions.html#Sequence.structureVariant.inner.end" title="Structural Variant Inner End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">end</a>" : &lt;<span style="color: darkgreen"><a href="datatypes.html#integer">integer</a></span>&gt; <span style="color: Gray">//</span> <span style="color: navy; opacity: 0.8"><a style="color: navy; opacity: 0.8" href="terminologies.html#unbound">Structural Variant Inner End</a></span>
    }
  }]
}
</pre>

 </div>
</div>

<div id="ttl2">
<a name="ttl"></a>
<p><b>Turtle Template</b></p>
 <div id="ttl-inner">
  <pre class="spec">
@prefix fhir: &lt;http://hl7.org/fhir/&gt; .<span style="float: right"><a href="rdf.html" title="Documentation for this format"><img src="help.png" alt="doco"/></a></span>


[ a fhir:<a href="sequence-definitions.html#Sequence" title="Raw data describing a biological sequence." class="dict"><b>Sequence</b></a>;
  fhir:nodeRole fhir:treeRoot; # if this is the parser root

  # from <a href="resource.html">Resource</a>: <a href="resource.html#id">.id</a>, <a href="resource.html#meta">.meta</a>, <a href="resource.html#implicitRules">.implicitRules</a>, and <a href="resource.html#language">.language</a>
  # from <a href="domainresource.html">DomainResource</a>: <a href="narrative.html#Narrative">.text</a>, <a href="references.html#contained">.contained</a>, <a href="extensibility.html">.extension</a>, and <a href="extensibility.html#modifierExtension">.modifierExtension</a>
  fhir:<a href="sequence-definitions.html#Sequence.identifier" title="A unique identifier for this particular sequence instance. This is a FHIR-defined id." class="dict">Sequence.identifier</a><a name="ttl-identifier"> </a>[ <a href="datatypes.html#Identifier">Identifier</a> ], ... ; # 0..* <span style="color: navy">Unique ID for this particular sequence. This is a FHIR-defined id</span>
  fhir:<a href="sequence-definitions.html#Sequence.type" title="Amino Acid Sequence/ DNA Sequence / RNA Sequence." class="dict">Sequence.type</a><a name="ttl-type"> </a>[ <a href="datatypes.html#code">code</a> ]; # 0..1 <span style="color: navy">aa | dna | rna</span>
  fhir:<a href="sequence-definitions.html#Sequence.coordinateSystem" title="Whether the sequence is numbered starting at 0 (0-based numbering or coordinates, inclusive start, exclusive end) or starting at 1 (1-based numbering, inclusive start and inclusive end)." class="dict">Sequence.coordinateSystem</a><a name="ttl-coordinateSystem"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 1..1 <span style="color: navy">Base number of coordinate system (0 for 0-based numbering or coordinates, inclusive start, exclusive end, 1 for 1-based numbering, inclusive start, inclusive end)</span>
  fhir:<a href="sequence-definitions.html#Sequence.patient" title="The patient whose sequencing results are described by this resource." class="dict">Sequence.patient</a><a name="ttl-patient"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="patient.html#Patient">Patient</a>) ]; # 0..1 <span style="color: navy">Who and/or what this is about</span>
  fhir:<a href="sequence-definitions.html#Sequence.specimen" title="Specimen used for sequencing." class="dict">Sequence.specimen</a><a name="ttl-specimen"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="specimen.html#Specimen">Specimen</a>) ]; # 0..1 <span style="color: navy">Specimen used for sequencing</span>
  fhir:<a href="sequence-definitions.html#Sequence.device" title="The method for sequencing, for example, chip information." class="dict">Sequence.device</a><a name="ttl-device"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="device.html#Device">Device</a>) ]; # 0..1 <span style="color: navy">The method for sequencing</span>
  fhir:<a href="sequence-definitions.html#Sequence.performer" title="The organization or lab that should be responsible for this result." class="dict">Sequence.performer</a><a name="ttl-performer"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="organization.html#Organization">Organization</a>) ]; # 0..1 <span style="color: navy">Who should be responsible for test result</span>
  fhir:<a href="sequence-definitions.html#Sequence.quantity" title="The number of copies of the seqeunce of interest. (RNASeq)." class="dict">Sequence.quantity</a><a name="ttl-quantity"> </a>[ <a href="datatypes.html#Quantity">Quantity</a> ]; # 0..1 <span style="color: navy">The number of copies of the seqeunce of interest.  (RNASeq)</span>
  fhir:<a href="sequence-definitions.html#Sequence.referenceSeq" title="A sequence that is used as a reference to describe variants that are present in a sequence analyzed." class="dict">Sequence.referenceSeq</a><a name="ttl-referenceSeq"> </a>[ # 0..1 <span style="color: navy">A sequence used as reference</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.chromosome" title="Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication ([SO:0000340](http://www.sequenceontology.org/browser/current_svn/term/SO:0000340))." class="dict">Sequence.referenceSeq.chromosome</a><a name="ttl-chromosome"> </a>[ <a href="datatypes.html#CodeableConcept">CodeableConcept</a> ]; # 0..1 <span style="color: navy">Chromosome containing genetic finding</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.genomeBuild" title="The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used." class="dict">Sequence.referenceSeq.genomeBuild</a><a name="ttl-genomeBuild"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.orientation" title="A relative reference to a DNA strand based on gene orientation. The strand that contains the open reading frame of the gene is the &quot;sense&quot; strand, and the opposite complementary strand is the &quot;antisense&quot; strand." class="dict">Sequence.referenceSeq.orientation</a><a name="ttl-orientation"> </a>[ <a href="datatypes.html#code">code</a> ]; # 0..1 <span style="color: navy">sense | antisense</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqId" title="Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences." class="dict">Sequence.referenceSeq.referenceSeqId</a><a name="ttl-referenceSeqId"> </a>[ <a href="datatypes.html#CodeableConcept">CodeableConcept</a> ]; # 0..1 <span style="color: navy">Reference identifier</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqPointer" title="A Pointer to another Sequence entity as reference sequence." class="dict">Sequence.referenceSeq.referenceSeqPointer</a><a name="ttl-referenceSeqPointer"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>) ]; # 0..1 <span style="color: navy">A Pointer to another Sequence entity as reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.referenceSeqString" title="A string like &quot;ACGT&quot;." class="dict">Sequence.referenceSeq.referenceSeqString</a><a name="ttl-referenceSeqString"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">A string to represent reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.strand" title="An absolute reference to a strand. The Watson strand is the strand whose 5'-end is on the short arm of the chromosome, and the Crick strand as the one whose 5'-end is on the long arm." class="dict">Sequence.referenceSeq.strand</a><a name="ttl-strand"> </a>[ <a href="datatypes.html#code">code</a> ]; # 0..1 <span style="color: navy">watson | crick</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.windowStart" title="Start position of the window on the reference sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">Sequence.referenceSeq.windowStart</a><a name="ttl-windowStart"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 1..1 <span style="color: navy">Start position of the window on the  reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.referenceSeq.windowEnd" title="End position of the window on the reference sequence. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">Sequence.referenceSeq.windowEnd</a><a name="ttl-windowEnd"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 1..1 <span style="color: navy">End position of the window on the reference sequence</span>
  ];
  fhir:<a href="sequence-definitions.html#Sequence.variant" title="The definition of variant here originates from Sequence ontology ([variant_of](http://www.sequenceontology.org/browser/current_svn/term/variant_of)). This element can represent amino acid or nucleic sequence change(including insertion,deletion,SNP,etc.)  It can represent some complex mutation or segment variation with the assist of CIGAR string." class="dict">Sequence.variant</a><a name="ttl-variant"> </a>[ # 0..* <span style="color: navy">Variant in sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.start" title="Start position of the variant on the  reference sequence.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">Sequence.variant.start</a><a name="ttl-start"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Start position of the variant on the  reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.end" title="End position of the variant on the reference sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">Sequence.variant.end</a><a name="ttl-end"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">End position of the variant on the reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.observedAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)).  Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed  sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end." class="dict">Sequence.variant.observedAllele</a><a name="ttl-observedAllele"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Allele that was observed</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.referenceAllele" title="An allele is one of a set of coexisting sequence variants of a gene ([SO:0001023](http://www.sequenceontology.org/browser/current_svn/term/SO:0001023)). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end." class="dict">Sequence.variant.referenceAllele</a><a name="ttl-referenceAllele"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Allele in the reference sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.cigar" title="Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation [here](http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm)." class="dict">Sequence.variant.cigar</a><a name="ttl-cigar"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Extended CIGAR string for aligning the sequence with reference bases</span>
    fhir:<a href="sequence-definitions.html#Sequence.variant.variantPointer" title="A pointer to an Observation containing variant information." class="dict">Sequence.variant.variantPointer</a><a name="ttl-variantPointer"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="observation.html#Observation">Observation</a>) ]; # 0..1 <span style="color: navy">Pointer to observed variant information</span>
  ], ...;
  fhir:<a href="sequence-definitions.html#Sequence.observedSeq" title="Sequence that was observed. It is the result marked by referenceSeq along with variant records on referenceSeq. This shall starts from referenceSeq.windowStart and end by referenceSeq.windowEnd." class="dict">Sequence.observedSeq</a><a name="ttl-observedSeq"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Sequence that was observed</span>
  fhir:<a href="sequence-definitions.html#Sequence.quality" title="An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score ([SO:0001686](http://www.sequenceontology.org/browser/current_svn/term/SO:0001686))." class="dict">Sequence.quality</a><a name="ttl-quality"> </a>[ # 0..* <span style="color: navy">An set of value as quality of sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.type" title="INDEL / SNP / Undefined variant." class="dict">Sequence.quality.type</a><a name="ttl-type"> </a>[ <a href="datatypes.html#code">code</a> ]; # 1..1 <span style="color: navy">indel | snp | unknown</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.standardSequence" title="Gold standard sequence used for comparing against." class="dict">Sequence.quality.standardSequence</a><a name="ttl-standardSequence"> </a>[ <a href="datatypes.html#CodeableConcept">CodeableConcept</a> ]; # 0..1 <span style="color: navy">Standard sequence for comparison</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.start" title="Start position of the sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">Sequence.quality.start</a><a name="ttl-start"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Start position of the sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.end" title="End position of the sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">Sequence.quality.end</a><a name="ttl-end"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">End position of the sequence</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.score" title="The score of an experimentally derived feature such as a p-value ([SO:0001685](http://www.sequenceontology.org/browser/current_svn/term/SO:0001685))." class="dict">Sequence.quality.score</a><a name="ttl-score"> </a>[ <a href="datatypes.html#Quantity">Quantity</a> ]; # 0..1 <span style="color: navy">Quality score for the comparison</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.method" title="Which method is used to get sequence quality." class="dict">Sequence.quality.method</a><a name="ttl-method"> </a>[ <a href="datatypes.html#CodeableConcept">CodeableConcept</a> ]; # 0..1 <span style="color: navy">Method to get quality</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.truthTP" title="True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict">Sequence.quality.truthTP</a><a name="ttl-truthTP"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">True positives from the perspective of the truth data</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.queryTP" title="True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event." class="dict">Sequence.quality.queryTP</a><a name="ttl-queryTP"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">True positives from the perspective of the query data</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.truthFN" title="False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here." class="dict">Sequence.quality.truthFN</a><a name="ttl-truthFN"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">False negatives</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.queryFP" title="False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here." class="dict">Sequence.quality.queryFP</a><a name="ttl-queryFP"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">False positives</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.gtFP" title="The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar)." class="dict">Sequence.quality.gtFP</a><a name="ttl-gtFP"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">False positives where the non-REF alleles in the Truth and Query Call Sets match</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.precision" title="QUERY.TP / (QUERY.TP + QUERY.FP)." class="dict">Sequence.quality.precision</a><a name="ttl-precision"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">Precision of comparison</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.recall" title="TRUTH.TP / (TRUTH.TP + TRUTH.FN)." class="dict">Sequence.quality.recall</a><a name="ttl-recall"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">Recall of comparison</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.fScore" title="Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall)." class="dict">Sequence.quality.fScore</a><a name="ttl-fScore"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">F-score</span>
    fhir:<a href="sequence-definitions.html#Sequence.quality.roc" title="Receiver Operator Characteristic (ROC) Curve  to give sensitivity/specificity tradeoff." class="dict">Sequence.quality.roc</a><a name="ttl-roc"> </a>[ # 0..1 <span style="color: navy">Receiver Operator Characteristic (ROC) Curve</span>
      fhir:<a href="sequence-definitions.html#Sequence.quality.roc.score" title="Invidual data point representing the GQ (genotype quality) score threshold." class="dict">Sequence.quality.roc.score</a><a name="ttl-score"> </a>[ <a href="datatypes.html#integer">integer</a> ], ... ; # 0..* <span style="color: navy">Genotype quality score</span>
      fhir:<a href="sequence-definitions.html#Sequence.quality.roc.numTP" title="The number of true positives if the GQ score threshold was set to &quot;score&quot; field value." class="dict">Sequence.quality.roc.numTP</a><a name="ttl-numTP"> </a>[ <a href="datatypes.html#integer">integer</a> ], ... ; # 0..* <span style="color: navy">Roc score true positive numbers</span>
      fhir:<a href="sequence-definitions.html#Sequence.quality.roc.numFP" title="The number of false positives if the GQ score threshold was set to &quot;score&quot; field value." class="dict">Sequence.quality.roc.numFP</a><a name="ttl-numFP"> </a>[ <a href="datatypes.html#integer">integer</a> ], ... ; # 0..* <span style="color: navy">Roc score false positive numbers</span>
      fhir:<a href="sequence-definitions.html#Sequence.quality.roc.numFN" title="The number of false negatives if the GQ score threshold was set to &quot;score&quot; field value." class="dict">Sequence.quality.roc.numFN</a><a name="ttl-numFN"> </a>[ <a href="datatypes.html#integer">integer</a> ], ... ; # 0..* <span style="color: navy">Roc score false negative numbers</span>
      fhir:<a href="sequence-definitions.html#Sequence.quality.roc.precision" title="Calculated precision if the GQ score threshold was set to &quot;score&quot; field value." class="dict">Sequence.quality.roc.precision</a><a name="ttl-precision"> </a>[ <a href="datatypes.html#decimal">decimal</a> ], ... ; # 0..* <span style="color: navy">Precision of the GQ score</span>
      fhir:<a href="sequence-definitions.html#Sequence.quality.roc.sensitivity" title="Calculated sensitivity if the GQ score threshold was set to &quot;score&quot; field value." class="dict">Sequence.quality.roc.sensitivity</a><a name="ttl-sensitivity"> </a>[ <a href="datatypes.html#decimal">decimal</a> ], ... ; # 0..* <span style="color: navy">Sensitivity of the GQ score</span>
      fhir:<a href="sequence-definitions.html#Sequence.quality.roc.fMeasure" title="Calculated fScore if the GQ score threshold was set to &quot;score&quot; field value." class="dict">Sequence.quality.roc.fMeasure</a><a name="ttl-fMeasure"> </a>[ <a href="datatypes.html#decimal">decimal</a> ], ... ; # 0..* <span style="color: navy">FScore of the GQ score</span>
    ];
  ], ...;
  fhir:<a href="sequence-definitions.html#Sequence.readCoverage" title="Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence." class="dict">Sequence.readCoverage</a><a name="ttl-readCoverage"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Average number of reads representing a given nucleotide in the reconstructed sequence</span>
  fhir:<a href="sequence-definitions.html#Sequence.repository" title="Configurations of the external repository. The repository shall store target's observedSeq or records related with target's observedSeq." class="dict">Sequence.repository</a><a name="ttl-repository"> </a>[ # 0..* <span style="color: navy">External repository which contains detailed report related with observedSeq in this resource</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.type" title="Click and see / RESTful API / Need login to see / RESTful API with authentication / Other ways to see resource." class="dict">Sequence.repository.type</a><a name="ttl-type"> </a>[ <a href="datatypes.html#code">code</a> ]; # 1..1 <span style="color: navy">directlink | openapi | login | oauth | other</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.url" title="URI of an external repository which contains further details about the genetics data." class="dict">Sequence.repository.url</a><a name="ttl-url"> </a>[ <a href="datatypes.html#uri">uri</a> ]; # 0..1 <span style="color: navy">URI of the repository</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.name" title="URI of an external repository which contains further details about the genetics data." class="dict">Sequence.repository.name</a><a name="ttl-name"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Repository's name</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.datasetId" title="Id of the variant in this external repository. The server will understand how to use this id to call for more info about datasets in external repository." class="dict">Sequence.repository.datasetId</a><a name="ttl-datasetId"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Id of the dataset that used to call for dataset in repository</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.variantsetId" title="Id of the variantset in this external repository. The server will understand how to use this id to call for more info about variantsets in external repository." class="dict">Sequence.repository.variantsetId</a><a name="ttl-variantsetId"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Id of the variantset that used to call for variantset in repository</span>
    fhir:<a href="sequence-definitions.html#Sequence.repository.readsetId" title="Id of the read in this external repository." class="dict">Sequence.repository.readsetId</a><a name="ttl-readsetId"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Id of the read</span>
  ], ...;
  fhir:<a href="sequence-definitions.html#Sequence.pointer" title="Pointer to next atomic sequence which at most contains one variant." class="dict">Sequence.pointer</a><a name="ttl-pointer"> </a>[ <a href="references.html#Reference">Reference</a>(<a href="sequence.html#Sequence">Sequence</a>) ], ... ; # 0..* <span style="color: navy">Pointer to next atomic sequence</span>
  fhir:<a href="sequence-definitions.html#Sequence.structureVariant" title="Information about chromosome structure variation." class="dict">Sequence.structureVariant</a><a name="ttl-structureVariant"> </a>[ # 0..* <span style="color: navy">Structural variant</span>
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.precision" title="Identify the exact boundaries of variant sequences. Each type of structure variant requires the DNA duplex to be broken and rejoined, and this creates a new sequence of bases at the rejoined sites, known as breakpoints or boundaries." class="dict">Sequence.structureVariant.precision</a><a name="ttl-precision"> </a>[ <a href="datatypes.html#string">string</a> ]; # 0..1 <span style="color: navy">Precision of boundaries</span>
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.reportedaCGHRatio" title="Structural Variant reported aCGH ratio." class="dict">Sequence.structureVariant.reportedaCGHRatio</a><a name="ttl-reportedaCGHRatio"> </a>[ <a href="datatypes.html#decimal">decimal</a> ]; # 0..1 <span style="color: navy">Structural Variant reported aCGH ratio</span>
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.length" title="Length of the variant choromosome." class="dict">Sequence.structureVariant.length</a><a name="ttl-length"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Length</span>
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.outer" title="Structural variant outer." class="dict">Sequence.structureVariant.outer</a><a name="ttl-outer"> </a>[ # 0..1 <span style="color: navy">Structural variant outer</span>
      fhir:<a href="sequence-definitions.html#Sequence.structureVariant.outer.start" title="Structural Variant Outer Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">Sequence.structureVariant.outer.start</a><a name="ttl-start"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Outer Start</span>
      fhir:<a href="sequence-definitions.html#Sequence.structureVariant.outer.end" title="Structural Variant Outer End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">Sequence.structureVariant.outer.end</a><a name="ttl-end"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Outer End</span>
    ];
    fhir:<a href="sequence-definitions.html#Sequence.structureVariant.inner" title="Structural variant inner." class="dict">Sequence.structureVariant.inner</a><a name="ttl-inner"> </a>[ # 0..1 <span style="color: navy">Structural variant inner</span>
      fhir:<a href="sequence-definitions.html#Sequence.structureVariant.inner.start" title="Structural Variant Inner Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive." class="dict">Sequence.structureVariant.inner.start</a><a name="ttl-start"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Inner Start</span>
      fhir:<a href="sequence-definitions.html#Sequence.structureVariant.inner.end" title="Structural Variant Inner End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position." class="dict">Sequence.structureVariant.inner.end</a><a name="ttl-end"> </a>[ <a href="datatypes.html#integer">integer</a> ]; # 0..1 <span style="color: navy">Structural Variant Inner End</span>
    ];
  ], ...;
]
</pre>

 </div>
</div>

<div id="vdiff2">
<p><b>Changes since Release 3</b></p>
 <div id="diff-inner2">
  <table class="grid">
  <tr class="diff-item">
    <td class="diff-left">
      <a href="sequence.html#Sequence">Sequence</a>
    </td>
    <td class="diff-right"></td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.referenceSeq.orientation</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-entry">
    <td class="diff-left">Sequence.referenceSeq.strand</td>
    <td class="diff-right">
      <ul>
        <li>Type changed from integer to code</li>
        <li>Add Binding http://hl7.org/fhir/ValueSet/strand-type (required)</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.quality.roc</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.quality.roc.score</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.quality.roc.numTP</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.quality.roc.numFP</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.quality.roc.numFN</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.quality.roc.precision</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.quality.roc.sensitivity</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.quality.roc.fMeasure</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.precision</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.reportedaCGHRatio</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.length</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.outer</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.outer.start</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.outer.end</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.inner</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.inner.start</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
  <tr class="diff-new">
    <td class="diff-left">Sequence.structureVariant.inner.end</td>
    <td class="diff-right">
      <ul>
        <li>Added Element</li>
      </ul>
    </td>
  </tr>
</table>

<p>See the <a href="diff.html">Full Difference</a> for further information</p>

  <p>
  This analysis is available as <a href="sequence.diff.xml">XML</a> or <a href="sequence.diff.json">JSON</a>.
  </p>
  <p>See <a href="sequence-version-maps.html">R3 &lt;--&gt; R4 Conversion Maps</a> (status = 16 tests that all execute ok. <span style="background-color: #ccffcc">16 fail round-trip testing</span> and <span style="background-color: #E0B0FF">16 r3 resources are invalid (0 errors).</span>)</p>

 </div>
</div>

  </div>
</div>

<p>&nbsp;</p>
<p>See the <a href="sequence-profiles.html">Profiles &amp; Extensions</a> and the alternate definitions:
Master Definition <a href="sequence.profile.xml.html">XML</a> + <a href="sequence.profile.json.html">JSON</a>,
<a href="xml.html">XML</a> <a href="sequence.xsd">Schema</a>/<a href="sequence.sch">Schematron</a> + <a href="json.html">JSON</a> 
<a href="sequence.schema.json.html">Schema</a>, <a href="sequence.shex.html">ShEx</a> (for <a href="rdf.html">Turtle</a>)  + <a href="sequence-profiles.html">see the extensions</a> &amp; the <a href="sequence-dependencies.html">dependency analysis</a></p>

<a name="tx"></a>
<h3 class="self-link-parent"><span class="sectioncount">10.6.3.1</span> 
Terminology Bindings
 <a href="sequence.html#tx" title="link to here" class="self-link"><svg viewBox="0 0 1792 1792" width="20" class="self-link" height="20"><path d="M1520 1216q0-40-28-68l-208-208q-28-28-68-28-42 0-72 32 3 3 19 18.5t21.5 21.5 15 19 13 25.5 3.5 27.5q0 40-28 68t-68 28q-15 0-27.5-3.5t-25.5-13-19-15-21.5-21.5-18.5-19q-33 31-33 73 0 40 28 68l206 207q27 27 68 27 40 0 68-26l147-146q28-28 28-67zm-703-705q0-40-28-68l-206-207q-28-28-68-28-39 0-68 27l-147 146q-28 28-28 67 0 40 28 68l208 208q27 27 68 27 42 0 72-31-3-3-19-18.5t-21.5-21.5-15-19-13-25.5-3.5-27.5q0-40 28-68t68-28q15 0 27.5 3.5t25.5 13 19 15 21.5 21.5 18.5 19q33-31 33-73zm895 705q0 120-85 203l-147 146q-83 83-203 83-121 0-204-85l-206-207q-83-83-83-203 0-123 88-209l-88-88q-86 88-208 88-120 0-204-84l-208-208q-84-84-84-204t85-203l147-146q83-83 203-83 121 0 204 85l206 207q83 83 83 203 0 123-88 209l88 88q86-88 208-88 120 0 204 84l208 208q84 84 84 204z" fill="navy"></path></svg></a></h3>
<table class="grid">
 <tr><th>Path</th><th>Definition</th><th>Type</th><th>Reference</th></tr>
 <tr><td valign="top" title="sequenceType">Sequence.type </td><td valign="top">Type if a sequence -- DNA, RNA, or amino acid sequence.</td><td><a href="terminologies.html#example">Example</a></td><td valign="top"><a href="valueset-sequence-type.html">sequenceType</a>  <!-- b --></td> </tr>
 <tr><td valign="top" title="chromosome-human">Sequence.referenceSeq.chromosome </td><td valign="top">Chromosome number for human.</td><td><a href="terminologies.html#example">Example</a></td><td valign="top"><a href="valueset-chromosome-human.html">chromosome-human</a>  <!-- b --></td> </tr>
 <tr><td valign="top" title="orientationType">Sequence.referenceSeq.orientation </td><td valign="top">Type for orientation.</td><td><a href="terminologies.html#required">Required</a></td><td valign="top"><a href="valueset-orientation-type.html">orientationType</a>  <!-- b --></td> </tr>
 <tr><td valign="top" title="E n s e m b l">Sequence.referenceSeq.referenceSeqId </td><td valign="top">Reference identifier.</td><td><a href="terminologies.html#example">Example</a></td><td valign="top"><a href="valueset-sequence-referenceSeq.html">ENSEMBL</a>  <!-- b --></td> </tr>
 <tr><td valign="top" title="strandType">Sequence.referenceSeq.strand </td><td valign="top">Type for strand.</td><td><a href="terminologies.html#required">Required</a></td><td valign="top"><a href="valueset-strand-type.html">strandType</a>  <!-- b --></td> </tr>
 <tr><td valign="top" title="qualityType">Sequence.quality.type </td><td valign="top">Type for quality report.</td><td><a href="terminologies.html#required">Required</a></td><td valign="top"><a href="valueset-quality-type.html">qualityType</a>  <!-- b --></td> </tr>
 <tr><td valign="top" title="F d a- standard sequence">Sequence.quality.standardSequence </td><td valign="top">Reference identifier of the sequence that used to mark the quality of tested samples.</td><td><a href="terminologies.html#example">Example</a></td><td valign="top"><a href="valueset-sequence-quality-standardSequence.html">FDA-StandardSequence</a>  <!-- b --></td> </tr>
 <tr><td valign="top" title="F d a- method">Sequence.quality.method </td><td valign="top">The method used to evaluate the numerical quality of the observed sequence.</td><td><a href="terminologies.html#example">Example</a></td><td valign="top"><a href="valueset-sequence-quality-method.html">FDA-Method</a>  <!-- b --></td> </tr>
 <tr><td valign="top" title="repositoryType">Sequence.repository.type </td><td valign="top">Type for access of external URI.</td><td><a href="terminologies.html#required">Required</a></td><td valign="top"><a href="valueset-repository-type.html">repositoryType</a>  <!-- b --></td> </tr>
</table>
<p> </p>


<a name="invs"> </a>
<h3 class="self-link-parent"><span class="sectioncount">10.6.3.2</span> Constraints <a href="sequence.html#invs" title="link to here" class="self-link"><svg viewBox="0 0 1792 1792" width="20" class="self-link" height="20"><path d="M1520 1216q0-40-28-68l-208-208q-28-28-68-28-42 0-72 32 3 3 19 18.5t21.5 21.5 15 19 13 25.5 3.5 27.5q0 40-28 68t-68 28q-15 0-27.5-3.5t-25.5-13-19-15-21.5-21.5-18.5-19q-33 31-33 73 0 40 28 68l206 207q27 27 68 27 40 0 68-26l147-146q28-28 28-67zm-703-705q0-40-28-68l-206-207q-28-28-68-28-39 0-68 27l-147 146q-28 28-28 67 0 40 28 68l208 208q27 27 68 27 42 0 72-31-3-3-19-18.5t-21.5-21.5-15-19-13-25.5-3.5-27.5q0-40 28-68t68-28q15 0 27.5 3.5t25.5 13 19 15 21.5 21.5 18.5 19q33-31 33-73zm895 705q0 120-85 203l-147 146q-83 83-203 83-121 0-204-85l-206-207q-83-83-83-203 0-123 88-209l-88-88q-86 88-208 88-120 0-204-84l-208-208q-84-84-84-204t85-203l147-146q83-83 203-83 121 0 204 85l206 207q83 83 83 203 0 123-88 209l88 88q86-88 208-88 120 0 204 84l208 208q84 84 84 204z" fill="navy"></path></svg></a></h3><table class="grid"><tr><td width="60"><b>id</b></td><td><b>Level</b></td><td><b>Location</b></td><td><b>Description</b></td><td><b><a href="fhirpath.html">Expression</a></b></td></tr><tr><td><b title="Formal Invariant Identifier">seq-3</b></td><td><a style="color: Maroon" href="conformance-rules.html#rule">Rule</a> </td><td>(base)</td><td>Only 0 and 1 are valid for coordinateSystem</td><td><span style="font-family: Courier New, monospace">coordinateSystem = 1 or coordinateSystem = 0</span></td></tr><tr><td><b title="Formal Invariant Identifier">seq-5</b></td><td><a style="color: Maroon" href="conformance-rules.html#rule">Rule</a> </td><td>Sequence.referenceSeq</td><td>GenomeBuild and chromosome must be both contained if either one of them is contained</td><td><span style="font-family: Courier New, monospace">(chromosome.empty() and genomeBuild.empty()) or (chromosome.exists() and genomeBuild.exists())</span></td></tr><tr><td><b title="Formal Invariant Identifier">seq-6</b></td><td><a style="color: Maroon" href="conformance-rules.html#rule">Rule</a> </td><td>Sequence.referenceSeq</td><td>Have and only have one of the following elements in referenceSeq : 1. genomeBuild ; 2 referenceSeqId; 3. referenceSeqPointer;  4. referenceSeqString;</td><td><span style="font-family: Courier New, monospace">(genomeBuild.count()+referenceSeqId.count()+ referenceSeqPointer.count()+ referenceSeqString.count()) = 1</span></td></tr></table>

  <!-- insert xhtml content here -->

<h2 class="self-link-parent"><span class="sectioncount">10.6.4<a name="10.6.4"> </a></span> Notes: <a href="sequence.html#10.6.4" title="link to here" class="self-link"><svg viewBox="0 0 1792 1792" width="20" class="self-link" height="20"><path d="M1520 1216q0-40-28-68l-208-208q-28-28-68-28-42 0-72 32 3 3 19 18.5t21.5 21.5 15 19 13 25.5 3.5 27.5q0 40-28 68t-68 28q-15 0-27.5-3.5t-25.5-13-19-15-21.5-21.5-18.5-19q-33 31-33 73 0 40 28 68l206 207q27 27 68 27 40 0 68-26l147-146q28-28 28-67zm-703-705q0-40-28-68l-206-207q-28-28-68-28-39 0-68 27l-147 146q-28 28-28 67 0 40 28 68l208 208q27 27 68 27 42 0 72-31-3-3-19-18.5t-21.5-21.5-15-19-13-25.5-3.5-27.5q0-40 28-68t68-28q15 0 27.5 3.5t25.5 13 19 15 21.5 21.5 18.5 19q33-31 33-73zm895 705q0 120-85 203l-147 146q-83 83-203 83-121 0-204-85l-206-207q-83-83-83-203 0-123 88-209l-88-88q-86 88-208 88-120 0-204-84l-208-208q-84-84-84-204t85-203l147-146q83-83 203-83 121 0 204 85l206 207q83 83 83 203 0 123-88 209l88 88q86-88 208-88 120 0 204 84l208 208q84 84 84 204z" fill="navy"></path></svg></a></h2>

<h3 class="self-link-parent"><span class="sectioncount">10.6.4.1<a name="10.6.4.1"> </a></span> Sequence Coordinate System <a href="sequence.html#10.6.4.1" title="link to here" class="self-link"><svg viewBox="0 0 1792 1792" width="20" class="self-link" height="20"><path d="M1520 1216q0-40-28-68l-208-208q-28-28-68-28-42 0-72 32 3 3 19 18.5t21.5 21.5 15 19 13 25.5 3.5 27.5q0 40-28 68t-68 28q-15 0-27.5-3.5t-25.5-13-19-15-21.5-21.5-18.5-19q-33 31-33 73 0 40 28 68l206 207q27 27 68 27 40 0 68-26l147-146q28-28 28-67zm-703-705q0-40-28-68l-206-207q-28-28-68-28-39 0-68 27l-147 146q-28 28-28 67 0 40 28 68l208 208q27 27 68 27 42 0 72-31-3-3-19-18.5t-21.5-21.5-15-19-13-25.5-3.5-27.5q0-40 28-68t68-28q15 0 27.5 3.5t25.5 13 19 15 21.5 21.5 18.5 19q33-31 33-73zm895 705q0 120-85 203l-147 146q-83 83-203 83-121 0-204-85l-206-207q-83-83-83-203 0-123 88-209l-88-88q-86 88-208 88-120 0-204-84l-208-208q-84-84-84-204t85-203l147-146q83-83 203-83 121 0 204 85l206 207q83 83 83 203 0 123-88 209l88 88q86-88 208-88 120 0 204 84l208 208q84 84 84 204z" fill="navy"></path></svg></a></h3>
<p>
	When saving the variant information, the nucleic acid will be numbered with order. Some files are using 0-based coordiantes (e.g. BCD file format) while some files are using 1-based coordinates (e.g. VCF file format) . The element coordinateSystem in Sequence Resource contains this information.
</p>
<p>
	Sequence.coordinateSystem constraints within two possible values: 0 for 0-based system, which will mark the sequence from number 0, while 1 for 1-based system, which will begin marking the first position with number 1. The significant difference between two system is the end position. In 0-based system, the end position is <b>exclusive</b> , which means the last position will not be contained in the sequence window while in 1-based system, the end position is <b>inclusive</b> , which means the last position is included in the sequence window. Note both systems has an inclusive start position.

</p>
<p>
	For example, ACGTGCAT will be numbered from 1 to 8 in 1-based system and will be numbered from 0 to 8 in 0-based system to mark flanks (i.e. place between two Nucleotide). So the interval [3,5] in 1-based system is GTG while interval [2,5) in 0-based system is same segment GTG.
</p>

<h3 class="self-link-parent"><span class="sectioncount">10.6.4.2<a name="10.6.4.2"> </a></span> Choice of Strand  <a href="sequence.html#10.6.4.2" title="link to here" class="self-link"><svg viewBox="0 0 1792 1792" width="20" class="self-link" height="20"><path d="M1520 1216q0-40-28-68l-208-208q-28-28-68-28-42 0-72 32 3 3 19 18.5t21.5 21.5 15 19 13 25.5 3.5 27.5q0 40-28 68t-68 28q-15 0-27.5-3.5t-25.5-13-19-15-21.5-21.5-18.5-19q-33 31-33 73 0 40 28 68l206 207q27 27 68 27 40 0 68-26l147-146q28-28 28-67zm-703-705q0-40-28-68l-206-207q-28-28-68-28-39 0-68 27l-147 146q-28 28-28 67 0 40 28 68l208 208q27 27 68 27 42 0 72-31-3-3-19-18.5t-21.5-21.5-15-19-13-25.5-3.5-27.5q0-40 28-68t68-28q15 0 27.5 3.5t25.5 13 19 15 21.5 21.5 18.5 19q33-31 33-73zm895 705q0 120-85 203l-147 146q-83 83-203 83-121 0-204-85l-206-207q-83-83-83-203 0-123 88-209l-88-88q-86 88-208 88-120 0-204-84l-208-208q-84-84-84-204t85-203l147-146q83-83 203-83 121 0 204 85l206 207q83 83 83 203 0 123-88 209l88 88q86-88 208-88 120 0 204 84l208 208q84 84 84 204z" fill="navy"></path></svg></a></h3>

<p>
There are lots of definition concerning with the Directionality of DNA or RNA. Here strand element in Sequence resource is defined to have constraints with value +1 and -1 , to indicate what is the direction for nucleotide series. In order to avoid confusion, we use +1 to express the "plus" strand (5' to 3') and -1 to express the "minus" strand (3' to 5'). Here is a very simple mapping that indicates which number the expression will be represented.
  <table border="1">
    <tr>
      <td><b>+1</b></td>
      <td><b>-1</b></td> 
    </tr>
    <tr>
      <td>Watson</td>
      <td>Crick</td>
    </tr>
    <tr>
      <td>Sense</td>
      <td>Antisense</td>
    </tr>
    <tr>
      <td>positive</td>
      <td>negative</td>
    </tr>
  </table>

</p>

<p>
Here is a small-scale example: if 5' GCGATATCGCAAA 3' is the data, then GC..AAA is plus strand while AAA..CG is the minus strand. 

</p>

<h3 class="self-link-parent"><span class="sectioncount">10.6.4.3<a name="10.6.4.3"> </a></span>  String usage for Reference Sequence and Observed Sequence  <a href="sequence.html#10.6.4.3" title="link to here" class="self-link"><svg viewBox="0 0 1792 1792" width="20" class="self-link" height="20"><path d="M1520 1216q0-40-28-68l-208-208q-28-28-68-28-42 0-72 32 3 3 19 18.5t21.5 21.5 15 19 13 25.5 3.5 27.5q0 40-28 68t-68 28q-15 0-27.5-3.5t-25.5-13-19-15-21.5-21.5-18.5-19q-33 31-33 73 0 40 28 68l206 207q27 27 68 27 40 0 68-26l147-146q28-28 28-67zm-703-705q0-40-28-68l-206-207q-28-28-68-28-39 0-68 27l-147 146q-28 28-28 67 0 40 28 68l208 208q27 27 68 27 42 0 72-31-3-3-19-18.5t-21.5-21.5-15-19-13-25.5-3.5-27.5q0-40 28-68t68-28q15 0 27.5 3.5t25.5 13 19 15 21.5 21.5 18.5 19q33-31 33-73zm895 705q0 120-85 203l-147 146q-83 83-203 83-121 0-204-85l-206-207q-83-83-83-203 0-123 88-209l-88-88q-86 88-208 88-120 0-204-84l-208-208q-84-84-84-204t85-203l147-146q83-83 203-83 121 0 204 85l206 207q83 83 83 203 0 123-88 209l88 88q86-88 208-88 120 0 204 84l208 208q84 84 84 204z" fill="navy"></path></svg></a></h3>
<p>
We hope that string of <b>observedSeq</b> can be constrained more than just any normal string but with notation tables. Here we present what the nucleotide acid stirng should be constrianed within the range:

  <table>
    <tr>
      <td>A --&gt; adenosine </td>
      <td>M --&gt; A C (amino)</td>
      <td>U --&gt; uridine </td>
      <td>H --&gt; A C T</td>
      <td>V --&gt; G C A</td>
    </tr>
    <tr>
      <td>C --&gt; cytidine</td> 
      <td>S --&gt; G C (strong)</td>
      <td>D --&gt; G A T</td>
    <td>K --&gt; G T (keto)</td>
    </tr>
    <tr>
      <td>G --&gt; guanine </td>
      <td>W --&gt; A T (weak)</td>
      <td>R --&gt; G A (purine)</td> 
      <td>N --&gt; A G C T (any)</td>
    </tr>
    <tr>
      <td>T --&gt; thymidine</td>
      <td>B --&gt; G T C</td>
      <td>Y --&gt; T C (pyrimidine)</td> 
      <td>- --&gt; gap of indeterminate length  </td>
    </tr>
  </table>

while the amino acid string should be constrained within the range:

  <table>
    <tr>
      <td>A alanine </td>
      <td>P proline</td>
      <td>B aspartate or asparagine</td>
      <td>Q glutamine</td>
    </tr>
    <tr>
      <td>C cystine </td>
      <td>R arginine</td>
      <td>D aspartate</td>
      <td>S serine</td>
    </tr>
    <tr>
      <td>E glutamate </td>
      <td>T threonine</td>
      <td>F phenylalanine </td>
      <td>U selenocysteine</td>
    </tr>
    <tr>
      <td>G glycine </td>
      <td>V valine</td>
      <td>H histidine </td>
      <td>W tryptophan </td>
    </tr>
    <tr>
      <td>I isoleucine </td>
      <td>Y tyrosine </td>
      <td>K lysine </td>
      <td>Z glutamate or glutamine</td>
    </tr>
    <tr>
      <td>L leucine</td>
      <td> X any </td>
      <td>M methionine </td>
      <td>* translation stop </td>
    </tr>
    <tr>
      <td>N asparagine </td>
      <td>- gap of indeterminate length </td>
    </tr>
  </table>

</p>

<a name="search"></a>
<h2 class="self-link-parent"><span class="sectioncount">10.6.5</span> Search Parameters <a href="sequence.html#search" title="link to here" class="self-link"><svg viewBox="0 0 1792 1792" width="20" class="self-link" height="20"><path d="M1520 1216q0-40-28-68l-208-208q-28-28-68-28-42 0-72 32 3 3 19 18.5t21.5 21.5 15 19 13 25.5 3.5 27.5q0 40-28 68t-68 28q-15 0-27.5-3.5t-25.5-13-19-15-21.5-21.5-18.5-19q-33 31-33 73 0 40 28 68l206 207q27 27 68 27 40 0 68-26l147-146q28-28 28-67zm-703-705q0-40-28-68l-206-207q-28-28-68-28-39 0-68 27l-147 146q-28 28-28 67 0 40 28 68l208 208q27 27 68 27 42 0 72-31-3-3-19-18.5t-21.5-21.5-15-19-13-25.5-3.5-27.5q0-40 28-68t68-28q15 0 27.5 3.5t25.5 13 19 15 21.5 21.5 18.5 19q33-31 33-73zm895 705q0 120-85 203l-147 146q-83 83-203 83-121 0-204-85l-206-207q-83-83-83-203 0-123 88-209l-88-88q-86 88-208 88-120 0-204-84l-208-208q-84-84-84-204t85-203l147-146q83-83 203-83 121 0 204 85l206 207q83 83 83 203 0 123-88 209l88 88q86-88 208-88 120 0 204 84l208 208q84 84 84 204z" fill="navy"></path></svg></a></h2>
<p>Search parameters for this resource. The <a href="search.html#all">common parameters</a> also apply. See <a href="search.html">Searching</a> for more information about searching in REST, messaging, and services.</p>
<table class="list">
<tr><td><b>Name</b></td><td><b>Type</b></td><td><b>Description</b></td><td><b>Expression</b></td><td><b>In Common</b></td></tr>
<tr><td><a name="sp-chromosome"> </a>chromosome</td><td><a href="search.html#token">token</a></td><td>Chromosome number of the reference sequence</td><td>Sequence.referenceSeq.chromosome</td><td></td></tr>
<tr><td><a name="sp-coordinate"> </a>coordinate</td><td><a href="search.html#composite">composite</a></td><td>Search parameter for region of the reference DNA sequence string. This will refer to part of a locus or part of a gene where search region will be represented in 1-based system. Since the coordinateSystem can either be 0-based or 1-based, this search query will include the result of both coordinateSystem that contains the equivalent segment of the gene or whole genome sequence. For example, a search for sequence can be represented as `coordinate=1$lt345$gt123`, this means it will search for the Sequence resource on chromosome 1 and with position &gt;123 and &lt;345, where in 1-based system resource, all strings within region 1:124-344 will be revealed, while in 0-based system resource, all strings within region 1:123-344 will be revealed. You may want to check detail about 0-based v.s. 1-based above.</td><td>On Sequence.variant:<br/>&nbsp;&nbsp;chromosome: %resource.referenceSeq.chromosome<br/>&nbsp;&nbsp;start: start<br/>&nbsp;&nbsp;end: end</td><td></td></tr>
<tr><td><a name="sp-end"> </a>end</td><td><a href="search.html#number">number</a></td><td>End position (0-based exclusive, which menas the acid at this position will not be included, 1-based inclusive, which means the acid at this position will be included) of the reference sequence.</td><td>Sequence.referenceSeq.windowEnd</td><td></td></tr>
<tr><td><a name="sp-identifier"> </a>identifier</td><td><a href="search.html#token">token</a></td><td>The unique identity for a particular sequence</td><td>Sequence.identifier</td><td></td></tr>
<tr><td><a name="sp-patient"> </a>patient</td><td><a href="search.html#reference">reference</a></td><td>The subject that the observation is about</td><td>Sequence.patient<br/>(<a href="patient.html">Patient</a>)</td><td></td></tr>
<tr><td><a name="sp-start"> </a>start</td><td><a href="search.html#number">number</a></td><td>Start position (0-based inclusive, 1-based inclusive, that means the nucleic acid or amino acid at this position will be included) of the reference sequence.</td><td>Sequence.referenceSeq.windowStart</td><td></td></tr>
<tr><td><a name="sp-type"> </a>type</td><td><a href="search.html#token">token</a></td><td>Amino Acid Sequence/ DNA Sequence / RNA Sequence</td><td>Sequence.type</td><td></td></tr>
</table>



</div>


				</div>  <!-- /inner-wrapper -->
            </div>  <!-- /row -->
        </div>  <!-- /container -->
        
    </div>  <!-- /segment-content -->


	<div id="segment-footer" class="segment">  <!-- segment-footer -->
		<div class="container">  <!-- container -->
			<div class="inner-wrapper">
				<p>
        &reg;&copy; HL7.org 2011+. FHIR Release 4 Candidate (v3.5.0-????) generated on Fri, Sep 14, 2018 07:28-0500. <a style="color: #ffffff" href="qa.html">QA Page</a>
        <br/>
        <span style="color: #FFFF77">
        Links: <a style="color: #b8dcf9" href="http://hl7.org/fhir/search.cfm">Search <img src="external.png" style="text-align: baseline"/></a> | 
               <a style="color: #b8dcf9" href="history.html">Version History</a> | 
               <a style="color: #b8dcf9" href="toc.html">Table of Contents</a> | 
               <a style="color: #b8dcf9" href="credits.html">Credits</a> | 
               <a style="color: #b8dcf9" href="http://services.w3.org/htmldiff?doc1=http%3A%2F%2Fhl7.org%2Ffhir%2FSTU3%2Fsequence.html&amp;doc2=http%3A%2F%2Fbuild.fhir.org%2Fsequence.html">Compare to DSTU3 <img src="external.png" style="text-align: baseline"/></a> |                
               <a style="color: #b8dcf9" href="http://services.w3.org/htmldiff?doc1=http%3A%2F%2Fhl7.org%2Ffhir%2F2018May%2Fsequence.html&amp;doc2=http%3A%2F%2Fbuild.fhir.org%2Fsequence.html">Ballot Comparison <img src="external.png" style="text-align: baseline"/></a> |                
               <a rel="license" style="color: #b8dcf9" href="license.html"><img src="cc0.png" alt="CC0" style="border-style: none;"/></a> | 
               <a style="color: #b8dcf9" href="http://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemAdd&amp;tracker_id=677" target="_blank">Propose a change <img src="external.png" style="text-align: baseline"/></a>   
        </span>
        </p>
			</div>  <!-- /inner-wrapper -->
		</div>  <!-- /container -->
	</div>  <!-- /segment-footer -->
  <!-- disqus thread -->
  <!-- disqus -->
  <!-- end disqus -->        
	
	<div id="segment-post-footer" class="segment hidden">  <!-- segment-post-footer -->
		<div class="container">  <!-- container -->
		</div>  <!-- /container -->
	</div>  <!-- /segment-post-footer -->
    
      <!-- JS and analytics only. -->
      <!-- Bootstrap core JavaScript
================================================== -->
  <!-- Placed at the end of the document so the pages load faster -->
<script src="./assets/js/jquery.js"> </script>     <!-- note keep space here, otherwise it will be transformed to empty tag -> fails -->
<script src="./dist/js/bootstrap.min.js"> </script>
<script src="./assets/js/respond.min.js"> </script>

<script src="./assets/js/fhir.js"> </script>

  <!-- Analytics Below
================================================== -->




<script src="external/jquery/jquery.js"> </script>
<script src="jquery-ui.min.js"> </script>
<script>
try {
  var currentTabIndex = sessionStorage.getItem('fhir-resource-tab-index');
}
catch(exception){
}

if (!currentTabIndex)
  currentTabIndex = '0';

$( '#tabs' ).tabs({
         active: currentTabIndex,
         activate: function( event, ui ) {
             var active = $('.selector').tabs('option', 'active');
             currentTabIndex = ui.newTab.index();
             document.activeElement.blur();
             try {
               sessionStorage.setItem('fhir-resource-tab-index', currentTabIndex);
             }
             catch(exception){
             }
         }
     });
</script>

</body>
</html>
