<!DOCTYPE HTML>
  <!-- template-definitions.html --><html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>Sequence - FHIR v3.5.0</title>

  
  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
  <meta name="author" content="http://hl7.org/fhir"/>

  <link rel="stylesheet" href="fhir.css"/>
  <link rel="Prev" href="http://hl7.org/fhir/sequence-definitions.html"/>

    <!-- Bootstrap core CSS -->
  <link rel="stylesheet" href="./dist/css/bootstrap.css"/>
  <link rel="stylesheet" href="./assets/css/bootstrap-fhir.css"/>

    <!-- Project extras -->
  <link rel="stylesheet" href="./assets/css/project.css"/>
  <link rel="stylesheet" href="./assets/css/pygments-manni.css"/>
	<link rel="stylesheet" href="jquery-ui.css"/>

    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- [if lt IE 9]>
  <script src="./assets/js/html5shiv.js"></script>
  <script src="./assets/js/respond.min.js"></script>
  <![endif] -->

    <!-- Favicons -->
  <link sizes="144x144" rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-144-precomposed.png"/>
  <link sizes="114x114" rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-114-precomposed.png"/>
  <link sizes="72x72" rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-72-precomposed.png"/>
  <link rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-57-precomposed.png"/>
  <link rel="shortcut icon" href="./assets/ico/favicon.png"/>

</head>
<body>
	<div id="segment-header" class="segment">  <!-- segment-header -->	
		<div class="container">  <!-- container -->
			<a no-external="true" id="logo" href="http://hl7.org/fhir"><img src="./assets/images/fhir-logo-www.png" alt="logo fhir"/> </a>
			<div id="hl7-status">
				<b>Custom NCATS Translator JHU Build</b><br /> <a href="https://ncats.nih.gov"><img src="https://ncats.nih.gov/sites/all/themes/ncats-2014/images/assets/ncats-logo.png" height="50" /></a> <a href="https://www.hopkinsmedicine.org/"><img src="/jhulogo.png" height="100" /></a>
			</div>       	
			
			<div id="hl7-nav">
				 <a no-external="true" id="hl7-logo" href="http://www.hl7.org">
					<img src="./assets/images/hl7-logo.png" width="42" alt="visit the hl7 website" height="50"/>
				</a> 
			</div>       	
			<div id="hl7-nav"><a no-external="true" id="hl7-logo" href="http://hl7.org/fhir/search.cfm"><img src="./assets/images/search.png" alt="Search FHIR"/></a></div> 
		</div>
		<div class="container">  <!-- container -->
	</div></div>  <!-- /segment-header -->	


	<div id="segment-navbar" class="segment">  <!-- segment-navbar -->
		<div id="stripe"> </div>
		<div class="container">  <!-- container -->
   <!-- HEADER CONTENT -->
 	      		 	      		      		
			<nav class="navbar navbar-inverse">
				<div class="container">
					<button data-target=".navbar-inverse-collapse" data-toggle="collapse" type="button" class="navbar-toggle">
						<span class="icon-bar"> </span>
						<span class="icon-bar"> </span>
						<span class="icon-bar"> </span>
					</button>
					<a href="index.html" class="navbar-brand hidden">FHIR</a>
					<div class="nav-collapse collapse navbar-inverse-collapse">
						<ul class="nav navbar-nav">
              <li><a href="./index.html">Home</a></li>
              <li><a href="./modules.html">Getting Started</a></li>
              <li><a href="./documentation.html">Documentation</a></li>
              <li><a href="./resourcelist.html">Resources</a></li>
              <li><a href="./profilelist.html">Profiles</a></li>
              <li><a href="./extensibility-registry.html">Extensions</a></li>
              <li><a href="./operationslist.html">Operations</a></li>
              <li><a href="./terminologies-systems.html">Terminologies</a></li>
						</ul>
					</div>  <!-- /.nav-collapse -->
				</div>  <!-- /.container -->
			</nav>  <!-- /.navbar -->
      		
  <!-- /HEADER CONTENT -->				
		</div>  <!-- /container -->
	</div>  <!-- /segment-navbar -->
	

	<div id="segment-breadcrumb" class="segment">  <!-- segment-breadcrumb -->
		<div class="container">  <!-- container -->	
			<ul class="breadcrumb">
        <li><a href="diagnostics-module.html"><img src="diagnostics.jpg"/> Diagnostics</a></li>        <li><a href="sequence.html">Sequence</a></li>        <li><b>Detailed Descriptions</b></li>          <!-- sequence.html / res-Detailed Descriptions / Sequence -->

			</ul>	
		</div>  <!-- /container -->
	</div>  <!-- /segment-breadcrumb -->
	

	<div id="segment-content" class="segment">  <!-- segment-content -->
	<div class="container">  <!-- container -->
            <div class="row">
            	<div class="inner-wrapper">
  <!-- CONTENT CONTENT -->


<div class="col-12">

<ul class="nav nav-tabs"><li><a href="sequence.html">Content</a></li><li><a href="sequence-examples.html">Examples</a></li><li class="active"><a href="#">Detailed Descriptions</a></li><li><a href="sequence-mappings.html">Mappings</a></li><li><a href="sequence-profiles.html">Profiles &amp; Extensions</a></li><li><a href="sequence-version-maps.html">R3 Conversions</a></li></ul>

<h2 class="self-link-parent"><span class="sectioncount">10.6.7<a name="10.6.7"> </a></span> Resource Sequence - Detailed Descriptions <a href="sequence-definitions.html#10.6.7" title="link to here" class="self-link"><svg viewBox="0 0 1792 1792" width="20" class="self-link" height="20"><path d="M1520 1216q0-40-28-68l-208-208q-28-28-68-28-42 0-72 32 3 3 19 18.5t21.5 21.5 15 19 13 25.5 3.5 27.5q0 40-28 68t-68 28q-15 0-27.5-3.5t-25.5-13-19-15-21.5-21.5-18.5-19q-33 31-33 73 0 40 28 68l206 207q27 27 68 27 40 0 68-26l147-146q28-28 28-67zm-703-705q0-40-28-68l-206-207q-28-28-68-28-39 0-68 27l-147 146q-28 28-28 67 0 40 28 68l208 208q27 27 68 27 42 0 72-31-3-3-19-18.5t-21.5-21.5-15-19-13-25.5-3.5-27.5q0-40 28-68t68-28q15 0 27.5 3.5t25.5 13 19 15 21.5 21.5 18.5 19q33-31 33-73zm895 705q0 120-85 203l-147 146q-83 83-203 83-121 0-204-85l-206-207q-83-83-83-203 0-123 88-209l-88-88q-86 88-208 88-120 0-204-84l-208-208q-84-84-84-204t85-203l147-146q83-83 203-83 121 0 204 85l206 207q83 83 83 203 0 123-88 209l88 88q86-88 208-88 120 0 204 84l208 208q84 84 84 204z" fill="navy"></path></svg></a></h2>
<table class="colstu"><tr><td><a href="http://www.hl7.org/Special/committees/clingenomics/index.cfm">Clinical Genomics <img src="external.png" style="text-align: baseline"/></a> Work Group</td><td><a href="versions.html#maturity">Maturity Level</a>: 1</td><td>&nbsp;<a href="versions.html#std-process" title="Maturity Level">Trial Use</a></td>  <!-- !ns! --><td><a href="compartmentdefinition.html">Compartments</a>: Not linked to any defined compartments</td></tr></table>


<p>Detailed Descriptions for the elements in the Sequence resource.</p>

<table class="dict">
  <tr><td colspan="2" class="structure"><a name="Sequence"> </a><b>Sequence</b></td></tr>
  <tr><td>Element Id</td><td>Sequence</td></tr>
  <tr><td>Definition</td><td><p>Raw data describing a biological sequence.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>1..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="domainresource.html">DomainResource</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td>Invariants</td><td><table class="dict">
<tr><td colspan="4"><b>Defined on this element</b></td></tr>
<tr><td width="60px"><b title="Formal Invariant Identifier">seq-3</b></td><td><a style="color: Maroon" href="conformance-rules.html#rule">Rule</a> </td><td>Only 0 and 1 are valid for coordinateSystem</td><td><span style="font-family: Courier New, monospace">coordinateSystem = 1 or coordinateSystem = 0</span></td></tr></table>
</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.identifier"> </a><b>Sequence.identifier</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.identifier</td></tr>
  <tr><td>Definition</td><td><p>A unique identifier for this particular sequence instance. This is a FHIR-defined id.</p>
</td></tr>
  <tr><td>Note</td><td>This is a business identifer, not a resource identifier (see <a href="resource.html#identifiers">discussion</a>)</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#Identifier">Identifier</a></td></tr>
  <tr><td>Requirements</td><td><p>Allows sequences to be distinguished and referenced.</p>
</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.type"> </a><b>Sequence.type</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.type</td></tr>
  <tr><td>Definition</td><td><p>Amino Acid Sequence/ DNA Sequence / RNA Sequence.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="terminologies.html">Terminology Binding</a></td><td><a href="valueset-sequence-type.html">sequenceType</a> (<a href="terminologies.html#example">Example</a>)</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#code">code</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.coordinateSystem"> </a><b>Sequence.coordinateSystem</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.coordinateSystem</td></tr>
  <tr><td>Definition</td><td><p>Whether the sequence is numbered starting at 0 (0-based numbering or coordinates, inclusive start, exclusive end) or starting at 1 (1-based numbering, inclusive start and inclusive end).</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>1..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.patient"> </a><b>Sequence.patient</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.patient</td></tr>
  <tr><td>Definition</td><td><p>The patient whose sequencing results are described by this resource.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="references.html#Reference">Reference</a>(<a href="patient.html">Patient</a>)</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.specimen"> </a><b>Sequence.specimen</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.specimen</td></tr>
  <tr><td>Definition</td><td><p>Specimen used for sequencing.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="references.html#Reference">Reference</a>(<a href="specimen.html">Specimen</a>)</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.device"> </a><b>Sequence.device</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.device</td></tr>
  <tr><td>Definition</td><td><p>The method for sequencing, for example, chip information.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="references.html#Reference">Reference</a>(<a href="device.html">Device</a>)</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.performer"> </a><b>Sequence.performer</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.performer</td></tr>
  <tr><td>Definition</td><td><p>The organization or lab that should be responsible for this result.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="references.html#Reference">Reference</a>(<a href="organization.html">Organization</a>)</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quantity"> </a><b>Sequence.quantity</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quantity</td></tr>
  <tr><td>Definition</td><td><p>The number of copies of the seqeunce of interest. (RNASeq).</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#Quantity">Quantity</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq"> </a><b>Sequence.referenceSeq</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.referenceSeq</td></tr>
  <tr><td>Definition</td><td><p>A sequence that is used as a reference to describe variants that are present in a sequence analyzed.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td>Invariants</td><td><table class="dict">
<tr><td colspan="4"><b>Defined on this element</b></td></tr>
<tr><td width="60px"><b title="Formal Invariant Identifier">seq-5</b></td><td><a style="color: Maroon" href="conformance-rules.html#rule">Rule</a> </td><td>GenomeBuild and chromosome must be both contained if either one of them is contained</td><td><span style="font-family: Courier New, monospace">(chromosome.empty() and genomeBuild.empty()) or (chromosome.exists() and genomeBuild.exists())</span></td></tr><tr><td width="60px"><b title="Formal Invariant Identifier">seq-6</b></td><td><a style="color: Maroon" href="conformance-rules.html#rule">Rule</a> </td><td>Have and only have one of the following elements in referenceSeq : 1. genomeBuild ; 2 referenceSeqId; 3. referenceSeqPointer;  4. referenceSeqString;</td><td><span style="font-family: Courier New, monospace">(genomeBuild.count()+referenceSeqId.count()+ referenceSeqPointer.count()+ referenceSeqString.count()) = 1</span></td></tr></table>
</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq.chromosome"> </a><b>Sequence.referenceSeq.chromosome</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.referenceSeq.chromosome</td></tr>
  <tr><td>Definition</td><td><p>Structural unit composed of a nucleic acid molecule which controls its own replication through the interaction of specific proteins at one or more origins of replication (<a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0000340">SO:0000340 <img src="external.png" style="text-align: baseline"/></a>).</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="terminologies.html">Terminology Binding</a></td><td><a href="valueset-chromosome-human.html">chromosome-human</a> (<a href="terminologies.html#example">Example</a>)</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq.genomeBuild"> </a><b>Sequence.referenceSeq.genomeBuild</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.referenceSeq.genomeBuild</td></tr>
  <tr><td>Definition</td><td><p>The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq.orientation"> </a><b>Sequence.referenceSeq.orientation</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.referenceSeq.orientation</td></tr>
  <tr><td>Definition</td><td><p>A relative reference to a DNA strand based on gene orientation. The strand that contains the open reading frame of the gene is the "sense" strand, and the opposite complementary strand is the "antisense" strand.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="terminologies.html">Terminology Binding</a></td><td><a href="valueset-orientation-type.html">orientationType</a> (<a href="terminologies.html#required">Required</a>)</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#code">code</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq.referenceSeqId"> </a><b>Sequence.referenceSeq.referenceSeqId</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.referenceSeq.referenceSeqId</td></tr>
  <tr><td>Definition</td><td><p>Reference identifier of reference sequence submitted to NCBI. It must match the type in the Sequence.type field. For example, the prefix, “NG_” identifies reference sequence for genes, “NM_” for messenger RNA transcripts, and “NP_” for amino acid sequences.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="terminologies.html">Terminology Binding</a></td><td><a href="valueset-sequence-referenceSeq.html">E n s e m b l</a> (<a href="terminologies.html#example">Example</a>)</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq.referenceSeqPointer"> </a><b>Sequence.referenceSeq.referenceSeqPointer</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.referenceSeq.referenceSeqPointer</td></tr>
  <tr><td>Definition</td><td><p>A Pointer to another Sequence entity as reference sequence.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="references.html#Reference">Reference</a>(<a href="sequence.html">Sequence</a>)</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq.referenceSeqString"> </a><b>Sequence.referenceSeq.referenceSeqString</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.referenceSeq.referenceSeqString</td></tr>
  <tr><td>Definition</td><td><p>A string like "ACGT".</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq.strand"> </a><b>Sequence.referenceSeq.strand</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.referenceSeq.strand</td></tr>
  <tr><td>Definition</td><td><p>An absolute reference to a strand. The Watson strand is the strand whose 5'-end is on the short arm of the chromosome, and the Crick strand as the one whose 5'-end is on the long arm.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="terminologies.html">Terminology Binding</a></td><td><a href="valueset-strand-type.html">strandType</a> (<a href="terminologies.html#required">Required</a>)</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#code">code</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq.windowStart"> </a><b>Sequence.referenceSeq.windowStart</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.referenceSeq.windowStart</td></tr>
  <tr><td>Definition</td><td><p>Start position of the window on the reference sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>1..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.referenceSeq.windowEnd"> </a><b>Sequence.referenceSeq.windowEnd</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.referenceSeq.windowEnd</td></tr>
  <tr><td>Definition</td><td><p>End position of the window on the reference sequence. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>1..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.variant"> </a><b>Sequence.variant</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.variant</td></tr>
  <tr><td>Definition</td><td><p>The definition of variant here originates from Sequence ontology (<a href="http://www.sequenceontology.org/browser/current_svn/term/variant_of">variant_of <img src="external.png" style="text-align: baseline"/></a>). This element can represent amino acid or nucleic sequence change(including insertion,deletion,SNP,etc.)  It can represent some complex mutation or segment variation with the assist of CIGAR string.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.variant.start"> </a><b>Sequence.variant.start</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.variant.start</td></tr>
  <tr><td>Definition</td><td><p>Start position of the variant on the  reference sequence.If the coordinate system is either 0-based or 1-based, then start position is inclusive.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.variant.end"> </a><b>Sequence.variant.end</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.variant.end</td></tr>
  <tr><td>Definition</td><td><p>End position of the variant on the reference sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.variant.observedAllele"> </a><b>Sequence.variant.observedAllele</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.variant.observedAllele</td></tr>
  <tr><td>Definition</td><td><p>An allele is one of a set of coexisting sequence variants of a gene (<a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0001023">SO:0001023 <img src="external.png" style="text-align: baseline"/></a>).  Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the observed  sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.variant.referenceAllele"> </a><b>Sequence.variant.referenceAllele</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.variant.referenceAllele</td></tr>
  <tr><td>Definition</td><td><p>An allele is one of a set of coexisting sequence variants of a gene (<a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0001023">SO:0001023 <img src="external.png" style="text-align: baseline"/></a>). Nucleotide(s)/amino acids from start position of sequence to stop position of sequence on the positive (+) strand of the reference sequence. When the sequence  type is DNA, it should be the sequence on the positive (+) strand. This will lay in the range between variant.start and variant.end.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.variant.cigar"> </a><b>Sequence.variant.cigar</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.variant.cigar</td></tr>
  <tr><td>Definition</td><td><p>Extended CIGAR string for aligning the sequence with reference bases. See detailed documentation <a href="http://support.illumina.com/help/SequencingAnalysisWorkflow/Content/Vault/Informatics/Sequencing_Analysis/CASAVA/swSEQ_mCA_ExtendedCIGARFormat.htm">here <img src="external.png" style="text-align: baseline"/></a>.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.variant.variantPointer"> </a><b>Sequence.variant.variantPointer</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.variant.variantPointer</td></tr>
  <tr><td>Definition</td><td><p>A pointer to an Observation containing variant information.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="references.html#Reference">Reference</a>(<a href="observation.html">Observation</a>)</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.observedSeq"> </a><b>Sequence.observedSeq</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.observedSeq</td></tr>
  <tr><td>Definition</td><td><p>Sequence that was observed. It is the result marked by referenceSeq along with variant records on referenceSeq. This shall starts from referenceSeq.windowStart and end by referenceSeq.windowEnd.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality"> </a><b>Sequence.quality</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality</td></tr>
  <tr><td>Definition</td><td><p>An experimental feature attribute that defines the quality of the feature in a quantitative way, such as a phred quality score (<a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0001686">SO:0001686 <img src="external.png" style="text-align: baseline"/></a>).</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.type"> </a><b>Sequence.quality.type</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.type</td></tr>
  <tr><td>Definition</td><td><p>INDEL / SNP / Undefined variant.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>1..1</td></tr>
  <tr><td><a href="terminologies.html">Terminology Binding</a></td><td><a href="valueset-quality-type.html">qualityType</a> (<a href="terminologies.html#required">Required</a>)</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#code">code</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.standardSequence"> </a><b>Sequence.quality.standardSequence</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.standardSequence</td></tr>
  <tr><td>Definition</td><td><p>Gold standard sequence used for comparing against.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="terminologies.html">Terminology Binding</a></td><td><a href="valueset-sequence-quality-standardSequence.html">F d a- standard sequence</a> (<a href="terminologies.html#example">Example</a>)</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.start"> </a><b>Sequence.quality.start</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.start</td></tr>
  <tr><td>Definition</td><td><p>Start position of the sequence. If the coordinate system is either 0-based or 1-based, then start position is inclusive.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.end"> </a><b>Sequence.quality.end</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.end</td></tr>
  <tr><td>Definition</td><td><p>End position of the sequence.If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.score"> </a><b>Sequence.quality.score</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.score</td></tr>
  <tr><td>Definition</td><td><p>The score of an experimentally derived feature such as a p-value (<a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0001685">SO:0001685 <img src="external.png" style="text-align: baseline"/></a>).</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#Quantity">Quantity</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.method"> </a><b>Sequence.quality.method</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.method</td></tr>
  <tr><td>Definition</td><td><p>Which method is used to get sequence quality.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="terminologies.html">Terminology Binding</a></td><td><a href="valueset-sequence-quality-method.html">F d a- method</a> (<a href="terminologies.html#example">Example</a>)</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#CodeableConcept">CodeableConcept</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.truthTP"> </a><b>Sequence.quality.truthTP</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.truthTP</td></tr>
  <tr><td>Definition</td><td><p>True positives, from the perspective of the truth data, i.e. the number of sites in the Truth Call Set for which there are paths through the Query Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.queryTP"> </a><b>Sequence.quality.queryTP</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.queryTP</td></tr>
  <tr><td>Definition</td><td><p>True positives, from the perspective of the query data, i.e. the number of sites in the Query Call Set for which there are paths through the Truth Call Set that are consistent with all of the alleles at this site, and for which there is an accurate genotype call for the event.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.truthFN"> </a><b>Sequence.quality.truthFN</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.truthFN</td></tr>
  <tr><td>Definition</td><td><p>False negatives, i.e. the number of sites in the Truth Call Set for which there is no path through the Query Call Set that is consistent with all of the alleles at this site, or sites for which there is an inaccurate genotype call for the event. Sites with correct variant but incorrect genotype are counted here.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.queryFP"> </a><b>Sequence.quality.queryFP</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.queryFP</td></tr>
  <tr><td>Definition</td><td><p>False positives, i.e. the number of sites in the Query Call Set for which there is no path through the Truth Call Set that is consistent with this site. Sites with correct variant but incorrect genotype are counted here.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.gtFP"> </a><b>Sequence.quality.gtFP</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.gtFP</td></tr>
  <tr><td>Definition</td><td><p>The number of false positives where the non-REF alleles in the Truth and Query Call Sets match (i.e. cases where the truth is 1/1 and the query is 0/1 or similar).</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.precision"> </a><b>Sequence.quality.precision</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.precision</td></tr>
  <tr><td>Definition</td><td><p>QUERY.TP / (QUERY.TP + QUERY.FP).</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.recall"> </a><b>Sequence.quality.recall</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.recall</td></tr>
  <tr><td>Definition</td><td><p>TRUTH.TP / (TRUTH.TP + TRUTH.FN).</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.fScore"> </a><b>Sequence.quality.fScore</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.fScore</td></tr>
  <tr><td>Definition</td><td><p>Harmonic mean of Recall and Precision, computed as: 2 * precision * recall / (precision + recall).</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.roc"> </a><b>Sequence.quality.roc</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.roc</td></tr>
  <tr><td>Definition</td><td><p>Receiver Operator Characteristic (ROC) Curve  to give sensitivity/specificity tradeoff.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.roc.score"> </a><b>Sequence.quality.roc.score</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.roc.score</td></tr>
  <tr><td>Definition</td><td><p>Invidual data point representing the GQ (genotype quality) score threshold.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.roc.numTP"> </a><b>Sequence.quality.roc.numTP</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.roc.numTP</td></tr>
  <tr><td>Definition</td><td><p>The number of true positives if the GQ score threshold was set to "score" field value.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.roc.numFP"> </a><b>Sequence.quality.roc.numFP</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.roc.numFP</td></tr>
  <tr><td>Definition</td><td><p>The number of false positives if the GQ score threshold was set to "score" field value.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.roc.numFN"> </a><b>Sequence.quality.roc.numFN</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.roc.numFN</td></tr>
  <tr><td>Definition</td><td><p>The number of false negatives if the GQ score threshold was set to "score" field value.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.roc.precision"> </a><b>Sequence.quality.roc.precision</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.roc.precision</td></tr>
  <tr><td>Definition</td><td><p>Calculated precision if the GQ score threshold was set to "score" field value.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.roc.sensitivity"> </a><b>Sequence.quality.roc.sensitivity</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.roc.sensitivity</td></tr>
  <tr><td>Definition</td><td><p>Calculated sensitivity if the GQ score threshold was set to "score" field value.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.quality.roc.fMeasure"> </a><b>Sequence.quality.roc.fMeasure</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.quality.roc.fMeasure</td></tr>
  <tr><td>Definition</td><td><p>Calculated fScore if the GQ score threshold was set to "score" field value.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.readCoverage"> </a><b>Sequence.readCoverage</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.readCoverage</td></tr>
  <tr><td>Definition</td><td><p>Coverage (read depth or depth) is the average number of reads representing a given nucleotide in the reconstructed sequence.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.repository"> </a><b>Sequence.repository</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.repository</td></tr>
  <tr><td>Definition</td><td><p>Configurations of the external repository. The repository shall store target's observedSeq or records related with target's observedSeq.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.repository.type"> </a><b>Sequence.repository.type</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.repository.type</td></tr>
  <tr><td>Definition</td><td><p>Click and see / RESTful API / Need login to see / RESTful API with authentication / Other ways to see resource.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>1..1</td></tr>
  <tr><td><a href="terminologies.html">Terminology Binding</a></td><td><a href="valueset-repository-type.html">repositoryType</a> (<a href="terminologies.html#required">Required</a>)</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#code">code</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.repository.url"> </a><b>Sequence.repository.url</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.repository.url</td></tr>
  <tr><td>Definition</td><td><p>URI of an external repository which contains further details about the genetics data.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#uri">uri</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.repository.name"> </a><b>Sequence.repository.name</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.repository.name</td></tr>
  <tr><td>Definition</td><td><p>URI of an external repository which contains further details about the genetics data.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.repository.datasetId"> </a><b>Sequence.repository.datasetId</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.repository.datasetId</td></tr>
  <tr><td>Definition</td><td><p>Id of the variant in this external repository. The server will understand how to use this id to call for more info about datasets in external repository.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.repository.variantsetId"> </a><b>Sequence.repository.variantsetId</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.repository.variantsetId</td></tr>
  <tr><td>Definition</td><td><p>Id of the variantset in this external repository. The server will understand how to use this id to call for more info about variantsets in external repository.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.repository.readsetId"> </a><b>Sequence.repository.readsetId</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.repository.readsetId</td></tr>
  <tr><td>Definition</td><td><p>Id of the read in this external repository.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.pointer"> </a><b>Sequence.pointer</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.pointer</td></tr>
  <tr><td>Definition</td><td><p>Pointer to next atomic sequence which at most contains one variant.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="references.html#Reference">Reference</a>(<a href="sequence.html">Sequence</a>)</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant"> </a><b>Sequence.structureVariant</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.structureVariant</td></tr>
  <tr><td>Definition</td><td><p>Information about chromosome structure variation.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..*</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.precision"> </a><b>Sequence.structureVariant.precision</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.structureVariant.precision</td></tr>
  <tr><td>Definition</td><td><p>Identify the exact boundaries of variant sequences. Each type of structure variant requires the DNA duplex to be broken and rejoined, and this creates a new sequence of bases at the rejoined sites, known as breakpoints or boundaries.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#string">string</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.reportedaCGHRatio"> </a><b>Sequence.structureVariant.reportedaCGHRatio</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.structureVariant.reportedaCGHRatio</td></tr>
  <tr><td>Definition</td><td><p>Structural Variant reported aCGH ratio.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#decimal">decimal</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.length"> </a><b>Sequence.structureVariant.length</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.structureVariant.length</td></tr>
  <tr><td>Definition</td><td><p>Length of the variant choromosome.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.outer"> </a><b>Sequence.structureVariant.outer</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.structureVariant.outer</td></tr>
  <tr><td>Definition</td><td><p>Structural variant outer.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.outer.start"> </a><b>Sequence.structureVariant.outer.start</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.structureVariant.outer.start</td></tr>
  <tr><td>Definition</td><td><p>Structural Variant Outer Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.outer.end"> </a><b>Sequence.structureVariant.outer.end</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.structureVariant.outer.end</td></tr>
  <tr><td>Definition</td><td><p>Structural Variant Outer End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.inner"> </a><b>Sequence.structureVariant.inner</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.structureVariant.inner</td></tr>
  <tr><td>Definition</td><td><p>Structural variant inner.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.inner.start"> </a><b>Sequence.structureVariant.inner.start</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.structureVariant.inner.start</td></tr>
  <tr><td>Definition</td><td><p>Structural Variant Inner Start.If the coordinate system is either 0-based or 1-based, then start position is inclusive.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
  <tr><td colspan="2" class="structure"><a name="Sequence.structureVariant.inner.end"> </a><b>Sequence.structureVariant.inner.end</b></td></tr>
  <tr><td>Element Id</td><td>Sequence.structureVariant.inner.end</td></tr>
  <tr><td>Definition</td><td><p>Structural Variant Inner End. If the coordinate system is 0-based then end is is exclusive and does not include the last position. If the coordinate system is 1-base, then end is inclusive and includes the last position.</p>
</td></tr>
  <tr><td><a href="conformance-rules.html#conformance">Control</a></td><td>0..1</td></tr>
  <tr><td><a href="datatypes.html">Type</a></td><td><a href="datatypes.html#integer">integer</a></td></tr>
  <tr><td><a href="search.html#summary">Summary</a></td><td>true</td></tr>
</table>



</div>


				</div>  <!-- /inner-wrapper -->
            </div>  <!-- /row -->
        </div>  <!-- /container -->
        
    </div>  <!-- /segment-content -->


	<div id="segment-footer" class="segment">  <!-- segment-footer -->
		<div class="container">  <!-- container -->
			<div class="inner-wrapper">
				<p>
        &reg;&copy; HL7.org 2011+. FHIR Release 4 Candidate (v3.5.0-????) generated on Fri, Sep 14, 2018 07:28-0500. <a style="color: #ffffff" href="qa.html">QA Page</a>
        <br/>
        <span style="color: #FFFF77">
        Links: <a style="color: #b8dcf9" href="http://hl7.org/fhir/search.cfm">Search <img src="external.png" style="text-align: baseline"/></a> | 
               <a style="color: #b8dcf9" href="history.html">Version History</a> | 
               <a style="color: #b8dcf9" href="toc.html">Table of Contents</a> | 
               <a style="color: #b8dcf9" href="credits.html">Credits</a> | 
               <a style="color: #b8dcf9" href="http://services.w3.org/htmldiff?doc1=http%3A%2F%2Fhl7.org%2Ffhir%2FSTU3%2Fsequence-definitions.html&amp;doc2=http%3A%2F%2Fbuild.fhir.org%2Fsequence-definitions.html">Compare to DSTU3 <img src="external.png" style="text-align: baseline"/></a> |                
               <a style="color: #b8dcf9" href="http://services.w3.org/htmldiff?doc1=http%3A%2F%2Fhl7.org%2Ffhir%2F2018May%2Fsequence-definitions.html&amp;doc2=http%3A%2F%2Fbuild.fhir.org%2Fsequence-definitions.html">Ballot Comparison <img src="external.png" style="text-align: baseline"/></a> |                
               <a rel="license" style="color: #b8dcf9" href="license.html"><img src="cc0.png" alt="CC0" style="border-style: none;"/></a> | 
               <a style="color: #b8dcf9" href="http://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemAdd&amp;tracker_id=677" target="_blank">Propose a change <img src="external.png" style="text-align: baseline"/></a>   
        </span>
        </p>
			</div>  <!-- /inner-wrapper -->
		</div>  <!-- /container -->
	</div>  <!-- /segment-footer -->
  <!-- disqus thread -->
  <!-- disqus -->
  <!-- end disqus -->        
	
	<div id="segment-post-footer" class="segment hidden">  <!-- segment-post-footer -->
		<div class="container">  <!-- container -->
		</div>  <!-- /container -->
	</div>  <!-- /segment-post-footer -->
    
      <!-- JS and analytics only. -->
      <!-- Bootstrap core JavaScript
================================================== -->
  <!-- Placed at the end of the document so the pages load faster -->
<script src="./assets/js/jquery.js"> </script>     <!-- note keep space here, otherwise it will be transformed to empty tag -> fails -->
<script src="./dist/js/bootstrap.min.js"> </script>
<script src="./assets/js/respond.min.js"> </script>

<script src="./assets/js/fhir.js"> </script>

  <!-- Analytics Below
================================================== -->

    
</body>
</html>
